CMakeLists.txt

```diff
@@ -3096,11 +3100,9 @@ if(SDL_STATIC)
     set_target_properties(SDL2-static PROPERTIES OUTPUT_NAME "SDL2")
   endif()
   set_target_properties(SDL2-static PROPERTIES POSITION_INDEPENDENT_CODE "${SDL_STATIC_PIC}")
-  # Note: The clang toolset for Visual Studio does not support /NODEFAULTLIB.
-  if(MSVC AND NOT SDL_LIBC AND NOT MSVC_CLANG AND NOT CMAKE_GENERATOR_PLATFORM STREQUAL "ARM")
-    set_target_properties(SDL2-static PROPERTIES LINK_FLAGS_RELEASE "/NODEFAULTLIB")
-    set_target_properties(SDL2-static PROPERTIES LINK_FLAGS_DEBUG "/NODEFAULTLIB")
-    set_target_properties(SDL2-static PROPERTIES STATIC_LIBRARY_FLAGS "/NODEFAULTLIB")
+  if(WIN32)
+    # The static SDL2 library always uses the system C library.
+    target_compile_definitions(SDL2-static PUBLIC HAVE_LIBC=1)
```

Comment:
I don't understand why this define is needed.
Without it, linking DevolutionX fails to link with the following error:

<details>

<summary>Link errors when building without HAVE_LIBC=1 for static SDL2</summary>

```
[1/25] cmd.exe /C "cmd.exe /C ""C:\Program Files\CMake\bin\cmake.exe" -E __create_def C:\Users\maarten\source\repos\devilutionX\build\test\CMakeFiles\libdevilutionx_so.dir\.\exports.def C:\Users\maarten\source\repos\devilutionX\build\test\CMakeFiles\libdevilutionx_so.dir\.\exports.def.objs && cd C:\Users\maarten\source\repos\devilutionX\build" && "C:\Program Files\CMake\bin\cmake.exe" -E vs_link_dll --intdir=test\CMakeFiles\libdevilutionx_so.dir --rc=C:\PROGRA~2\WI3CF2~1\10\bin\100190~1.0\x64\rc.exe --mt=C:\PROGRA~2\WI3CF2~1\10\bin\100190~1.0\x64\mt.exe --manifests  -- C:\PROGRA~2\MICROS~1\2019\COMMUN~1\VC\Tools\MSVC\1429~1.301\bin\Hostx64\x64\link.exe /nologo @CMakeFiles\libdevilutionx_so.rsp  /out:libdevilutionx_so.dll /implib:test\libdevilutionx_so.lib /pdb:libdevilutionx_so.pdb /dll /version:0.0 /machine:x64 /debug /INCREMENTAL  /DEF:test\CMakeFiles\libdevilutionx_so.dir\.\exports.def  && cd ."
FAILED: libdevilutionx_so.dll test/libdevilutionx_so.lib 
cmd.exe /C "cmd.exe /C ""C:\Program Files\CMake\bin\cmake.exe" -E __create_def C:\Users\maarten\source\repos\devilutionX\build\test\CMakeFiles\libdevilutionx_so.dir\.\exports.def C:\Users\maarten\source\repos\devilutionX\build\test\CMakeFiles\libdevilutionx_so.dir\.\exports.def.objs && cd C:\Users\maarten\source\repos\devilutionX\build" && "C:\Program Files\CMake\bin\cmake.exe" -E vs_link_dll --intdir=test\CMakeFiles\libdevilutionx_so.dir --rc=C:\PROGRA~2\WI3CF2~1\10\bin\100190~1.0\x64\rc.exe --mt=C:\PROGRA~2\WI3CF2~1\10\bin\100190~1.0\x64\mt.exe --manifests  -- C:\PROGRA~2\MICROS~1\2019\COMMUN~1\VC\Tools\MSVC\1429~1.301\bin\Hostx64\x64\link.exe /nologo @CMakeFiles\libdevilutionx_so.rsp  /out:libdevilutionx_so.dll /implib:test\libdevilutionx_so.lib /pdb:libdevilutionx_so.pdb /dll /version:0.0 /machine:x64 /debug /INCREMENTAL  /DEF:test\CMakeFiles\libdevilutionx_so.dir\.\exports.def  && cd ."
LINK Pass 1: command "C:\PROGRA~2\MICROS~1\2019\COMMUN~1\VC\Tools\MSVC\1429~1.301\bin\Hostx64\x64\link.exe /nologo @CMakeFiles\libdevilutionx_so.rsp /out:libdevilutionx_so.dll /implib:test\libdevilutionx_so.lib /pdb:libdevilutionx_so.pdb /dll /version:0.0 /machine:x64 /debug /INCREMENTAL /DEF:test\CMakeFiles\libdevilutionx_so.dir\.\exports.def /MANIFEST /MANIFESTFILE:test\CMakeFiles\libdevilutionx_so.dir/intermediate.manifest test\CMakeFiles\libdevilutionx_so.dir/manifest.res" failed (exit code 1120) with the following output:
   Creating library test\libdevilutionx_so.lib and object test\libdevilutionx_so.exp
LINK : warning LNK4217: symbol 'zts_init_from_storage' defined in 'zt.lib(Controls.cpp.obj)' is imported by 'zerotier_native.cpp.obj' in function '"void __cdecl devilution::net::zerotier_network_start(void)" (?zerotier_network_start@net@devilution@@YAXXZ)'
LINK : warning LNK4217: symbol 'zts_init_set_event_handler' defined in 'zt.lib(Controls.cpp.obj)' is imported by 'zerotier_native.cpp.obj' in function '"void __cdecl devilution::net::zerotier_network_start(void)" (?zerotier_network_start@net@devilution@@YAXXZ)'
LINK : warning LNK4217: symbol 'zts_net_join' defined in 'zt.lib(Controls.cpp.obj)' is imported by 'zerotier_native.cpp.obj' in function '"void __cdecl devilution::net::Callback(void *)" (?Callback@net@devilution@@YAXPEAX@Z)'
LINK : warning LNK4217: symbol 'zts_node_start' defined in 'zt.lib(Controls.cpp.obj)' is imported by 'zerotier_native.cpp.obj' in function '"void __cdecl devilution::net::zerotier_network_start(void)" (?zerotier_network_start@net@devilution@@YAXXZ)'
zt.lib(Topology.cpp.obj) : error LNK2001: unresolved external symbol __imp__invalid_parameter
zt.lib(Switch.cpp.obj) : error LNK2001: unresolved external symbol __imp__invalid_parameter
zt.lib(CertificateOfMembership.cpp.obj) : error LNK2001: unresolved external symbol __imp__invalid_parameter
zt.lib(IncomingPacket.cpp.obj) : error LNK2001: unresolved external symbol __imp__invalid_parameter
zt.lib(NetworkConfig.cpp.obj) : error LNK2001: unresolved external symbol __imp__invalid_parameter
zt.lib(Network.cpp.obj) : error LNK2001: unresolved external symbol __imp__invalid_parameter
zt.lib(Bond.cpp.obj) : error LNK2001: unresolved external symbol __imp__invalid_parameter
zt.lib(Peer.cpp.obj) : error LNK2001: unresolved external symbol __imp__invalid_parameter
zt.lib(PortMapper.cpp.obj) : error LNK2001: unresolved external symbol __imp__invalid_parameter
zt.lib(VirtualTap.cpp.obj) : error LNK2001: unresolved external symbol __imp__invalid_parameter
zt.lib(Multicaster.cpp.obj) : error LNK2001: unresolved external symbol __imp__invalid_parameter
zt.lib(Trace.cpp.obj) : error LNK2001: unresolved external symbol __imp__invalid_parameter
zt.lib(OSUtils.cpp.obj) : error LNK2001: unresolved external symbol __imp__invalid_parameter
zt.lib(NodeService.cpp.obj) : error LNK2001: unresolved external symbol __imp__invalid_parameter
zt.lib(Controls.cpp.obj) : error LNK2001: unresolved external symbol __imp__invalid_parameter
zt.lib(Node.cpp.obj) : error LNK2001: unresolved external symbol __imp__invalid_parameter
SDL_audiolib.lib(Stream.cpp.obj) : error LNK2001: unresolved external symbol __imp__invalid_parameter
SDL_audiolib.lib(stream_p.cpp.obj) : error LNK2001: unresolved external symbol __imp__invalid_parameter
zt.lib(Utilities.cpp.obj) : error LNK2001: unresolved external symbol __imp__invalid_parameter
zt.lib(Events.cpp.obj) : error LNK2001: unresolved external symbol __imp__invalid_parameter
renderers.cpp.obj : error LNK2001: unresolved external symbol __imp__invalid_parameter
fmtd.lib(format.cc.obj) : error LNK2001: unresolved external symbol __imp__invalid_parameter
libsmackerdec.lib(SmackerDecoder.cpp.obj) : error LNK2001: unresolved external symbol __imp__invalid_parameter
libsmackerdec.lib(LogError.cpp.obj) : error LNK2001: unresolved external symbol __imp__invalid_parameter
zerotier_native.cpp.obj : error LNK2001: unresolved external symbol __imp__invalid_parameter
zerotier_lwip.cpp.obj : error LNK2001: unresolved external symbol __imp__invalid_parameter
game_controller.cpp.obj : error LNK2001: unresolved external symbol __imp__invalid_parameter
gamepad.cpp.obj : error LNK2001: unresolved external symbol __imp__invalid_parameter
soundsample.cpp.obj : error LNK2001: unresolved external symbol __imp__invalid_parameter
tcp_client.cpp.obj : error LNK2001: unresolved external symbol __imp__invalid_parameter
tcp_server.cpp.obj : error LNK2001: unresolved external symbol __imp__invalid_parameter
protocol_zt.cpp.obj : error LNK2001: unresolved external symbol __imp__invalid_parameter
locale.cpp.obj : error LNK2001: unresolved external symbol __imp__invalid_parameter
effects.cpp.obj : error LNK2001: unresolved external symbol __imp__invalid_parameter
sound.cpp.obj : error LNK2001: unresolved external symbol __imp__invalid_parameter
push_aulib_decoder.cpp.obj : error LNK2001: unresolved external symbol __imp__invalid_parameter
packet.cpp.obj : error LNK2001: unresolved external symbol __imp__invalid_parameter
storm_net.cpp.obj : error LNK2001: unresolved external symbol __imp__invalid_parameter
storm_svid.cpp.obj : error LNK2001: unresolved external symbol __imp__invalid_parameter
misc_msg.cpp.obj : error LNK2001: unresolved external symbol __imp__invalid_parameter
abstract_net.cpp.obj : error LNK2001: unresolved external symbol __imp__invalid_parameter
base.cpp.obj : error LNK2001: unresolved external symbol __imp__invalid_parameter
frame_queue.cpp.obj : error LNK2001: unresolved external symbol __imp__invalid_parameter
loopback.cpp.obj : error LNK2001: unresolved external symbol __imp__invalid_parameter
mainpanel.cpp.obj : error LNK2001: unresolved external symbol __imp__invalid_parameter
spell_book.cpp.obj : error LNK2001: unresolved external symbol __imp__invalid_parameter
spell_icons.cpp.obj : error LNK2001: unresolved external symbol __imp__invalid_parameter
spell_list.cpp.obj : error LNK2001: unresolved external symbol __imp__invalid_parameter
selstart.cpp.obj : error LNK2001: unresolved external symbol __imp__invalid_parameter
selyesno.cpp.obj : error LNK2001: unresolved external symbol __imp__invalid_parameter
title.cpp.obj : error LNK2001: unresolved external symbol __imp__invalid_parameter
charpanel.cpp.obj : error LNK2001: unresolved external symbol __imp__invalid_parameter
selconn.cpp.obj : error LNK2001: unresolved external symbol __imp__invalid_parameter
selgame.cpp.obj : error LNK2001: unresolved external symbol __imp__invalid_parameter
selhero.cpp.obj : error LNK2001: unresolved external symbol __imp__invalid_parameter
selok.cpp.obj : error LNK2001: unresolved external symbol __imp__invalid_parameter
dialogs.cpp.obj : error LNK2001: unresolved external symbol __imp__invalid_parameter
mainmenu.cpp.obj : error LNK2001: unresolved external symbol __imp__invalid_parameter
progress.cpp.obj : error LNK2001: unresolved external symbol __imp__invalid_parameter
settingsmenu.cpp.obj : error LNK2001: unresolved external symbol __imp__invalid_parameter
utf8.cpp.obj : error LNK2001: unresolved external symbol __imp__invalid_parameter
art.cpp.obj : error LNK2001: unresolved external symbol __imp__invalid_parameter
credits.cpp.obj : error LNK2001: unresolved external symbol __imp__invalid_parameter
diabloui.cpp.obj : error LNK2001: unresolved external symbol __imp__invalid_parameter
file_util.cpp.obj : error LNK2001: unresolved external symbol __imp__invalid_parameter
language.cpp.obj : error LNK2001: unresolved external symbol __imp__invalid_parameter
logged_fstream.cpp.obj : error LNK2001: unresolved external symbol __imp__invalid_parameter
paths.cpp.obj : error LNK2001: unresolved external symbol __imp__invalid_parameter
xpbar.cpp.obj : error LNK2001: unresolved external symbol __imp__invalid_parameter
itemlabels.cpp.obj : error LNK2001: unresolved external symbol __imp__invalid_parameter
console.cpp.obj : error LNK2001: unresolved external symbol __imp__invalid_parameter
display.cpp.obj : error LNK2001: unresolved external symbol __imp__invalid_parameter
mpq_writer.cpp.obj : error LNK2001: unresolved external symbol __imp__invalid_parameter
autopickup.cpp.obj : error LNK2001: unresolved external symbol __imp__invalid_parameter
chatlog.cpp.obj : error LNK2001: unresolved external symbol __imp__invalid_parameter
stash.cpp.obj : error LNK2001: unresolved external symbol __imp__invalid_parameter
text_render.cpp.obj : error LNK2001: unresolved external symbol __imp__invalid_parameter
trn.cpp.obj : error LNK2001: unresolved external symbol __imp__invalid_parameter
mpq_reader.cpp.obj : error LNK2001: unresolved external symbol __imp__invalid_parameter
mpq_sdl_rwops.cpp.obj : error LNK2001: unresolved external symbol __imp__invalid_parameter
demomode.cpp.obj : error LNK2001: unresolved external symbol __imp__invalid_parameter
cel_render.cpp.obj : error LNK2001: unresolved external symbol __imp__invalid_parameter
cl2_render.cpp.obj : error LNK2001: unresolved external symbol __imp__invalid_parameter
dun_render.cpp.obj : error LNK2001: unresolved external symbol __imp__invalid_parameter
joystick.cpp.obj : error LNK2001: unresolved external symbol __imp__invalid_parameter
modifier_hints.cpp.obj : error LNK2001: unresolved external symbol __imp__invalid_parameter
plrctrls.cpp.obj : error LNK2001: unresolved external symbol __imp__invalid_parameter
animationinfo.cpp.obj : error LNK2001: unresolved external symbol __imp__invalid_parameter
tmsg.cpp.obj : error LNK2001: unresolved external symbol __imp__invalid_parameter
towners.cpp.obj : error LNK2001: unresolved external symbol __imp__invalid_parameter
trigs.cpp.obj : error LNK2001: unresolved external symbol __imp__invalid_parameter
controller_motion.cpp.obj : error LNK2001: unresolved external symbol __imp__invalid_parameter
restrict.cpp.obj : error LNK2001: unresolved external symbol __imp__invalid_parameter
scrollrt.cpp.obj : error LNK2001: unresolved external symbol __imp__invalid_parameter
stores.cpp.obj : error LNK2001: unresolved external symbol __imp__invalid_parameter
themes.cpp.obj : error LNK2001: unresolved external symbol __imp__invalid_parameter
player.cpp.obj : error LNK2001: unresolved external symbol __imp__invalid_parameter
plrmsg.cpp.obj : error LNK2001: unresolved external symbol __imp__invalid_parameter
portal.cpp.obj : error LNK2001: unresolved external symbol __imp__invalid_parameter
quests.cpp.obj : error LNK2001: unresolved external symbol __imp__invalid_parameter
options.cpp.obj : error LNK2001: unresolved external symbol __imp__invalid_parameter
palette.cpp.obj : error LNK2001: unresolved external symbol __imp__invalid_parameter
path.cpp.obj : error LNK2001: unresolved external symbol __imp__invalid_parameter
pfile.cpp.obj : error LNK2001: unresolved external symbol __imp__invalid_parameter
monster.cpp.obj : error LNK2001: unresolved external symbol __imp__invalid_parameter
msg.cpp.obj : error LNK2001: unresolved external symbol __imp__invalid_parameter
multi.cpp.obj : error LNK2001: unresolved external symbol __imp__invalid_parameter
objects.cpp.obj : error LNK2001: unresolved external symbol __imp__invalid_parameter
loadsave.cpp.obj : error LNK2001: unresolved external symbol __imp__invalid_parameter
minitext.cpp.obj : error LNK2001: unresolved external symbol __imp__invalid_parameter
misdat.cpp.obj : error LNK2001: unresolved external symbol __imp__invalid_parameter
missiles.cpp.obj : error LNK2001: unresolved external symbol __imp__invalid_parameter
interfac.cpp.obj : error LNK2001: unresolved external symbol __imp__invalid_parameter
inv.cpp.obj : error LNK2001: unresolved external symbol __imp__invalid_parameter
items.cpp.obj : error LNK2001: unresolved external symbol __imp__invalid_parameter
lighting.cpp.obj : error LNK2001: unresolved external symbol __imp__invalid_parameter
gendung.cpp.obj : error LNK2001: unresolved external symbol __imp__invalid_parameter
gmenu.cpp.obj : error LNK2001: unresolved external symbol __imp__invalid_parameter
help.cpp.obj : error LNK2001: unresolved external symbol __imp__invalid_parameter
init.cpp.obj : error LNK2001: unresolved external symbol __imp__invalid_parameter
dx.cpp.obj : error LNK2001: unresolved external symbol __imp__invalid_parameter
encrypt.cpp.obj : error LNK2001: unresolved external symbol __imp__invalid_parameter
error.cpp.obj : error LNK2001: unresolved external symbol __imp__invalid_parameter
assets.cpp.obj : error LNK2001: unresolved external symbol __imp__invalid_parameter
diablo.cpp.obj : error LNK2001: unresolved external symbol __imp__invalid_parameter
doom.cpp.obj : error LNK2001: unresolved external symbol __imp__invalid_parameter
drlg_l2.cpp.obj : error LNK2001: unresolved external symbol __imp__invalid_parameter
dthread.cpp.obj : error LNK2001: unresolved external symbol __imp__invalid_parameter
control.cpp.obj : error LNK2001: unresolved external symbol __imp__invalid_parameter
cursor.cpp.obj : error LNK2001: unresolved external symbol __imp__invalid_parameter
dead.cpp.obj : error LNK2001: unresolved external symbol __imp__invalid_parameter
debug.cpp.obj : error LNK2001: unresolved external symbol __imp__invalid_parameter
appfat.cpp.obj : error LNK2001: unresolved external symbol __imp__invalid_parameter
automap.cpp.obj : error LNK2001: unresolved external symbol __imp__invalid_parameter
capture.cpp.obj : error LNK2001: unresolved external symbol __imp__invalid_parameter
codec.cpp.obj : error LNK2001: unresolved external symbol __imp__invalid_parameter
zt.lib(Topology.cpp.obj) : error LNK2001: unresolved external symbol __imp__CrtDbgReport
zt.lib(Switch.cpp.obj) : error LNK2001: unresolved external symbol __imp__CrtDbgReport
zt.lib(CertificateOfMembership.cpp.obj) : error LNK2001: unresolved external symbol __imp__CrtDbgReport
zt.lib(IncomingPacket.cpp.obj) : error LNK2001: unresolved external symbol __imp__CrtDbgReport
zt.lib(NetworkConfig.cpp.obj) : error LNK2001: unresolved external symbol __imp__CrtDbgReport
zt.lib(Network.cpp.obj) : error LNK2001: unresolved external symbol __imp__CrtDbgReport
zt.lib(Bond.cpp.obj) : error LNK2001: unresolved external symbol __imp__CrtDbgReport
zt.lib(Peer.cpp.obj) : error LNK2001: unresolved external symbol __imp__CrtDbgReport
zt.lib(PortMapper.cpp.obj) : error LNK2001: unresolved external symbol __imp__CrtDbgReport
zt.lib(VirtualTap.cpp.obj) : error LNK2001: unresolved external symbol __imp__CrtDbgReport
zt.lib(Multicaster.cpp.obj) : error LNK2001: unresolved external symbol __imp__CrtDbgReport
zt.lib(Trace.cpp.obj) : error LNK2001: unresolved external symbol __imp__CrtDbgReport
zt.lib(OSUtils.cpp.obj) : error LNK2001: unresolved external symbol __imp__CrtDbgReport
zt.lib(NodeService.cpp.obj) : error LNK2001: unresolved external symbol __imp__CrtDbgReport
zt.lib(Controls.cpp.obj) : error LNK2001: unresolved external symbol __imp__CrtDbgReport
zt.lib(Node.cpp.obj) : error LNK2001: unresolved external symbol __imp__CrtDbgReport
SDL_audiolib.lib(Stream.cpp.obj) : error LNK2001: unresolved external symbol __imp__CrtDbgReport
SDL_audiolib.lib(stream_p.cpp.obj) : error LNK2001: unresolved external symbol __imp__CrtDbgReport
zt.lib(Utilities.cpp.obj) : error LNK2001: unresolved external symbol __imp__CrtDbgReport
zt.lib(Events.cpp.obj) : error LNK2001: unresolved external symbol __imp__CrtDbgReport
renderers.cpp.obj : error LNK2001: unresolved external symbol __imp__CrtDbgReport
fmtd.lib(format.cc.obj) : error LNK2001: unresolved external symbol __imp__CrtDbgReport
libsmackerdec.lib(SmackerDecoder.cpp.obj) : error LNK2001: unresolved external symbol __imp__CrtDbgReport
libsmackerdec.lib(LogError.cpp.obj) : error LNK2001: unresolved external symbol __imp__CrtDbgReport
zerotier_native.cpp.obj : error LNK2001: unresolved external symbol __imp__CrtDbgReport
zerotier_lwip.cpp.obj : error LNK2001: unresolved external symbol __imp__CrtDbgReport
game_controller.cpp.obj : error LNK2001: unresolved external symbol __imp__CrtDbgReport
gamepad.cpp.obj : error LNK2001: unresolved external symbol __imp__CrtDbgReport
soundsample.cpp.obj : error LNK2001: unresolved external symbol __imp__CrtDbgReport
tcp_client.cpp.obj : error LNK2001: unresolved external symbol __imp__CrtDbgReport
tcp_server.cpp.obj : error LNK2001: unresolved external symbol __imp__CrtDbgReport
protocol_zt.cpp.obj : error LNK2001: unresolved external symbol __imp__CrtDbgReport
locale.cpp.obj : error LNK2001: unresolved external symbol __imp__CrtDbgReport
effects.cpp.obj : error LNK2001: unresolved external symbol __imp__CrtDbgReport
sound.cpp.obj : error LNK2001: unresolved external symbol __imp__CrtDbgReport
push_aulib_decoder.cpp.obj : error LNK2001: unresolved external symbol __imp__CrtDbgReport
packet.cpp.obj : error LNK2001: unresolved external symbol __imp__CrtDbgReport
storm_net.cpp.obj : error LNK2001: unresolved external symbol __imp__CrtDbgReport
storm_svid.cpp.obj : error LNK2001: unresolved external symbol __imp__CrtDbgReport
misc_msg.cpp.obj : error LNK2001: unresolved external symbol __imp__CrtDbgReport
abstract_net.cpp.obj : error LNK2001: unresolved external symbol __imp__CrtDbgReport
base.cpp.obj : error LNK2001: unresolved external symbol __imp__CrtDbgReport
frame_queue.cpp.obj : error LNK2001: unresolved external symbol __imp__CrtDbgReport
loopback.cpp.obj : error LNK2001: unresolved external symbol __imp__CrtDbgReport
mainpanel.cpp.obj : error LNK2001: unresolved external symbol __imp__CrtDbgReport
spell_book.cpp.obj : error LNK2001: unresolved external symbol __imp__CrtDbgReport
spell_icons.cpp.obj : error LNK2001: unresolved external symbol __imp__CrtDbgReport
spell_list.cpp.obj : error LNK2001: unresolved external symbol __imp__CrtDbgReport
selstart.cpp.obj : error LNK2001: unresolved external symbol __imp__CrtDbgReport
selyesno.cpp.obj : error LNK2001: unresolved external symbol __imp__CrtDbgReport
title.cpp.obj : error LNK2001: unresolved external symbol __imp__CrtDbgReport
charpanel.cpp.obj : error LNK2001: unresolved external symbol __imp__CrtDbgReport
selconn.cpp.obj : error LNK2001: unresolved external symbol __imp__CrtDbgReport
selgame.cpp.obj : error LNK2001: unresolved external symbol __imp__CrtDbgReport
selhero.cpp.obj : error LNK2001: unresolved external symbol __imp__CrtDbgReport
selok.cpp.obj : error LNK2001: unresolved external symbol __imp__CrtDbgReport
dialogs.cpp.obj : error LNK2001: unresolved external symbol __imp__CrtDbgReport
mainmenu.cpp.obj : error LNK2001: unresolved external symbol __imp__CrtDbgReport
progress.cpp.obj : error LNK2001: unresolved external symbol __imp__CrtDbgReport
settingsmenu.cpp.obj : error LNK2001: unresolved external symbol __imp__CrtDbgReport
utf8.cpp.obj : error LNK2001: unresolved external symbol __imp__CrtDbgReport
art.cpp.obj : error LNK2001: unresolved external symbol __imp__CrtDbgReport
credits.cpp.obj : error LNK2001: unresolved external symbol __imp__CrtDbgReport
diabloui.cpp.obj : error LNK2001: unresolved external symbol __imp__CrtDbgReport
file_util.cpp.obj : error LNK2001: unresolved external symbol __imp__CrtDbgReport
language.cpp.obj : error LNK2001: unresolved external symbol __imp__CrtDbgReport
logged_fstream.cpp.obj : error LNK2001: unresolved external symbol __imp__CrtDbgReport
paths.cpp.obj : error LNK2001: unresolved external symbol __imp__CrtDbgReport
xpbar.cpp.obj : error LNK2001: unresolved external symbol __imp__CrtDbgReport
itemlabels.cpp.obj : error LNK2001: unresolved external symbol __imp__CrtDbgReport
console.cpp.obj : error LNK2001: unresolved external symbol __imp__CrtDbgReport
display.cpp.obj : error LNK2001: unresolved external symbol __imp__CrtDbgReport
mpq_writer.cpp.obj : error LNK2001: unresolved external symbol __imp__CrtDbgReport
autopickup.cpp.obj : error LNK2001: unresolved external symbol __imp__CrtDbgReport
chatlog.cpp.obj : error LNK2001: unresolved external symbol __imp__CrtDbgReport
stash.cpp.obj : error LNK2001: unresolved external symbol __imp__CrtDbgReport
text_render.cpp.obj : error LNK2001: unresolved external symbol __imp__CrtDbgReport
trn.cpp.obj : error LNK2001: unresolved external symbol __imp__CrtDbgReport
mpq_reader.cpp.obj : error LNK2001: unresolved external symbol __imp__CrtDbgReport
mpq_sdl_rwops.cpp.obj : error LNK2001: unresolved external symbol __imp__CrtDbgReport
demomode.cpp.obj : error LNK2001: unresolved external symbol __imp__CrtDbgReport
cel_render.cpp.obj : error LNK2001: unresolved external symbol __imp__CrtDbgReport
cl2_render.cpp.obj : error LNK2001: unresolved external symbol __imp__CrtDbgReport
dun_render.cpp.obj : error LNK2001: unresolved external symbol __imp__CrtDbgReport
joystick.cpp.obj : error LNK2001: unresolved external symbol __imp__CrtDbgReport
modifier_hints.cpp.obj : error LNK2001: unresolved external symbol __imp__CrtDbgReport
plrctrls.cpp.obj : error LNK2001: unresolved external symbol __imp__CrtDbgReport
animationinfo.cpp.obj : error LNK2001: unresolved external symbol __imp__CrtDbgReport
tmsg.cpp.obj : error LNK2001: unresolved external symbol __imp__CrtDbgReport
towners.cpp.obj : error LNK2001: unresolved external symbol __imp__CrtDbgReport
trigs.cpp.obj : error LNK2001: unresolved external symbol __imp__CrtDbgReport
controller_motion.cpp.obj : error LNK2001: unresolved external symbol __imp__CrtDbgReport
restrict.cpp.obj : error LNK2001: unresolved external symbol __imp__CrtDbgReport
scrollrt.cpp.obj : error LNK2001: unresolved external symbol __imp__CrtDbgReport
stores.cpp.obj : error LNK2001: unresolved external symbol __imp__CrtDbgReport
themes.cpp.obj : error LNK2001: unresolved external symbol __imp__CrtDbgReport
player.cpp.obj : error LNK2001: unresolved external symbol __imp__CrtDbgReport
plrmsg.cpp.obj : error LNK2001: unresolved external symbol __imp__CrtDbgReport
portal.cpp.obj : error LNK2001: unresolved external symbol __imp__CrtDbgReport
quests.cpp.obj : error LNK2001: unresolved external symbol __imp__CrtDbgReport
options.cpp.obj : error LNK2001: unresolved external symbol __imp__CrtDbgReport
palette.cpp.obj : error LNK2001: unresolved external symbol __imp__CrtDbgReport
path.cpp.obj : error LNK2001: unresolved external symbol __imp__CrtDbgReport
pfile.cpp.obj : error LNK2001: unresolved external symbol __imp__CrtDbgReport
monster.cpp.obj : error LNK2001: unresolved external symbol __imp__CrtDbgReport
msg.cpp.obj : error LNK2001: unresolved external symbol __imp__CrtDbgReport
multi.cpp.obj : error LNK2001: unresolved external symbol __imp__CrtDbgReport
objects.cpp.obj : error LNK2001: unresolved external symbol __imp__CrtDbgReport
loadsave.cpp.obj : error LNK2001: unresolved external symbol __imp__CrtDbgReport
minitext.cpp.obj : error LNK2001: unresolved external symbol __imp__CrtDbgReport
misdat.cpp.obj : error LNK2001: unresolved external symbol __imp__CrtDbgReport
missiles.cpp.obj : error LNK2001: unresolved external symbol __imp__CrtDbgReport
interfac.cpp.obj : error LNK2001: unresolved external symbol __imp__CrtDbgReport
inv.cpp.obj : error LNK2001: unresolved external symbol __imp__CrtDbgReport
items.cpp.obj : error LNK2001: unresolved external symbol __imp__CrtDbgReport
lighting.cpp.obj : error LNK2001: unresolved external symbol __imp__CrtDbgReport
gendung.cpp.obj : error LNK2001: unresolved external symbol __imp__CrtDbgReport
gmenu.cpp.obj : error LNK2001: unresolved external symbol __imp__CrtDbgReport
help.cpp.obj : error LNK2001: unresolved external symbol __imp__CrtDbgReport
init.cpp.obj : error LNK2001: unresolved external symbol __imp__CrtDbgReport
dx.cpp.obj : error LNK2001: unresolved external symbol __imp__CrtDbgReport
encrypt.cpp.obj : error LNK2001: unresolved external symbol __imp__CrtDbgReport
error.cpp.obj : error LNK2001: unresolved external symbol __imp__CrtDbgReport
assets.cpp.obj : error LNK2001: unresolved external symbol __imp__CrtDbgReport
diablo.cpp.obj : error LNK2001: unresolved external symbol __imp__CrtDbgReport
doom.cpp.obj : error LNK2001: unresolved external symbol __imp__CrtDbgReport
drlg_l2.cpp.obj : error LNK2001: unresolved external symbol __imp__CrtDbgReport
dthread.cpp.obj : error LNK2001: unresolved external symbol __imp__CrtDbgReport
control.cpp.obj : error LNK2001: unresolved external symbol __imp__CrtDbgReport
cursor.cpp.obj : error LNK2001: unresolved external symbol __imp__CrtDbgReport
dead.cpp.obj : error LNK2001: unresolved external symbol __imp__CrtDbgReport
debug.cpp.obj : error LNK2001: unresolved external symbol __imp__CrtDbgReport
appfat.cpp.obj : error LNK2001: unresolved external symbol __imp__CrtDbgReport
automap.cpp.obj : error LNK2001: unresolved external symbol __imp__CrtDbgReport
capture.cpp.obj : error LNK2001: unresolved external symbol __imp__CrtDbgReport
codec.cpp.obj : error LNK2001: unresolved external symbol __imp__CrtDbgReport
zt.lib(miniwget.c.obj) : error LNK2001: unresolved external symbol __imp___stdio_common_vsprintf
zt.lib(minisoap.c.obj) : error LNK2001: unresolved external symbol __imp___stdio_common_vsprintf
zt.lib(connecthostport.c.obj) : error LNK2001: unresolved external symbol __imp___stdio_common_vsprintf
zt.lib(VirtualTap.cpp.obj) : error LNK2001: unresolved external symbol __imp___stdio_common_vsprintf
zt.lib(Bond.cpp.obj) : error LNK2001: unresolved external symbol __imp___stdio_common_vsprintf
zt.lib(miniupnpc.c.obj) : error LNK2001: unresolved external symbol __imp___stdio_common_vsprintf
zt.lib(minissdpc.c.obj) : error LNK2001: unresolved external symbol __imp___stdio_common_vsprintf
diabloui.cpp.obj : error LNK2001: unresolved external symbol __imp___stdio_common_vsprintf
effects.cpp.obj : error LNK2001: unresolved external symbol __imp___stdio_common_vsprintf
zt.lib(sys_arch.c.obj) : error LNK2001: unresolved external symbol __imp___stdio_common_vsprintf
zt.lib(OSUtils.cpp.obj) : error LNK2001: unresolved external symbol __imp___stdio_common_vsprintf
stores.cpp.obj : error LNK2001: unresolved external symbol __imp___stdio_common_vsprintf
demomode.cpp.obj : error LNK2001: unresolved external symbol __imp___stdio_common_vsprintf
text_render.cpp.obj : error LNK2001: unresolved external symbol __imp___stdio_common_vsprintf
console.cpp.obj : error LNK2001: unresolved external symbol __imp___stdio_common_vsprintf
palette.cpp.obj : error LNK2001: unresolved external symbol __imp___stdio_common_vsprintf
pfile.cpp.obj : error LNK2001: unresolved external symbol __imp___stdio_common_vsprintf
player.cpp.obj : error LNK2001: unresolved external symbol __imp___stdio_common_vsprintf
scrollrt.cpp.obj : error LNK2001: unresolved external symbol __imp___stdio_common_vsprintf
init.cpp.obj : error LNK2001: unresolved external symbol __imp___stdio_common_vsprintf
items.cpp.obj : error LNK2001: unresolved external symbol __imp___stdio_common_vsprintf
monster.cpp.obj : error LNK2001: unresolved external symbol __imp___stdio_common_vsprintf
objects.cpp.obj : error LNK2001: unresolved external symbol __imp___stdio_common_vsprintf
appfat.cpp.obj : error LNK2001: unresolved external symbol __imp___stdio_common_vsprintf
capture.cpp.obj : error LNK2001: unresolved external symbol __imp___stdio_common_vsprintf
debug.cpp.obj : error LNK2001: unresolved external symbol __imp___stdio_common_vsprintf
diablo.cpp.obj : error LNK2001: unresolved external symbol __imp___stdio_common_vsprintf
msvcprtd.lib(locale0_implib.obj) : error LNK2001: unresolved external symbol __CxxFrameHandler4
MSVCRTD.lib(gshandlereh4.obj) : error LNK2001: unresolved external symbol __CxxFrameHandler4
MSVCRTD.lib(new_scalar_nothrow.obj) : error LNK2001: unresolved external symbol __CxxFrameHandler4
zt.lib(IncomingPacket.cpp.obj) : error LNK2001: unresolved external symbol __CxxFrameHandler4
zt.lib(Capability.cpp.obj) : error LNK2001: unresolved external symbol __CxxFrameHandler4
zt.lib(Tag.cpp.obj) : error LNK2001: unresolved external symbol __CxxFrameHandler4
zt.lib(Revocation.cpp.obj) : error LNK2001: unresolved external symbol __CxxFrameHandler4
zt.lib(OutboundMulticast.cpp.obj) : error LNK2001: unresolved external symbol __CxxFrameHandler4
zt.lib(CertificateOfMembership.cpp.obj) : error LNK2001: unresolved external symbol __CxxFrameHandler4
zt.lib(CertificateOfOwnership.cpp.obj) : error LNK2001: unresolved external symbol __CxxFrameHandler4
zt.lib(Membership.cpp.obj) : error LNK2001: unresolved external symbol __CxxFrameHandler4
zt.lib(Peer.cpp.obj) : error LNK2001: unresolved external symbol __CxxFrameHandler4
zt.lib(Topology.cpp.obj) : error LNK2001: unresolved external symbol __CxxFrameHandler4
zt.lib(Switch.cpp.obj) : error LNK2001: unresolved external symbol __CxxFrameHandler4
zt.lib(SelfAwareness.cpp.obj) : error LNK2001: unresolved external symbol __CxxFrameHandler4
zt.lib(Multicaster.cpp.obj) : error LNK2001: unresolved external symbol __CxxFrameHandler4
zt.lib(Trace.cpp.obj) : error LNK2001: unresolved external symbol __CxxFrameHandler4
zt.lib(Network.cpp.obj) : error LNK2001: unresolved external symbol __CxxFrameHandler4
zt.lib(Bond.cpp.obj) : error LNK2001: unresolved external symbol __CxxFrameHandler4
zt.lib(Node.cpp.obj) : error LNK2001: unresolved external symbol __CxxFrameHandler4
zt.lib(PortMapper.cpp.obj) : error LNK2001: unresolved external symbol __CxxFrameHandler4
zt.lib(VirtualTap.cpp.obj) : error LNK2001: unresolved external symbol __CxxFrameHandler4
zt.lib(Packet.cpp.obj) : error LNK2001: unresolved external symbol __CxxFrameHandler4
zt.lib(Identity.cpp.obj) : error LNK2001: unresolved external symbol __CxxFrameHandler4
zt.lib(OSUtils.cpp.obj) : error LNK2001: unresolved external symbol __CxxFrameHandler4
zt.lib(NodeService.cpp.obj) : error LNK2001: unresolved external symbol __CxxFrameHandler4
zt.lib(Controls.cpp.obj) : error LNK2001: unresolved external symbol __CxxFrameHandler4
SDL_audiolib.lib(stream_p.cpp.obj) : error LNK2001: unresolved external symbol __CxxFrameHandler4
zt.lib(Utilities.cpp.obj) : error LNK2001: unresolved external symbol __CxxFrameHandler4
zt.lib(Events.cpp.obj) : error LNK2001: unresolved external symbol __CxxFrameHandler4
zt.lib(Utils.cpp.obj) : error LNK2001: unresolved external symbol __CxxFrameHandler4
SDL_audiolib.lib(aulib.cpp.obj) : error LNK2001: unresolved external symbol __CxxFrameHandler4
SDL_audiolib.lib(DecoderDrmp3.cpp.obj) : error LNK2001: unresolved external symbol __CxxFrameHandler4
SDL_audiolib.lib(DecoderDrwav.cpp.obj) : error LNK2001: unresolved external symbol __CxxFrameHandler4
SDL_audiolib.lib(Resampler.cpp.obj) : error LNK2001: unresolved external symbol __CxxFrameHandler4
libsmackerdec.lib(LogError.cpp.obj) : error LNK2001: unresolved external symbol __CxxFrameHandler4
SDL_audiolib.lib(ResamplerSpeex.cpp.obj) : error LNK2001: unresolved external symbol __CxxFrameHandler4
SDL_audiolib.lib(Stream.cpp.obj) : error LNK2001: unresolved external symbol __CxxFrameHandler4
SDL_audiolib.lib(Decoder.cpp.obj) : error LNK2001: unresolved external symbol __CxxFrameHandler4
gamepad.cpp.obj : error LNK2001: unresolved external symbol __CxxFrameHandler4
renderers.cpp.obj : error LNK2001: unresolved external symbol __CxxFrameHandler4
fmtd.lib(format.cc.obj) : error LNK2001: unresolved external symbol __CxxFrameHandler4
libsmackerdec.lib(SmackerDecoder.cpp.obj) : error LNK2001: unresolved external symbol __CxxFrameHandler4
protocol_zt.cpp.obj : error LNK2001: unresolved external symbol __CxxFrameHandler4
zerotier_native.cpp.obj : error LNK2001: unresolved external symbol __CxxFrameHandler4
zerotier_lwip.cpp.obj : error LNK2001: unresolved external symbol __CxxFrameHandler4
game_controller.cpp.obj : error LNK2001: unresolved external symbol __CxxFrameHandler4
push_aulib_decoder.cpp.obj : error LNK2001: unresolved external symbol __CxxFrameHandler4
soundsample.cpp.obj : error LNK2001: unresolved external symbol __CxxFrameHandler4
tcp_client.cpp.obj : error LNK2001: unresolved external symbol __CxxFrameHandler4
tcp_server.cpp.obj : error LNK2001: unresolved external symbol __CxxFrameHandler4
storm_svid.cpp.obj : error LNK2001: unresolved external symbol __CxxFrameHandler4
misc_msg.cpp.obj : error LNK2001: unresolved external symbol __CxxFrameHandler4
locale.cpp.obj : error LNK2001: unresolved external symbol __CxxFrameHandler4
sound.cpp.obj : error LNK2001: unresolved external symbol __CxxFrameHandler4
frame_queue.cpp.obj : error LNK2001: unresolved external symbol __CxxFrameHandler4
loopback.cpp.obj : error LNK2001: unresolved external symbol __CxxFrameHandler4
packet.cpp.obj : error LNK2001: unresolved external symbol __CxxFrameHandler4
storm_net.cpp.obj : error LNK2001: unresolved external symbol __CxxFrameHandler4
spell_icons.cpp.obj : error LNK2001: unresolved external symbol __CxxFrameHandler4
spell_list.cpp.obj : error LNK2001: unresolved external symbol __CxxFrameHandler4
abstract_net.cpp.obj : error LNK2001: unresolved external symbol __CxxFrameHandler4
base.cpp.obj : error LNK2001: unresolved external symbol __CxxFrameHandler4
charpanel.cpp.obj : error LNK2001: unresolved external symbol __CxxFrameHandler4
info_box.cpp.obj : error LNK2001: unresolved external symbol __CxxFrameHandler4
mainpanel.cpp.obj : error LNK2001: unresolved external symbol __CxxFrameHandler4
spell_book.cpp.obj : error LNK2001: unresolved external symbol __CxxFrameHandler4
selok.cpp.obj : error LNK2001: unresolved external symbol __CxxFrameHandler4
selstart.cpp.obj : error LNK2001: unresolved external symbol __CxxFrameHandler4
selyesno.cpp.obj : error LNK2001: unresolved external symbol __CxxFrameHandler4
title.cpp.obj : error LNK2001: unresolved external symbol __CxxFrameHandler4
settingsmenu.cpp.obj : error LNK2001: unresolved external symbol __CxxFrameHandler4
selconn.cpp.obj : error LNK2001: unresolved external symbol __CxxFrameHandler4
selgame.cpp.obj : error LNK2001: unresolved external symbol __CxxFrameHandler4
selhero.cpp.obj : error LNK2001: unresolved external symbol __CxxFrameHandler4
dialogs.cpp.obj : error LNK2001: unresolved external symbol __CxxFrameHandler4
mainmenu.cpp.obj : error LNK2001: unresolved external symbol __CxxFrameHandler4
progress.cpp.obj : error LNK2001: unresolved external symbol __CxxFrameHandler4
scrollbar.cpp.obj : error LNK2001: unresolved external symbol __CxxFrameHandler4
art.cpp.obj : error LNK2001: unresolved external symbol __CxxFrameHandler4
button.cpp.obj : error LNK2001: unresolved external symbol __CxxFrameHandler4
credits.cpp.obj : error LNK2001: unresolved external symbol __CxxFrameHandler4
diabloui.cpp.obj : error LNK2001: unresolved external symbol __CxxFrameHandler4
language.cpp.obj : error LNK2001: unresolved external symbol __CxxFrameHandler4
logged_fstream.cpp.obj : error LNK2001: unresolved external symbol __CxxFrameHandler4
paths.cpp.obj : error LNK2001: unresolved external symbol __CxxFrameHandler4
sdl_bilinear_scale.cpp.obj : error LNK2001: unresolved external symbol __CxxFrameHandler4
itemlabels.cpp.obj : error LNK2001: unresolved external symbol __CxxFrameHandler4
console.cpp.obj : error LNK2001: unresolved external symbol __CxxFrameHandler4
display.cpp.obj : error LNK2001: unresolved external symbol __CxxFrameHandler4
file_util.cpp.obj : error LNK2001: unresolved external symbol __CxxFrameHandler4
chatlog.cpp.obj : error LNK2001: unresolved external symbol __CxxFrameHandler4
monhealthbar.cpp.obj : error LNK2001: unresolved external symbol __CxxFrameHandler4
stash.cpp.obj : error LNK2001: unresolved external symbol __CxxFrameHandler4
xpbar.cpp.obj : error LNK2001: unresolved external symbol __CxxFrameHandler4
mpq_reader.cpp.obj : error LNK2001: unresolved external symbol __CxxFrameHandler4
mpq_sdl_rwops.cpp.obj : error LNK2001: unresolved external symbol __CxxFrameHandler4
mpq_writer.cpp.obj : error LNK2001: unresolved external symbol __CxxFrameHandler4
autopickup.cpp.obj : error LNK2001: unresolved external symbol __CxxFrameHandler4
animationinfo.cpp.obj : error LNK2001: unresolved external symbol __CxxFrameHandler4
demomode.cpp.obj : error LNK2001: unresolved external symbol __CxxFrameHandler4
load_cel.cpp.obj : error LNK2001: unresolved external symbol __CxxFrameHandler4
text_render.cpp.obj : error LNK2001: unresolved external symbol __CxxFrameHandler4
joystick.cpp.obj : error LNK2001: unresolved external symbol __CxxFrameHandler4
menu_controls.cpp.obj : error LNK2001: unresolved external symbol __CxxFrameHandler4
modifier_hints.cpp.obj : error LNK2001: unresolved external symbol __CxxFrameHandler4
plrctrls.cpp.obj : error LNK2001: unresolved external symbol __CxxFrameHandler4
town.cpp.obj : error LNK2001: unresolved external symbol __CxxFrameHandler4
towners.cpp.obj : error LNK2001: unresolved external symbol __CxxFrameHandler4
trigs.cpp.obj : error LNK2001: unresolved external symbol __CxxFrameHandler4
controller_motion.cpp.obj : error LNK2001: unresolved external symbol __CxxFrameHandler4
scrollrt.cpp.obj : error LNK2001: unresolved external symbol __CxxFrameHandler4
setmaps.cpp.obj : error LNK2001: unresolved external symbol __CxxFrameHandler4
stores.cpp.obj : error LNK2001: unresolved external symbol __CxxFrameHandler4
tmsg.cpp.obj : error LNK2001: unresolved external symbol __CxxFrameHandler4
plrmsg.cpp.obj : error LNK2001: unresolved external symbol __CxxFrameHandler4
portal.cpp.obj : error LNK2001: unresolved external symbol __CxxFrameHandler4
quests.cpp.obj : error LNK2001: unresolved external symbol __CxxFrameHandler4
restrict.cpp.obj : error LNK2001: unresolved external symbol __CxxFrameHandler4
options.cpp.obj : error LNK2001: unresolved external symbol __CxxFrameHandler4
palette.cpp.obj : error LNK2001: unresolved external symbol __CxxFrameHandler4
pfile.cpp.obj : error LNK2001: unresolved external symbol __CxxFrameHandler4
player.cpp.obj : error LNK2001: unresolved external symbol __CxxFrameHandler4
msg.cpp.obj : error LNK2001: unresolved external symbol __CxxFrameHandler4
multi.cpp.obj : error LNK2001: unresolved external symbol __CxxFrameHandler4
nthread.cpp.obj : error LNK2001: unresolved external symbol __CxxFrameHandler4
objects.cpp.obj : error LNK2001: unresolved external symbol __CxxFrameHandler4
minitext.cpp.obj : error LNK2001: unresolved external symbol __CxxFrameHandler4
misdat.cpp.obj : error LNK2001: unresolved external symbol __CxxFrameHandler4
missiles.cpp.obj : error LNK2001: unresolved external symbol __CxxFrameHandler4
monster.cpp.obj : error LNK2001: unresolved external symbol __CxxFrameHandler4
inv.cpp.obj : error LNK2001: unresolved external symbol __CxxFrameHandler4
items.cpp.obj : error LNK2001: unresolved external symbol __CxxFrameHandler4
lighting.cpp.obj : error LNK2001: unresolved external symbol __CxxFrameHandler4
loadsave.cpp.obj : error LNK2001: unresolved external symbol __CxxFrameHandler4
help.cpp.obj : error LNK2001: unresolved external symbol __CxxFrameHandler4
hwcursor.cpp.obj : error LNK2001: unresolved external symbol __CxxFrameHandler4
init.cpp.obj : error LNK2001: unresolved external symbol __CxxFrameHandler4
interfac.cpp.obj : error LNK2001: unresolved external symbol __CxxFrameHandler4
error.cpp.obj : error LNK2001: unresolved external symbol __CxxFrameHandler4
assets.cpp.obj : error LNK2001: unresolved external symbol __CxxFrameHandler4
gendung.cpp.obj : error LNK2001: unresolved external symbol __CxxFrameHandler4
gmenu.cpp.obj : error LNK2001: unresolved external symbol __CxxFrameHandler4
drlg_l4.cpp.obj : error LNK2001: unresolved external symbol __CxxFrameHandler4
dthread.cpp.obj : error LNK2001: unresolved external symbol __CxxFrameHandler4
dx.cpp.obj : error LNK2001: unresolved external symbol __CxxFrameHandler4
encrypt.cpp.obj : error LNK2001: unresolved external symbol __CxxFrameHandler4
doom.cpp.obj : error LNK2001: unresolved external symbol __CxxFrameHandler4
drlg_l1.cpp.obj : error LNK2001: unresolved external symbol __CxxFrameHandler4
drlg_l2.cpp.obj : error LNK2001: unresolved external symbol __CxxFrameHandler4
drlg_l3.cpp.obj : error LNK2001: unresolved external symbol __CxxFrameHandler4
control.cpp.obj : error LNK2001: unresolved external symbol __CxxFrameHandler4
cursor.cpp.obj : error LNK2001: unresolved external symbol __CxxFrameHandler4
debug.cpp.obj : error LNK2001: unresolved external symbol __CxxFrameHandler4
diablo.cpp.obj : error LNK2001: unresolved external symbol __CxxFrameHandler4
appfat.cpp.obj : error LNK2001: unresolved external symbol __CxxFrameHandler4
automap.cpp.obj : error LNK2001: unresolved external symbol __CxxFrameHandler4
capture.cpp.obj : error LNK2001: unresolved external symbol __CxxFrameHandler4
codec.cpp.obj : error LNK2001: unresolved external symbol __CxxFrameHandler4
zt.lib(portlistingparse.c.obj) : error LNK2001: unresolved external symbol memcpy
zt.lib(Capability.cpp.obj) : error LNK2001: unresolved external symbol memcpy
zt.lib(Tag.cpp.obj) : error LNK2001: unresolved external symbol memcpy
zt.lib(Revocation.cpp.obj) : error LNK2001: unresolved external symbol memcpy
zt.lib(minissdpc.c.obj) : error LNK2001: unresolved external symbol memcpy
zt.lib(miniwget.c.obj) : error LNK2001: unresolved external symbol memcpy
zt.lib(minisoap.c.obj) : error LNK2001: unresolved external symbol memcpy
zt.lib(upnpreplyparse.c.obj) : error LNK2001: unresolved external symbol memcpy
zt.lib(CertificateOfOwnership.cpp.obj) : error LNK2001: unresolved external symbol memcpy
zt.lib(Membership.cpp.obj) : error LNK2001: unresolved external symbol memcpy
zt.lib(IncomingPacket.cpp.obj) : error LNK2001: unresolved external symbol memcpy
zt.lib(igd_desc_parse.c.obj) : error LNK2001: unresolved external symbol memcpy
zt.lib(miniupnpc.c.obj) : error LNK2001: unresolved external symbol memcpy
zt.lib(ethip6.c.obj) : error LNK2001: unresolved external symbol memcpy
zt.lib(AES.cpp.obj) : error LNK2001: unresolved external symbol memcpy
zt.lib(OutboundMulticast.cpp.obj) : error LNK2001: unresolved external symbol memcpy
zt.lib(Peer.cpp.obj) : error LNK2001: unresolved external symbol memcpy
zt.lib(Topology.cpp.obj) : error LNK2001: unresolved external symbol memcpy
zt.lib(Switch.cpp.obj) : error LNK2001: unresolved external symbol memcpy
zt.lib(SelfAwareness.cpp.obj) : error LNK2001: unresolved external symbol memcpy
zt.lib(Multicaster.cpp.obj) : error LNK2001: unresolved external symbol memcpy
zt.lib(NetworkConfig.cpp.obj) : error LNK2001: unresolved external symbol memcpy
zt.lib(Network.cpp.obj) : error LNK2001: unresolved external symbol memcpy
zt.lib(Bond.cpp.obj) : error LNK2001: unresolved external symbol memcpy
zt.lib(Node.cpp.obj) : error LNK2001: unresolved external symbol memcpy
zt.lib(PortMapper.cpp.obj) : error LNK2001: unresolved external symbol memcpy
zt.lib(VirtualTap.cpp.obj) : error LNK2001: unresolved external symbol memcpy
zt.lib(Packet.cpp.obj) : error LNK2001: unresolved external symbol memcpy
zt.lib(OSUtils.cpp.obj) : error LNK2001: unresolved external symbol memcpy
zt.lib(NodeService.cpp.obj) : error LNK2001: unresolved external symbol memcpy
zt.lib(Controls.cpp.obj) : error LNK2001: unresolved external symbol memcpy
zt.lib(SHA512.cpp.obj) : error LNK2001: unresolved external symbol memcpy
zt.lib(netdb.c.obj) : error LNK2001: unresolved external symbol memcpy
zt.lib(Events.cpp.obj) : error LNK2001: unresolved external symbol memcpy
zt.lib(C25519.cpp.obj) : error LNK2001: unresolved external symbol memcpy
zt.lib(InetAddress.cpp.obj) : error LNK2001: unresolved external symbol memcpy
zt.lib(etharp.c.obj) : error LNK2001: unresolved external symbol memcpy
zt.lib(tcp_out.c.obj) : error LNK2001: unresolved external symbol memcpy
zt.lib(Utilities.cpp.obj) : error LNK2001: unresolved external symbol memcpy
zt.lib(dns.c.obj) : error LNK2001: unresolved external symbol memcpy
zt.lib(icmp.c.obj) : error LNK2001: unresolved external symbol memcpy
zt.lib(ip6_frag.c.obj) : error LNK2001: unresolved external symbol memcpy
zt.lib(icmp6.c.obj) : error LNK2001: unresolved external symbol memcpy
zt.lib(nd6.c.obj) : error LNK2001: unresolved external symbol memcpy
zt.lib(pbuf.c.obj) : error LNK2001: unresolved external symbol memcpy
zt.lib(tcp.c.obj) : error LNK2001: unresolved external symbol memcpy
zt.lib(ethernet.c.obj) : error LNK2001: unresolved external symbol memcpy
zt.lib(ip4_frag.c.obj) : error LNK2001: unresolved external symbol memcpy
SDL_audiolib.lib(dr_mp3.c.obj) : error LNK2001: unresolved external symbol memcpy
SDL_audiolib.lib(dr_wav.c.obj) : error LNK2001: unresolved external symbol memcpy
zt.lib(sockets.c.obj) : error LNK2001: unresolved external symbol memcpy
zt.lib(ip6_addr.c.obj) : error LNK2001: unresolved external symbol memcpy
sodium.lib(ed25519_ref10.c.obj) : error LNK2001: unresolved external symbol memcpy
SDL_audiolib.lib(Resampler.cpp.obj) : error LNK2001: unresolved external symbol memcpy
SDL_audiolib.lib(stream_p.cpp.obj) : error LNK2001: unresolved external symbol memcpy
SDL_audiolib.lib(sampleconv.cpp.obj) : error LNK2001: unresolved external symbol memcpy
sodium.lib(argon2-fill-block-ssse3.c.obj) : error LNK2001: unresolved external symbol memcpy
sodium.lib(blake2b-long.c.obj) : error LNK2001: unresolved external symbol memcpy
sodium.lib(argon2.c.obj) : error LNK2001: unresolved external symbol memcpy
sodium.lib(argon2-encoding.c.obj) : error LNK2001: unresolved external symbol memcpy
sodium.lib(argon2-core.c.obj) : error LNK2001: unresolved external symbol memcpy
sodium.lib(blake2b-ref.c.obj) : error LNK2001: unresolved external symbol memcpy
sodium.lib(argon2-fill-block-ref.c.obj) : error LNK2001: unresolved external symbol memcpy
sodium.lib(argon2-fill-block-avx2.c.obj) : error LNK2001: unresolved external symbol memcpy
ZLIB.lib(trees.c.obj) : error LNK2001: unresolved external symbol memcpy
libsmackerdec.lib(SmackerDecoder.cpp.obj) : error LNK2001: unresolved external symbol memcpy
libsmackerdec.lib(LogError.cpp.obj) : error LNK2001: unresolved external symbol memcpy
sodium.lib(utils.c.obj) : error LNK2001: unresolved external symbol memcpy
libmpq.lib(extract.c.obj) : error LNK2001: unresolved external symbol memcpy
libmpq.lib(explode.c.obj) : error LNK2001: unresolved external symbol memcpy
ZLIB.lib(inflate.c.obj) : error LNK2001: unresolved external symbol memcpy
ZLIB.lib(deflate.c.obj) : error LNK2001: unresolved external symbol memcpy
PKWare.lib(implode.cpp.obj) : error LNK2001: unresolved external symbol memcpy
PKWare.lib(explode.cpp.obj) : error LNK2001: unresolved external symbol memcpy
libmpq.lib(mpq.c.obj) : error LNK2001: unresolved external symbol memcpy
libmpq.lib(common.c.obj) : error LNK2001: unresolved external symbol memcpy
libpng16_staticd.lib(pngmem.c.obj) : error LNK2001: unresolved external symbol memcpy
libpng16_staticd.lib(pngwutil.c.obj) : error LNK2001: unresolved external symbol memcpy
libpng16_staticd.lib(pngrutil.c.obj) : error LNK2001: unresolved external symbol memcpy
fmtd.lib(format.cc.obj) : error LNK2001: unresolved external symbol memcpy
libpng16_staticd.lib(pngread.c.obj) : error LNK2001: unresolved external symbol memcpy
libpng16_staticd.lib(pngwrite.c.obj) : error LNK2001: unresolved external symbol memcpy
libpng16_staticd.lib(pngerror.c.obj) : error LNK2001: unresolved external symbol memcpy
libpng16_staticd.lib(pngset.c.obj) : error LNK2001: unresolved external symbol memcpy
tcp_server.cpp.obj : error LNK2001: unresolved external symbol memcpy
protocol_zt.cpp.obj : error LNK2001: unresolved external symbol memcpy
zerotier_native.cpp.obj : error LNK2001: unresolved external symbol memcpy
zerotier_lwip.cpp.obj : error LNK2001: unresolved external symbol memcpy
sound.cpp.obj : error LNK2001: unresolved external symbol memcpy
push_aulib_decoder.cpp.obj : error LNK2001: unresolved external symbol memcpy
soundsample.cpp.obj : error LNK2001: unresolved external symbol memcpy
tcp_client.cpp.obj : error LNK2001: unresolved external symbol memcpy
loopback.cpp.obj : error LNK2001: unresolved external symbol memcpy
packet.cpp.obj : error LNK2001: unresolved external symbol memcpy
storm_net.cpp.obj : error LNK2001: unresolved external symbol memcpy
locale.cpp.obj : error LNK2001: unresolved external symbol memcpy
spell_list.cpp.obj : error LNK2001: unresolved external symbol memcpy
abstract_net.cpp.obj : error LNK2001: unresolved external symbol memcpy
base.cpp.obj : error LNK2001: unresolved external symbol memcpy
frame_queue.cpp.obj : error LNK2001: unresolved external symbol memcpy
settingsmenu.cpp.obj : error LNK2001: unresolved external symbol memcpy
selgame.cpp.obj : error LNK2001: unresolved external symbol memcpy
selhero.cpp.obj : error LNK2001: unresolved external symbol memcpy
charpanel.cpp.obj : error LNK2001: unresolved external symbol memcpy
utf8.cpp.obj : error LNK2001: unresolved external symbol memcpy
credits.cpp.obj : error LNK2001: unresolved external symbol memcpy
diabloui.cpp.obj : error LNK2001: unresolved external symbol memcpy
dialogs.cpp.obj : error LNK2001: unresolved external symbol memcpy
file_util.cpp.obj : error LNK2001: unresolved external symbol memcpy
language.cpp.obj : error LNK2001: unresolved external symbol memcpy
logged_fstream.cpp.obj : error LNK2001: unresolved external symbol memcpy
paths.cpp.obj : error LNK2001: unresolved external symbol memcpy
xpbar.cpp.obj : error LNK2001: unresolved external symbol memcpy
itemlabels.cpp.obj : error LNK2001: unresolved external symbol memcpy
console.cpp.obj : error LNK2001: unresolved external symbol memcpy
display.cpp.obj : error LNK2001: unresolved external symbol memcpy
mpq_sdl_rwops.cpp.obj : error LNK2001: unresolved external symbol memcpy
mpq_writer.cpp.obj : error LNK2001: unresolved external symbol memcpy
chatlog.cpp.obj : error LNK2001: unresolved external symbol memcpy
stash.cpp.obj : error LNK2001: unresolved external symbol memcpy
dun_render.cpp.obj : error LNK2001: unresolved external symbol memcpy
text_render.cpp.obj : error LNK2001: unresolved external symbol memcpy
surface.cpp.obj : error LNK2001: unresolved external symbol memcpy
mpq_reader.cpp.obj : error LNK2001: unresolved external symbol memcpy
plrctrls.cpp.obj : error LNK2001: unresolved external symbol memcpy
demomode.cpp.obj : error LNK2001: unresolved external symbol memcpy
cel_render.cpp.obj : error LNK2001: unresolved external symbol memcpy
cl2_render.cpp.obj : error LNK2001: unresolved external symbol memcpy
stores.cpp.obj : error LNK2001: unresolved external symbol memcpy
tmsg.cpp.obj : error LNK2001: unresolved external symbol memcpy
towners.cpp.obj : error LNK2001: unresolved external symbol memcpy
trigs.cpp.obj : error LNK2001: unresolved external symbol memcpy
quests.cpp.obj : error LNK2001: unresolved external symbol memcpy
restrict.cpp.obj : error LNK2001: unresolved external symbol memcpy
scrollrt.cpp.obj : error LNK2001: unresolved external symbol memcpy
sha.cpp.obj : error LNK2001: unresolved external symbol memcpy
options.cpp.obj : error LNK2001: unresolved external symbol memcpy
palette.cpp.obj : error LNK2001: unresolved external symbol memcpy
pfile.cpp.obj : error LNK2001: unresolved external symbol memcpy
plrmsg.cpp.obj : error LNK2001: unresolved external symbol memcpy
monster.cpp.obj : error LNK2001: unresolved external symbol memcpy
msg.cpp.obj : error LNK2001: unresolved external symbol memcpy
multi.cpp.obj : error LNK2001: unresolved external symbol memcpy
objects.cpp.obj : error LNK2001: unresolved external symbol memcpy
lighting.cpp.obj : error LNK2001: unresolved external symbol memcpy
loadsave.cpp.obj : error LNK2001: unresolved external symbol memcpy
minitext.cpp.obj : error LNK2001: unresolved external symbol memcpy
misdat.cpp.obj : error LNK2001: unresolved external symbol memcpy
help.cpp.obj : error LNK2001: unresolved external symbol memcpy
init.cpp.obj : error LNK2001: unresolved external symbol memcpy
inv.cpp.obj : error LNK2001: unresolved external symbol memcpy
items.cpp.obj : error LNK2001: unresolved external symbol memcpy
drlg_l3.cpp.obj : error LNK2001: unresolved external symbol memcpy
encrypt.cpp.obj : error LNK2001: unresolved external symbol memcpy
error.cpp.obj : error LNK2001: unresolved external symbol memcpy
assets.cpp.obj : error LNK2001: unresolved external symbol memcpy
cursor.cpp.obj : error LNK2001: unresolved external symbol memcpy
dead.cpp.obj : error LNK2001: unresolved external symbol memcpy
debug.cpp.obj : error LNK2001: unresolved external symbol memcpy
diablo.cpp.obj : error LNK2001: unresolved external symbol memcpy
automap.cpp.obj : error LNK2001: unresolved external symbol memcpy
capture.cpp.obj : error LNK2001: unresolved external symbol memcpy
codec.cpp.obj : error LNK2001: unresolved external symbol memcpy
control.cpp.obj : error LNK2001: unresolved external symbol memcpy
zt.lib(Peer.cpp.obj) : error LNK2001: unresolved external symbol memmove
zt.lib(Switch.cpp.obj) : error LNK2001: unresolved external symbol memmove
zt.lib(miniupnpc.c.obj) : error LNK2001: unresolved external symbol memmove
zt.lib(IncomingPacket.cpp.obj) : error LNK2001: unresolved external symbol memmove
zt.lib(VirtualTap.cpp.obj) : error LNK2001: unresolved external symbol memmove
zt.lib(Packet.cpp.obj) : error LNK2001: unresolved external symbol memmove
zt.lib(Network.cpp.obj) : error LNK2001: unresolved external symbol memmove
zt.lib(Bond.cpp.obj) : error LNK2001: unresolved external symbol memmove
zt.lib(OSUtils.cpp.obj) : error LNK2001: unresolved external symbol memmove
zt.lib(NodeService.cpp.obj) : error LNK2001: unresolved external symbol memmove
zt.lib(Controls.cpp.obj) : error LNK2001: unresolved external symbol memmove
zt.lib(PortMapper.cpp.obj) : error LNK2001: unresolved external symbol memmove
SDL_audiolib.lib(dr_mp3.c.obj) : error LNK2001: unresolved external symbol memmove
zt.lib(def.c.obj) : error LNK2001: unresolved external symbol memmove
zt.lib(ip6_frag.c.obj) : error LNK2001: unresolved external symbol memmove
zt.lib(Utilities.cpp.obj) : error LNK2001: unresolved external symbol memmove
sodium.lib(crypto_secretbox_easy.c.obj) : error LNK2001: unresolved external symbol memmove
SDL_audiolib.lib(Stream.cpp.obj) : error LNK2001: unresolved external symbol memmove
SDL_audiolib.lib(Resampler.cpp.obj) : error LNK2001: unresolved external symbol memmove
SDL_audiolib.lib(stream_p.cpp.obj) : error LNK2001: unresolved external symbol memmove
PKWare.lib(implode.cpp.obj) : error LNK2001: unresolved external symbol memmove
PKWare.lib(explode.cpp.obj) : error LNK2001: unresolved external symbol memmove
libsmackerdec.lib(SmackerDecoder.cpp.obj) : error LNK2001: unresolved external symbol memmove
libsmackerdec.lib(LogError.cpp.obj) : error LNK2001: unresolved external symbol memmove
tcp_server.cpp.obj : error LNK2001: unresolved external symbol memmove
protocol_zt.cpp.obj : error LNK2001: unresolved external symbol memmove
game_controller.cpp.obj : error LNK2001: unresolved external symbol memmove
fmtd.lib(format.cc.obj) : error LNK2001: unresolved external symbol memmove
locale.cpp.obj : error LNK2001: unresolved external symbol memmove
sound.cpp.obj : error LNK2001: unresolved external symbol memmove
push_aulib_decoder.cpp.obj : error LNK2001: unresolved external symbol memmove
tcp_client.cpp.obj : error LNK2001: unresolved external symbol memmove
loopback.cpp.obj : error LNK2001: unresolved external symbol memmove
packet.cpp.obj : error LNK2001: unresolved external symbol memmove
storm_net.cpp.obj : error LNK2001: unresolved external symbol memmove
misc_msg.cpp.obj : error LNK2001: unresolved external symbol memmove
spell_list.cpp.obj : error LNK2001: unresolved external symbol memmove
abstract_net.cpp.obj : error LNK2001: unresolved external symbol memmove
base.cpp.obj : error LNK2001: unresolved external symbol memmove
frame_queue.cpp.obj : error LNK2001: unresolved external symbol memmove
selok.cpp.obj : error LNK2001: unresolved external symbol memmove
selstart.cpp.obj : error LNK2001: unresolved external symbol memmove
selyesno.cpp.obj : error LNK2001: unresolved external symbol memmove
charpanel.cpp.obj : error LNK2001: unresolved external symbol memmove
settingsmenu.cpp.obj : error LNK2001: unresolved external symbol memmove
selconn.cpp.obj : error LNK2001: unresolved external symbol memmove
selgame.cpp.obj : error LNK2001: unresolved external symbol memmove
selhero.cpp.obj : error LNK2001: unresolved external symbol memmove
paths.cpp.obj : error LNK2001: unresolved external symbol memmove
credits.cpp.obj : error LNK2001: unresolved external symbol memmove
diabloui.cpp.obj : error LNK2001: unresolved external symbol memmove
mainmenu.cpp.obj : error LNK2001: unresolved external symbol memmove
display.cpp.obj : error LNK2001: unresolved external symbol memmove
file_util.cpp.obj : error LNK2001: unresolved external symbol memmove
language.cpp.obj : error LNK2001: unresolved external symbol memmove
logged_fstream.cpp.obj : error LNK2001: unresolved external symbol memmove
stash.cpp.obj : error LNK2001: unresolved external symbol memmove
xpbar.cpp.obj : error LNK2001: unresolved external symbol memmove
itemlabels.cpp.obj : error LNK2001: unresolved external symbol memmove
console.cpp.obj : error LNK2001: unresolved external symbol memmove
text_render.cpp.obj : error LNK2001: unresolved external symbol memmove
mpq_reader.cpp.obj : error LNK2001: unresolved external symbol memmove
mpq_writer.cpp.obj : error LNK2001: unresolved external symbol memmove
chatlog.cpp.obj : error LNK2001: unresolved external symbol memmove
trigs.cpp.obj : error LNK2001: unresolved external symbol memmove
joystick.cpp.obj : error LNK2001: unresolved external symbol memmove
plrctrls.cpp.obj : error LNK2001: unresolved external symbol memmove
demomode.cpp.obj : error LNK2001: unresolved external symbol memmove
plrmsg.cpp.obj : error LNK2001: unresolved external symbol memmove
quests.cpp.obj : error LNK2001: unresolved external symbol memmove
stores.cpp.obj : error LNK2001: unresolved external symbol memmove
towners.cpp.obj : error LNK2001: unresolved external symbol memmove
monster.cpp.obj : error LNK2001: unresolved external symbol memmove
objects.cpp.obj : error LNK2001: unresolved external symbol memmove
options.cpp.obj : error LNK2001: unresolved external symbol memmove
pfile.cpp.obj : error LNK2001: unresolved external symbol memmove
items.cpp.obj : error LNK2001: unresolved external symbol memmove
loadsave.cpp.obj : error LNK2001: unresolved external symbol memmove
minitext.cpp.obj : error LNK2001: unresolved external symbol memmove
misdat.cpp.obj : error LNK2001: unresolved external symbol memmove
gendung.cpp.obj : error LNK2001: unresolved external symbol memmove
help.cpp.obj : error LNK2001: unresolved external symbol memmove
init.cpp.obj : error LNK2001: unresolved external symbol memmove
inv.cpp.obj : error LNK2001: unresolved external symbol memmove
debug.cpp.obj : error LNK2001: unresolved external symbol memmove
diablo.cpp.obj : error LNK2001: unresolved external symbol memmove
error.cpp.obj : error LNK2001: unresolved external symbol memmove
assets.cpp.obj : error LNK2001: unresolved external symbol memmove
automap.cpp.obj : error LNK2001: unresolved external symbol memmove
capture.cpp.obj : error LNK2001: unresolved external symbol memmove
control.cpp.obj : error LNK2001: unresolved external symbol memmove
cursor.cpp.obj : error LNK2001: unresolved external symbol memmove
zt.lib(portlistingparse.c.obj) : error LNK2001: unresolved external symbol memset
zt.lib(getgateway.c.obj) : error LNK2001: unresolved external symbol memset
zt.lib(receivedata.c.obj) : error LNK2001: unresolved external symbol memset
zt.lib(minissdpc.c.obj) : error LNK2001: unresolved external symbol memset
zt.lib(miniwget.c.obj) : error LNK2001: unresolved external symbol memset
zt.lib(connecthostport.c.obj) : error LNK2001: unresolved external symbol memset
zt.lib(upnpreplyparse.c.obj) : error LNK2001: unresolved external symbol memset
zt.lib(natpmp.c.obj) : error LNK2001: unresolved external symbol memset
zt.lib(CertificateOfMembership.cpp.obj) : error LNK2001: unresolved external symbol memset
zt.lib(Membership.cpp.obj) : error LNK2001: unresolved external symbol memset
zt.lib(IncomingPacket.cpp.obj) : error LNK2001: unresolved external symbol memset
zt.lib(Peer.cpp.obj) : error LNK2001: unresolved external symbol memset
zt.lib(Topology.cpp.obj) : error LNK2001: unresolved external symbol memset
zt.lib(SelfAwareness.cpp.obj) : error LNK2001: unresolved external symbol memset
zt.lib(miniupnpc.c.obj) : error LNK2001: unresolved external symbol memset
zt.lib(Trace.cpp.obj) : error LNK2001: unresolved external symbol memset
zt.lib(NetworkConfig.cpp.obj) : error LNK2001: unresolved external symbol memset
zt.lib(Network.cpp.obj) : error LNK2001: unresolved external symbol memset
zt.lib(Bond.cpp.obj) : error LNK2001: unresolved external symbol memset
zt.lib(Node.cpp.obj) : error LNK2001: unresolved external symbol memset
zt.lib(PortMapper.cpp.obj) : error LNK2001: unresolved external symbol memset
zt.lib(VirtualTap.cpp.obj) : error LNK2001: unresolved external symbol memset
zt.lib(Packet.cpp.obj) : error LNK2001: unresolved external symbol memset
zt.lib(InetAddress.cpp.obj) : error LNK2001: unresolved external symbol memset
zt.lib(Identity.cpp.obj) : error LNK2001: unresolved external symbol memset
zt.lib(OSUtils.cpp.obj) : error LNK2001: unresolved external symbol memset
zt.lib(NodeService.cpp.obj) : error LNK2001: unresolved external symbol memset
zt.lib(Utilities.cpp.obj) : error LNK2001: unresolved external symbol memset
zt.lib(dns.c.obj) : error LNK2001: unresolved external symbol memset
zt.lib(netdb.c.obj) : error LNK2001: unresolved external symbol memset
zt.lib(C25519.cpp.obj) : error LNK2001: unresolved external symbol memset
zt.lib(Sockets.cpp.obj) : error LNK2001: unresolved external symbol memset
zt.lib(ip4_frag.c.obj) : error LNK2001: unresolved external symbol memset
zt.lib(tcp_in.c.obj) : error LNK2001: unresolved external symbol memset
zt.lib(ip6_frag.c.obj) : error LNK2001: unresolved external symbol memset
zt.lib(netif.c.obj) : error LNK2001: unresolved external symbol memset
zt.lib(netbuf.c.obj) : error LNK2001: unresolved external symbol memset
zt.lib(tcp.c.obj) : error LNK2001: unresolved external symbol memset
zt.lib(udp.c.obj) : error LNK2001: unresolved external symbol memset
SDL_audiolib.lib(dr_mp3.c.obj) : error LNK2001: unresolved external symbol memset
SDL_audiolib.lib(dr_wav.c.obj) : error LNK2001: unresolved external symbol memset
zt.lib(sockets.c.obj) : error LNK2001: unresolved external symbol memset
zt.lib(sys_arch.c.obj) : error LNK2001: unresolved external symbol memset
sodium.lib(salsa20_xmm6int-avx2.c.obj) : error LNK2001: unresolved external symbol memset
sodium.lib(argon2.c.obj) : error LNK2001: unresolved external symbol memset
sodium.lib(ed25519_ref10.c.obj) : error LNK2001: unresolved external symbol memset
SDL_audiolib.lib(stream_p.cpp.obj) : error LNK2001: unresolved external symbol memset
sodium.lib(chacha20_ref.c.obj) : error LNK2001: unresolved external symbol memset
sodium.lib(chacha20_dolbeau-avx2.c.obj) : error LNK2001: unresolved external symbol memset
sodium.lib(chacha20_dolbeau-ssse3.c.obj) : error LNK2001: unresolved external symbol memset
sodium.lib(salsa20_xmm6int-sse2.c.obj) : error LNK2001: unresolved external symbol memset
sodium.lib(argon2-fill-block-ref.c.obj) : error LNK2001: unresolved external symbol memset
sodium.lib(argon2-fill-block-avx2.c.obj) : error LNK2001: unresolved external symbol memset
sodium.lib(argon2-fill-block-ssse3.c.obj) : error LNK2001: unresolved external symbol memset
sodium.lib(x25519_ref10.c.obj) : error LNK2001: unresolved external symbol memset
sodium.lib(utils.c.obj) : error LNK2001: unresolved external symbol memset
sodium.lib(pwhash_argon2i.c.obj) : error LNK2001: unresolved external symbol memset
sodium.lib(pwhash_argon2id.c.obj) : error LNK2001: unresolved external symbol memset
sodium.lib(blake2b-ref.c.obj) : error LNK2001: unresolved external symbol memset
libmpq.lib(huffman.c.obj) : error LNK2001: unresolved external symbol memset
ZLIB.lib(deflate.c.obj) : error LNK2001: unresolved external symbol memset
libsmackerdec.lib(SmackerDecoder.cpp.obj) : error LNK2001: unresolved external symbol memset
sodium.lib(crypto_secretbox_easy.c.obj) : error LNK2001: unresolved external symbol memset
fmtd.lib(format.cc.obj) : error LNK2001: unresolved external symbol memset
PKWare.lib(implode.cpp.obj) : error LNK2001: unresolved external symbol memset
libmpq.lib(extract.c.obj) : error LNK2001: unresolved external symbol memset
libmpq.lib(explode.c.obj) : error LNK2001: unresolved external symbol memset
libpng16_staticd.lib(pngset.c.obj) : error LNK2001: unresolved external symbol memset
libpng16_staticd.lib(pngmem.c.obj) : error LNK2001: unresolved external symbol memset
libpng16_staticd.lib(pngwutil.c.obj) : error LNK2001: unresolved external symbol memset
libpng16_staticd.lib(pngrutil.c.obj) : error LNK2001: unresolved external symbol memset
libpng16_staticd.lib(png.c.obj) : error LNK2001: unresolved external symbol memset
libpng16_staticd.lib(pngread.c.obj) : error LNK2001: unresolved external symbol memset
libpng16_staticd.lib(pngwrite.c.obj) : error LNK2001: unresolved external symbol memset
libpng16_staticd.lib(pngrtran.c.obj) : error LNK2001: unresolved external symbol memset
push_aulib_decoder.cpp.obj : error LNK2001: unresolved external symbol memset
tcp_client.cpp.obj : error LNK2001: unresolved external symbol memset
tcp_server.cpp.obj : error LNK2001: unresolved external symbol memset
protocol_zt.cpp.obj : error LNK2001: unresolved external symbol memset
frame_queue.cpp.obj : error LNK2001: unresolved external symbol memset
loopback.cpp.obj : error LNK2001: unresolved external symbol memset
packet.cpp.obj : error LNK2001: unresolved external symbol memset
misc_msg.cpp.obj : error LNK2001: unresolved external symbol memset
selgame.cpp.obj : error LNK2001: unresolved external symbol memset
selhero.cpp.obj : error LNK2001: unresolved external symbol memset
abstract_net.cpp.obj : error LNK2001: unresolved external symbol memset
base.cpp.obj : error LNK2001: unresolved external symbol memset
mpq_writer.cpp.obj : error LNK2001: unresolved external symbol memset
language.cpp.obj : error LNK2001: unresolved external symbol memset
logged_fstream.cpp.obj : error LNK2001: unresolved external symbol memset
art.cpp.obj : error LNK2001: unresolved external symbol memset
dun_render.cpp.obj : error LNK2001: unresolved external symbol memset
text_render.cpp.obj : error LNK2001: unresolved external symbol memset
mpq_reader.cpp.obj : error LNK2001: unresolved external symbol memset
mpq_sdl_rwops.cpp.obj : error LNK2001: unresolved external symbol memset
town.cpp.obj : error LNK2001: unresolved external symbol memset
demomode.cpp.obj : error LNK2001: unresolved external symbol memset
cel_render.cpp.obj : error LNK2001: unresolved external symbol memset
cl2_render.cpp.obj : error LNK2001: unresolved external symbol memset
pfile.cpp.obj : error LNK2001: unresolved external symbol memset
player.cpp.obj : error LNK2001: unresolved external symbol memset
scrollrt.cpp.obj : error LNK2001: unresolved external symbol memset
sync.cpp.obj : error LNK2001: unresolved external symbol memset
objects.cpp.obj : error LNK2001: unresolved external symbol memset
options.cpp.obj : error LNK2001: unresolved external symbol memset
pack.cpp.obj : error LNK2001: unresolved external symbol memset
path.cpp.obj : error LNK2001: unresolved external symbol memset
loadsave.cpp.obj : error LNK2001: unresolved external symbol memset
misdat.cpp.obj : error LNK2001: unresolved external symbol memset
msg.cpp.obj : error LNK2001: unresolved external symbol memset
multi.cpp.obj : error LNK2001: unresolved external symbol memset
gendung.cpp.obj : error LNK2001: unresolved external symbol memset
gmenu.cpp.obj : error LNK2001: unresolved external symbol memset
items.cpp.obj : error LNK2001: unresolved external symbol memset
lighting.cpp.obj : error LNK2001: unresolved external symbol memset
drlg_l3.cpp.obj : error LNK2001: unresolved external symbol memset
drlg_l4.cpp.obj : error LNK2001: unresolved external symbol memset
engine.cpp.obj : error LNK2001: unresolved external symbol memset
error.cpp.obj : error LNK2001: unresolved external symbol memset
automap.cpp.obj : error LNK2001: unresolved external symbol memset
capture.cpp.obj : error LNK2001: unresolved external symbol memset
codec.cpp.obj : error LNK2001: unresolved external symbol memset
diablo.cpp.obj : error LNK2001: unresolved external symbol memset
zt.lib(CertificateOfMembership.cpp.obj) : error LNK2019: unresolved external symbol __std_exception_copy referenced in function "public: void __cdecl std::_Tree_val<struct std::_Tree_simple_types<struct std::pair<unsigned __int64 const ,unsigned __int64> > >::_Erase_head<class std::allocator<struct std::_Tree_node<struct std::pair<unsigned __int64 const ,unsigned __int64>,void *> > >(class std::allocator<struct std::_Tree_node<struct std::pair<unsigned __int64 const ,unsigned __int64>,void *> > &)" (??$_Erase_head@V?$allocator@U?$_Tree_node@U?$pair@$$CB_K_K@std@@PEAX@std@@@std@@@?$_Tree_val@U?$_Tree_simple_types@U?$pair@$$CB_K_K@std@@@std@@@std@@QEAAXAEAV?$allocator@U?$_Tree_node@U?$pair@$$CB_K_K@std@@PEAX@std@@@1@@Z)
zt.lib(IncomingPacket.cpp.obj) : error LNK2001: unresolved external symbol __std_exception_copy
MSVCRTD.lib(throw_bad_alloc.obj) : error LNK2001: unresolved external symbol __std_exception_copy
zt.lib(Bond.cpp.obj) : error LNK2001: unresolved external symbol __std_exception_copy
zt.lib(Peer.cpp.obj) : error LNK2001: unresolved external symbol __std_exception_copy
zt.lib(Topology.cpp.obj) : error LNK2001: unresolved external symbol __std_exception_copy
zt.lib(Switch.cpp.obj) : error LNK2001: unresolved external symbol __std_exception_copy
zt.lib(VirtualTap.cpp.obj) : error LNK2001: unresolved external symbol __std_exception_copy
zt.lib(Multicaster.cpp.obj) : error LNK2001: unresolved external symbol __std_exception_copy
zt.lib(Trace.cpp.obj) : error LNK2001: unresolved external symbol __std_exception_copy
zt.lib(Network.cpp.obj) : error LNK2001: unresolved external symbol __std_exception_copy
zt.lib(NodeService.cpp.obj) : error LNK2001: unresolved external symbol __std_exception_copy
zt.lib(Controls.cpp.obj) : error LNK2001: unresolved external symbol __std_exception_copy
zt.lib(Node.cpp.obj) : error LNK2001: unresolved external symbol __std_exception_copy
zt.lib(PortMapper.cpp.obj) : error LNK2001: unresolved external symbol __std_exception_copy
SDL_audiolib.lib(Stream.cpp.obj) : error LNK2001: unresolved external symbol __std_exception_copy
SDL_audiolib.lib(stream_p.cpp.obj) : error LNK2001: unresolved external symbol __std_exception_copy
zt.lib(Utilities.cpp.obj) : error LNK2001: unresolved external symbol __std_exception_copy
zt.lib(OSUtils.cpp.obj) : error LNK2001: unresolved external symbol __std_exception_copy
game_controller.cpp.obj : error LNK2001: unresolved external symbol __std_exception_copy
fmtd.lib(format.cc.obj) : error LNK2001: unresolved external symbol __std_exception_copy
libsmackerdec.lib(SmackerDecoder.cpp.obj) : error LNK2001: unresolved external symbol __std_exception_copy
libsmackerdec.lib(LogError.cpp.obj) : error LNK2001: unresolved external symbol __std_exception_copy
tcp_client.cpp.obj : error LNK2001: unresolved external symbol __std_exception_copy
tcp_server.cpp.obj : error LNK2001: unresolved external symbol __std_exception_copy
protocol_zt.cpp.obj : error LNK2001: unresolved external symbol __std_exception_copy
zerotier_native.cpp.obj : error LNK2001: unresolved external symbol __std_exception_copy
misc_msg.cpp.obj : error LNK2001: unresolved external symbol __std_exception_copy
locale.cpp.obj : error LNK2001: unresolved external symbol __std_exception_copy
sound.cpp.obj : error LNK2001: unresolved external symbol __std_exception_copy
push_aulib_decoder.cpp.obj : error LNK2001: unresolved external symbol __std_exception_copy
loopback.cpp.obj : error LNK2001: unresolved external symbol __std_exception_copy
packet.cpp.obj : error LNK2001: unresolved external symbol __std_exception_copy
storm_net.cpp.obj : error LNK2001: unresolved external symbol __std_exception_copy
storm_svid.cpp.obj : error LNK2001: unresolved external symbol __std_exception_copy
spell_list.cpp.obj : error LNK2001: unresolved external symbol __std_exception_copy
abstract_net.cpp.obj : error LNK2001: unresolved external symbol __std_exception_copy
base.cpp.obj : error LNK2001: unresolved external symbol __std_exception_copy
frame_queue.cpp.obj : error LNK2001: unresolved external symbol __std_exception_copy
selstart.cpp.obj : error LNK2001: unresolved external symbol __std_exception_copy
selyesno.cpp.obj : error LNK2001: unresolved external symbol __std_exception_copy
title.cpp.obj : error LNK2001: unresolved external symbol __std_exception_copy
charpanel.cpp.obj : error LNK2001: unresolved external symbol __std_exception_copy
selconn.cpp.obj : error LNK2001: unresolved external symbol __std_exception_copy
selgame.cpp.obj : error LNK2001: unresolved external symbol __std_exception_copy
selhero.cpp.obj : error LNK2001: unresolved external symbol __std_exception_copy
selok.cpp.obj : error LNK2001: unresolved external symbol __std_exception_copy
dialogs.cpp.obj : error LNK2001: unresolved external symbol __std_exception_copy
mainmenu.cpp.obj : error LNK2001: unresolved external symbol __std_exception_copy
progress.cpp.obj : error LNK2001: unresolved external symbol __std_exception_copy
settingsmenu.cpp.obj : error LNK2001: unresolved external symbol __std_exception_copy
logged_fstream.cpp.obj : error LNK2001: unresolved external symbol __std_exception_copy
paths.cpp.obj : error LNK2001: unresolved external symbol __std_exception_copy
credits.cpp.obj : error LNK2001: unresolved external symbol __std_exception_copy
diabloui.cpp.obj : error LNK2001: unresolved external symbol __std_exception_copy
console.cpp.obj : error LNK2001: unresolved external symbol __std_exception_copy
display.cpp.obj : error LNK2001: unresolved external symbol __std_exception_copy
file_util.cpp.obj : error LNK2001: unresolved external symbol __std_exception_copy
language.cpp.obj : error LNK2001: unresolved external symbol __std_exception_copy
chatlog.cpp.obj : error LNK2001: unresolved external symbol __std_exception_copy
stash.cpp.obj : error LNK2001: unresolved external symbol __std_exception_copy
xpbar.cpp.obj : error LNK2001: unresolved external symbol __std_exception_copy
itemlabels.cpp.obj : error LNK2001: unresolved external symbol __std_exception_copy
mpq_reader.cpp.obj : error LNK2001: unresolved external symbol __std_exception_copy
mpq_sdl_rwops.cpp.obj : error LNK2001: unresolved external symbol __std_exception_copy
mpq_writer.cpp.obj : error LNK2001: unresolved external symbol __std_exception_copy
autopickup.cpp.obj : error LNK2001: unresolved external symbol __std_exception_copy
joystick.cpp.obj : error LNK2001: unresolved external symbol __std_exception_copy
plrctrls.cpp.obj : error LNK2001: unresolved external symbol __std_exception_copy
demomode.cpp.obj : error LNK2001: unresolved external symbol __std_exception_copy
text_render.cpp.obj : error LNK2001: unresolved external symbol __std_exception_copy
stores.cpp.obj : error LNK2001: unresolved external symbol __std_exception_copy
tmsg.cpp.obj : error LNK2001: unresolved external symbol __std_exception_copy
towners.cpp.obj : error LNK2001: unresolved external symbol __std_exception_copy
trigs.cpp.obj : error LNK2001: unresolved external symbol __std_exception_copy
player.cpp.obj : error LNK2001: unresolved external symbol __std_exception_copy
plrmsg.cpp.obj : error LNK2001: unresolved external symbol __std_exception_copy
restrict.cpp.obj : error LNK2001: unresolved external symbol __std_exception_copy
scrollrt.cpp.obj : error LNK2001: unresolved external symbol __std_exception_copy
msg.cpp.obj : error LNK2001: unresolved external symbol __std_exception_copy
objects.cpp.obj : error LNK2001: unresolved external symbol __std_exception_copy
options.cpp.obj : error LNK2001: unresolved external symbol __std_exception_copy
pfile.cpp.obj : error LNK2001: unresolved external symbol __std_exception_copy
minitext.cpp.obj : error LNK2001: unresolved external symbol __std_exception_copy
misdat.cpp.obj : error LNK2001: unresolved external symbol __std_exception_copy
missiles.cpp.obj : error LNK2001: unresolved external symbol __std_exception_copy
monster.cpp.obj : error LNK2001: unresolved external symbol __std_exception_copy
init.cpp.obj : error LNK2001: unresolved external symbol __std_exception_copy
inv.cpp.obj : error LNK2001: unresolved external symbol __std_exception_copy
items.cpp.obj : error LNK2001: unresolved external symbol __std_exception_copy
loadsave.cpp.obj : error LNK2001: unresolved external symbol __std_exception_copy
error.cpp.obj : error LNK2001: unresolved external symbol __std_exception_copy
assets.cpp.obj : error LNK2001: unresolved external symbol __std_exception_copy
gendung.cpp.obj : error LNK2001: unresolved external symbol __std_exception_copy
help.cpp.obj : error LNK2001: unresolved external symbol __std_exception_copy
debug.cpp.obj : error LNK2001: unresolved external symbol __std_exception_copy
diablo.cpp.obj : error LNK2001: unresolved external symbol __std_exception_copy
drlg_l2.cpp.obj : error LNK2001: unresolved external symbol __std_exception_copy
dthread.cpp.obj : error LNK2001: unresolved external symbol __std_exception_copy
automap.cpp.obj : error LNK2001: unresolved external symbol __std_exception_copy
capture.cpp.obj : error LNK2001: unresolved external symbol __std_exception_copy
control.cpp.obj : error LNK2001: unresolved external symbol __std_exception_copy
cursor.cpp.obj : error LNK2001: unresolved external symbol __std_exception_copy
zt.lib(CertificateOfMembership.cpp.obj) : error LNK2019: unresolved external symbol __std_exception_destroy referenced in function "void __cdecl std::_Construct_in_place<struct std::_Container_proxy,struct std::_Container_base12 *>(struct std::_Container_proxy &,struct std::_Container_base12 * &&)" (??$_Construct_in_place@U_Container_proxy@std@@PEAU_Container_base12@2@@std@@YAXAEAU_Container_proxy@0@$$QEAPEAU_Container_base12@0@@Z)
zt.lib(IncomingPacket.cpp.obj) : error LNK2001: unresolved external symbol __std_exception_destroy
MSVCRTD.lib(throw_bad_alloc.obj) : error LNK2001: unresolved external symbol __std_exception_destroy
zt.lib(Bond.cpp.obj) : error LNK2001: unresolved external symbol __std_exception_destroy
zt.lib(Peer.cpp.obj) : error LNK2001: unresolved external symbol __std_exception_destroy
zt.lib(Topology.cpp.obj) : error LNK2001: unresolved external symbol __std_exception_destroy
zt.lib(Switch.cpp.obj) : error LNK2001: unresolved external symbol __std_exception_destroy
zt.lib(VirtualTap.cpp.obj) : error LNK2001: unresolved external symbol __std_exception_destroy
zt.lib(Multicaster.cpp.obj) : error LNK2001: unresolved external symbol __std_exception_destroy
zt.lib(Trace.cpp.obj) : error LNK2001: unresolved external symbol __std_exception_destroy
zt.lib(Network.cpp.obj) : error LNK2001: unresolved external symbol __std_exception_destroy
zt.lib(NodeService.cpp.obj) : error LNK2001: unresolved external symbol __std_exception_destroy
zt.lib(Controls.cpp.obj) : error LNK2001: unresolved external symbol __std_exception_destroy
zt.lib(Node.cpp.obj) : error LNK2001: unresolved external symbol __std_exception_destroy
zt.lib(PortMapper.cpp.obj) : error LNK2001: unresolved external symbol __std_exception_destroy
SDL_audiolib.lib(Stream.cpp.obj) : error LNK2001: unresolved external symbol __std_exception_destroy
SDL_audiolib.lib(stream_p.cpp.obj) : error LNK2001: unresolved external symbol __std_exception_destroy
zt.lib(Utilities.cpp.obj) : error LNK2001: unresolved external symbol __std_exception_destroy
zt.lib(OSUtils.cpp.obj) : error LNK2001: unresolved external symbol __std_exception_destroy
game_controller.cpp.obj : error LNK2001: unresolved external symbol __std_exception_destroy
fmtd.lib(format.cc.obj) : error LNK2001: unresolved external symbol __std_exception_destroy
libsmackerdec.lib(SmackerDecoder.cpp.obj) : error LNK2001: unresolved external symbol __std_exception_destroy
libsmackerdec.lib(LogError.cpp.obj) : error LNK2001: unresolved external symbol __std_exception_destroy
tcp_client.cpp.obj : error LNK2001: unresolved external symbol __std_exception_destroy
tcp_server.cpp.obj : error LNK2001: unresolved external symbol __std_exception_destroy
protocol_zt.cpp.obj : error LNK2001: unresolved external symbol __std_exception_destroy
zerotier_native.cpp.obj : error LNK2001: unresolved external symbol __std_exception_destroy
misc_msg.cpp.obj : error LNK2001: unresolved external symbol __std_exception_destroy
locale.cpp.obj : error LNK2001: unresolved external symbol __std_exception_destroy
sound.cpp.obj : error LNK2001: unresolved external symbol __std_exception_destroy
push_aulib_decoder.cpp.obj : error LNK2001: unresolved external symbol __std_exception_destroy
loopback.cpp.obj : error LNK2001: unresolved external symbol __std_exception_destroy
packet.cpp.obj : error LNK2001: unresolved external symbol __std_exception_destroy
storm_net.cpp.obj : error LNK2001: unresolved external symbol __std_exception_destroy
storm_svid.cpp.obj : error LNK2001: unresolved external symbol __std_exception_destroy
spell_list.cpp.obj : error LNK2001: unresolved external symbol __std_exception_destroy
abstract_net.cpp.obj : error LNK2001: unresolved external symbol __std_exception_destroy
base.cpp.obj : error LNK2001: unresolved external symbol __std_exception_destroy
frame_queue.cpp.obj : error LNK2001: unresolved external symbol __std_exception_destroy
selstart.cpp.obj : error LNK2001: unresolved external symbol __std_exception_destroy
selyesno.cpp.obj : error LNK2001: unresolved external symbol __std_exception_destroy
title.cpp.obj : error LNK2001: unresolved external symbol __std_exception_destroy
charpanel.cpp.obj : error LNK2001: unresolved external symbol __std_exception_destroy
selconn.cpp.obj : error LNK2001: unresolved external symbol __std_exception_destroy
selgame.cpp.obj : error LNK2001: unresolved external symbol __std_exception_destroy
selhero.cpp.obj : error LNK2001: unresolved external symbol __std_exception_destroy
selok.cpp.obj : error LNK2001: unresolved external symbol __std_exception_destroy
dialogs.cpp.obj : error LNK2001: unresolved external symbol __std_exception_destroy
mainmenu.cpp.obj : error LNK2001: unresolved external symbol __std_exception_destroy
progress.cpp.obj : error LNK2001: unresolved external symbol __std_exception_destroy
settingsmenu.cpp.obj : error LNK2001: unresolved external symbol __std_exception_destroy
logged_fstream.cpp.obj : error LNK2001: unresolved external symbol __std_exception_destroy
paths.cpp.obj : error LNK2001: unresolved external symbol __std_exception_destroy
credits.cpp.obj : error LNK2001: unresolved external symbol __std_exception_destroy
diabloui.cpp.obj : error LNK2001: unresolved external symbol __std_exception_destroy
console.cpp.obj : error LNK2001: unresolved external symbol __std_exception_destroy
display.cpp.obj : error LNK2001: unresolved external symbol __std_exception_destroy
file_util.cpp.obj : error LNK2001: unresolved external symbol __std_exception_destroy
language.cpp.obj : error LNK2001: unresolved external symbol __std_exception_destroy
chatlog.cpp.obj : error LNK2001: unresolved external symbol __std_exception_destroy
stash.cpp.obj : error LNK2001: unresolved external symbol __std_exception_destroy
xpbar.cpp.obj : error LNK2001: unresolved external symbol __std_exception_destroy
itemlabels.cpp.obj : error LNK2001: unresolved external symbol __std_exception_destroy
mpq_reader.cpp.obj : error LNK2001: unresolved external symbol __std_exception_destroy
mpq_sdl_rwops.cpp.obj : error LNK2001: unresolved external symbol __std_exception_destroy
mpq_writer.cpp.obj : error LNK2001: unresolved external symbol __std_exception_destroy
autopickup.cpp.obj : error LNK2001: unresolved external symbol __std_exception_destroy
joystick.cpp.obj : error LNK2001: unresolved external symbol __std_exception_destroy
plrctrls.cpp.obj : error LNK2001: unresolved external symbol __std_exception_destroy
demomode.cpp.obj : error LNK2001: unresolved external symbol __std_exception_destroy
text_render.cpp.obj : error LNK2001: unresolved external symbol __std_exception_destroy
stores.cpp.obj : error LNK2001: unresolved external symbol __std_exception_destroy
tmsg.cpp.obj : error LNK2001: unresolved external symbol __std_exception_destroy
towners.cpp.obj : error LNK2001: unresolved external symbol __std_exception_destroy
trigs.cpp.obj : error LNK2001: unresolved external symbol __std_exception_destroy
player.cpp.obj : error LNK2001: unresolved external symbol __std_exception_destroy
plrmsg.cpp.obj : error LNK2001: unresolved external symbol __std_exception_destroy
restrict.cpp.obj : error LNK2001: unresolved external symbol __std_exception_destroy
scrollrt.cpp.obj : error LNK2001: unresolved external symbol __std_exception_destroy
msg.cpp.obj : error LNK2001: unresolved external symbol __std_exception_destroy
objects.cpp.obj : error LNK2001: unresolved external symbol __std_exception_destroy
options.cpp.obj : error LNK2001: unresolved external symbol __std_exception_destroy
pfile.cpp.obj : error LNK2001: unresolved external symbol __std_exception_destroy
minitext.cpp.obj : error LNK2001: unresolved external symbol __std_exception_destroy
misdat.cpp.obj : error LNK2001: unresolved external symbol __std_exception_destroy
missiles.cpp.obj : error LNK2001: unresolved external symbol __std_exception_destroy
monster.cpp.obj : error LNK2001: unresolved external symbol __std_exception_destroy
init.cpp.obj : error LNK2001: unresolved external symbol __std_exception_destroy
inv.cpp.obj : error LNK2001: unresolved external symbol __std_exception_destroy
items.cpp.obj : error LNK2001: unresolved external symbol __std_exception_destroy
loadsave.cpp.obj : error LNK2001: unresolved external symbol __std_exception_destroy
error.cpp.obj : error LNK2001: unresolved external symbol __std_exception_destroy
assets.cpp.obj : error LNK2001: unresolved external symbol __std_exception_destroy
gendung.cpp.obj : error LNK2001: unresolved external symbol __std_exception_destroy
help.cpp.obj : error LNK2001: unresolved external symbol __std_exception_destroy
debug.cpp.obj : error LNK2001: unresolved external symbol __std_exception_destroy
diablo.cpp.obj : error LNK2001: unresolved external symbol __std_exception_destroy
drlg_l2.cpp.obj : error LNK2001: unresolved external symbol __std_exception_destroy
dthread.cpp.obj : error LNK2001: unresolved external symbol __std_exception_destroy
automap.cpp.obj : error LNK2001: unresolved external symbol __std_exception_destroy
capture.cpp.obj : error LNK2001: unresolved external symbol __std_exception_destroy
control.cpp.obj : error LNK2001: unresolved external symbol __std_exception_destroy
cursor.cpp.obj : error LNK2001: unresolved external symbol __std_exception_destroy
automap.cpp.obj : error LNK2019: unresolved external symbol __imp__stricmp referenced in function "void __cdecl devilution::`anonymous namespace'::DrawAutomapText(struct devilution::Surface const &)" (?DrawAutomapText@?A0x168ab6cf@devilution@@YAXAEBUSurface@2@@Z)
language.cpp.obj : error LNK2001: unresolved external symbol __imp__stricmp
effects.cpp.obj : error LNK2001: unresolved external symbol __imp__stricmp
MSVCRTD.lib(throw_bad_alloc.obj) : error LNK2001: unresolved external symbol _CxxThrowException
zt.lib(IncomingPacket.cpp.obj) : error LNK2001: unresolved external symbol _CxxThrowException
zt.lib(Capability.cpp.obj) : error LNK2001: unresolved external symbol _CxxThrowException
zt.lib(Tag.cpp.obj) : error LNK2001: unresolved external symbol _CxxThrowException
zt.lib(Revocation.cpp.obj) : error LNK2001: unresolved external symbol _CxxThrowException
zt.lib(OutboundMulticast.cpp.obj) : error LNK2001: unresolved external symbol _CxxThrowException
zt.lib(CertificateOfMembership.cpp.obj) : error LNK2001: unresolved external symbol _CxxThrowException
zt.lib(CertificateOfOwnership.cpp.obj) : error LNK2001: unresolved external symbol _CxxThrowException
zt.lib(Membership.cpp.obj) : error LNK2001: unresolved external symbol _CxxThrowException
zt.lib(Peer.cpp.obj) : error LNK2001: unresolved external symbol _CxxThrowException
zt.lib(Topology.cpp.obj) : error LNK2001: unresolved external symbol _CxxThrowException
zt.lib(Switch.cpp.obj) : error LNK2001: unresolved external symbol _CxxThrowException
zt.lib(SelfAwareness.cpp.obj) : error LNK2001: unresolved external symbol _CxxThrowException
zt.lib(Trace.cpp.obj) : error LNK2001: unresolved external symbol _CxxThrowException
zt.lib(NetworkConfig.cpp.obj) : error LNK2001: unresolved external symbol _CxxThrowException
zt.lib(Network.cpp.obj) : error LNK2001: unresolved external symbol _CxxThrowException
zt.lib(Bond.cpp.obj) : error LNK2001: unresolved external symbol _CxxThrowException
zt.lib(PortMapper.cpp.obj) : error LNK2001: unresolved external symbol _CxxThrowException
zt.lib(VirtualTap.cpp.obj) : error LNK2001: unresolved external symbol _CxxThrowException
zt.lib(Packet.cpp.obj) : error LNK2001: unresolved external symbol _CxxThrowException
zt.lib(Multicaster.cpp.obj) : error LNK2001: unresolved external symbol _CxxThrowException
zt.lib(OSUtils.cpp.obj) : error LNK2001: unresolved external symbol _CxxThrowException
zt.lib(NodeService.cpp.obj) : error LNK2001: unresolved external symbol _CxxThrowException
zt.lib(Controls.cpp.obj) : error LNK2001: unresolved external symbol _CxxThrowException
zt.lib(Node.cpp.obj) : error LNK2001: unresolved external symbol _CxxThrowException
SDL_audiolib.lib(Stream.cpp.obj) : error LNK2001: unresolved external symbol _CxxThrowException
SDL_audiolib.lib(stream_p.cpp.obj) : error LNK2001: unresolved external symbol _CxxThrowException
zt.lib(Utilities.cpp.obj) : error LNK2001: unresolved external symbol _CxxThrowException
zt.lib(Events.cpp.obj) : error LNK2001: unresolved external symbol _CxxThrowException
game_controller.cpp.obj : error LNK2001: unresolved external symbol _CxxThrowException
fmtd.lib(format.cc.obj) : error LNK2001: unresolved external symbol _CxxThrowException
libsmackerdec.lib(SmackerDecoder.cpp.obj) : error LNK2001: unresolved external symbol _CxxThrowException
libsmackerdec.lib(LogError.cpp.obj) : error LNK2001: unresolved external symbol _CxxThrowException
tcp_client.cpp.obj : error LNK2001: unresolved external symbol _CxxThrowException
tcp_server.cpp.obj : error LNK2001: unresolved external symbol _CxxThrowException
protocol_zt.cpp.obj : error LNK2001: unresolved external symbol _CxxThrowException
zerotier_native.cpp.obj : error LNK2001: unresolved external symbol _CxxThrowException
misc_msg.cpp.obj : error LNK2001: unresolved external symbol _CxxThrowException
locale.cpp.obj : error LNK2001: unresolved external symbol _CxxThrowException
sound.cpp.obj : error LNK2001: unresolved external symbol _CxxThrowException
push_aulib_decoder.cpp.obj : error LNK2001: unresolved external symbol _CxxThrowException
loopback.cpp.obj : error LNK2001: unresolved external symbol _CxxThrowException
packet.cpp.obj : error LNK2001: unresolved external symbol _CxxThrowException
storm_net.cpp.obj : error LNK2001: unresolved external symbol _CxxThrowException
storm_svid.cpp.obj : error LNK2001: unresolved external symbol _CxxThrowException
spell_list.cpp.obj : error LNK2001: unresolved external symbol _CxxThrowException
abstract_net.cpp.obj : error LNK2001: unresolved external symbol _CxxThrowException
base.cpp.obj : error LNK2001: unresolved external symbol _CxxThrowException
frame_queue.cpp.obj : error LNK2001: unresolved external symbol _CxxThrowException
selstart.cpp.obj : error LNK2001: unresolved external symbol _CxxThrowException
selyesno.cpp.obj : error LNK2001: unresolved external symbol _CxxThrowException
title.cpp.obj : error LNK2001: unresolved external symbol _CxxThrowException
charpanel.cpp.obj : error LNK2001: unresolved external symbol _CxxThrowException
selconn.cpp.obj : error LNK2001: unresolved external symbol _CxxThrowException
selgame.cpp.obj : error LNK2001: unresolved external symbol _CxxThrowException
selhero.cpp.obj : error LNK2001: unresolved external symbol _CxxThrowException
selok.cpp.obj : error LNK2001: unresolved external symbol _CxxThrowException
dialogs.cpp.obj : error LNK2001: unresolved external symbol _CxxThrowException
mainmenu.cpp.obj : error LNK2001: unresolved external symbol _CxxThrowException
progress.cpp.obj : error LNK2001: unresolved external symbol _CxxThrowException
settingsmenu.cpp.obj : error LNK2001: unresolved external symbol _CxxThrowException
logged_fstream.cpp.obj : error LNK2001: unresolved external symbol _CxxThrowException
paths.cpp.obj : error LNK2001: unresolved external symbol _CxxThrowException
credits.cpp.obj : error LNK2001: unresolved external symbol _CxxThrowException
diabloui.cpp.obj : error LNK2001: unresolved external symbol _CxxThrowException
console.cpp.obj : error LNK2001: unresolved external symbol _CxxThrowException
display.cpp.obj : error LNK2001: unresolved external symbol _CxxThrowException
file_util.cpp.obj : error LNK2001: unresolved external symbol _CxxThrowException
language.cpp.obj : error LNK2001: unresolved external symbol _CxxThrowException
chatlog.cpp.obj : error LNK2001: unresolved external symbol _CxxThrowException
stash.cpp.obj : error LNK2001: unresolved external symbol _CxxThrowException
xpbar.cpp.obj : error LNK2001: unresolved external symbol _CxxThrowException
itemlabels.cpp.obj : error LNK2001: unresolved external symbol _CxxThrowException
mpq_reader.cpp.obj : error LNK2001: unresolved external symbol _CxxThrowException
mpq_sdl_rwops.cpp.obj : error LNK2001: unresolved external symbol _CxxThrowException
mpq_writer.cpp.obj : error LNK2001: unresolved external symbol _CxxThrowException
autopickup.cpp.obj : error LNK2001: unresolved external symbol _CxxThrowException
joystick.cpp.obj : error LNK2001: unresolved external symbol _CxxThrowException
plrctrls.cpp.obj : error LNK2001: unresolved external symbol _CxxThrowException
demomode.cpp.obj : error LNK2001: unresolved external symbol _CxxThrowException
text_render.cpp.obj : error LNK2001: unresolved external symbol _CxxThrowException
stores.cpp.obj : error LNK2001: unresolved external symbol _CxxThrowException
tmsg.cpp.obj : error LNK2001: unresolved external symbol _CxxThrowException
towners.cpp.obj : error LNK2001: unresolved external symbol _CxxThrowException
trigs.cpp.obj : error LNK2001: unresolved external symbol _CxxThrowException
player.cpp.obj : error LNK2001: unresolved external symbol _CxxThrowException
plrmsg.cpp.obj : error LNK2001: unresolved external symbol _CxxThrowException
restrict.cpp.obj : error LNK2001: unresolved external symbol _CxxThrowException
scrollrt.cpp.obj : error LNK2001: unresolved external symbol _CxxThrowException
msg.cpp.obj : error LNK2001: unresolved external symbol _CxxThrowException
objects.cpp.obj : error LNK2001: unresolved external symbol _CxxThrowException
options.cpp.obj : error LNK2001: unresolved external symbol _CxxThrowException
pfile.cpp.obj : error LNK2001: unresolved external symbol _CxxThrowException
minitext.cpp.obj : error LNK2001: unresolved external symbol _CxxThrowException
misdat.cpp.obj : error LNK2001: unresolved external symbol _CxxThrowException
missiles.cpp.obj : error LNK2001: unresolved external symbol _CxxThrowException
monster.cpp.obj : error LNK2001: unresolved external symbol _CxxThrowException
init.cpp.obj : error LNK2001: unresolved external symbol _CxxThrowException
inv.cpp.obj : error LNK2001: unresolved external symbol _CxxThrowException
items.cpp.obj : error LNK2001: unresolved external symbol _CxxThrowException
loadsave.cpp.obj : error LNK2001: unresolved external symbol _CxxThrowException
error.cpp.obj : error LNK2001: unresolved external symbol _CxxThrowException
assets.cpp.obj : error LNK2001: unresolved external symbol _CxxThrowException
gendung.cpp.obj : error LNK2001: unresolved external symbol _CxxThrowException
help.cpp.obj : error LNK2001: unresolved external symbol _CxxThrowException
debug.cpp.obj : error LNK2001: unresolved external symbol _CxxThrowException
diablo.cpp.obj : error LNK2001: unresolved external symbol _CxxThrowException
drlg_l2.cpp.obj : error LNK2001: unresolved external symbol _CxxThrowException
dthread.cpp.obj : error LNK2001: unresolved external symbol _CxxThrowException
automap.cpp.obj : error LNK2001: unresolved external symbol _CxxThrowException
capture.cpp.obj : error LNK2001: unresolved external symbol _CxxThrowException
control.cpp.obj : error LNK2001: unresolved external symbol _CxxThrowException
cursor.cpp.obj : error LNK2001: unresolved external symbol _CxxThrowException
capture.cpp.obj : error LNK2019: unresolved external symbol __imp__get_stream_buffer_pointers referenced in function "protected: void __cdecl std::basic_filebuf<char,struct std::char_traits<char> >::_Init(struct _iobuf *,enum std::basic_filebuf<char,struct std::char_traits<char> >::_Initfl)" (?_Init@?$basic_filebuf@DU?$char_traits@D@std@@@std@@IEAAXPEAU_iobuf@@W4_Initfl@12@@Z)
demomode.cpp.obj : error LNK2001: unresolved external symbol __imp__get_stream_buffer_pointers
mpq_writer.cpp.obj : error LNK2001: unresolved external symbol __imp__get_stream_buffer_pointers
file_util.cpp.obj : error LNK2001: unresolved external symbol __imp__get_stream_buffer_pointers
libmpq.lib(mpq.c.obj) : error LNK2001: unresolved external symbol __imp_fclose
zt.lib(OSUtils.cpp.obj) : error LNK2001: unresolved external symbol __imp_fclose
zt.lib(NodeService.cpp.obj) : error LNK2001: unresolved external symbol __imp_fclose
file_util.cpp.obj : error LNK2001: unresolved external symbol __imp_fclose
libpng16_staticd.lib(png.c.obj) : error LNK2001: unresolved external symbol __imp_fclose
libpng16_staticd.lib(pngread.c.obj) : error LNK2001: unresolved external symbol __imp_fclose
libpng16_staticd.lib(pngwrite.c.obj) : error LNK2001: unresolved external symbol __imp_fclose
capture.cpp.obj : error LNK2001: unresolved external symbol __imp_fclose
restrict.cpp.obj : error LNK2001: unresolved external symbol __imp_fclose
demomode.cpp.obj : error LNK2001: unresolved external symbol __imp_fclose
mpq_writer.cpp.obj : error LNK2001: unresolved external symbol __imp_fclose
zt.lib(api_msg.c.obj) : error LNK2001: unresolved external symbol __imp_fflush
zt.lib(tcp_out.c.obj) : error LNK2001: unresolved external symbol __imp_fflush
zt.lib(udp.c.obj) : error LNK2001: unresolved external symbol __imp_fflush
zt.lib(memp.c.obj) : error LNK2001: unresolved external symbol __imp_fflush
zt.lib(ip4_frag.c.obj) : error LNK2001: unresolved external symbol __imp_fflush
zt.lib(etharp.c.obj) : error LNK2001: unresolved external symbol __imp_fflush
zt.lib(netif.c.obj) : error LNK2001: unresolved external symbol __imp_fflush
zt.lib(netbuf.c.obj) : error LNK2001: unresolved external symbol __imp_fflush
zt.lib(api_lib.c.obj) : error LNK2001: unresolved external symbol __imp_fflush
zt.lib(tcp.c.obj) : error LNK2001: unresolved external symbol __imp_fflush
libpng16_staticd.lib(pngwrite.c.obj) : error LNK2001: unresolved external symbol __imp_fflush
libpng16_staticd.lib(pngwio.c.obj) : error LNK2001: unresolved external symbol __imp_fflush
zt.lib(sockets.c.obj) : error LNK2001: unresolved external symbol __imp_fflush
zt.lib(pbuf.c.obj) : error LNK2001: unresolved external symbol __imp_fflush
capture.cpp.obj : error LNK2001: unresolved external symbol __imp_fflush
demomode.cpp.obj : error LNK2001: unresolved external symbol __imp_fflush
mpq_writer.cpp.obj : error LNK2001: unresolved external symbol __imp_fflush
file_util.cpp.obj : error LNK2001: unresolved external symbol __imp_fflush
capture.cpp.obj : error LNK2019: unresolved external symbol __imp_fgetc referenced in function "bool __cdecl std::_Fgetc<char>(char &,struct _iobuf *)" (??$_Fgetc@D@std@@YA_NAEADPEAU_iobuf@@@Z)
demomode.cpp.obj : error LNK2001: unresolved external symbol __imp_fgetc
mpq_writer.cpp.obj : error LNK2001: unresolved external symbol __imp_fgetc
file_util.cpp.obj : error LNK2001: unresolved external symbol __imp_fgetc
capture.cpp.obj : error LNK2019: unresolved external symbol __imp_fgetpos referenced in function "protected: virtual class std::fpos<struct _Mbstatet> __cdecl std::basic_filebuf<char,struct std::char_traits<char> >::seekoff(__int64,int,int)" (?seekoff@?$basic_filebuf@DU?$char_traits@D@std@@@std@@MEAA?AV?$fpos@U_Mbstatet@@@2@_JHH@Z)
demomode.cpp.obj : error LNK2001: unresolved external symbol __imp_fgetpos
mpq_writer.cpp.obj : error LNK2001: unresolved external symbol __imp_fgetpos
file_util.cpp.obj : error LNK2001: unresolved external symbol __imp_fgetpos
fmtd.lib(format.cc.obj) : error LNK2001: unresolved external symbol __imp_fputc
capture.cpp.obj : error LNK2001: unresolved external symbol __imp_fputc
demomode.cpp.obj : error LNK2001: unresolved external symbol __imp_fputc
mpq_writer.cpp.obj : error LNK2001: unresolved external symbol __imp_fputc
file_util.cpp.obj : error LNK2001: unresolved external symbol __imp_fputc
libpng16_staticd.lib(pngrio.c.obj) : error LNK2001: unresolved external symbol __imp_fread
libmpq.lib(mpq.c.obj) : error LNK2001: unresolved external symbol __imp_fread
zt.lib(OSUtils.cpp.obj) : error LNK2001: unresolved external symbol __imp_fread
zt.lib(NodeService.cpp.obj) : error LNK2001: unresolved external symbol __imp_fread
capture.cpp.obj : error LNK2001: unresolved external symbol __imp_fread
demomode.cpp.obj : error LNK2001: unresolved external symbol __imp_fread
mpq_writer.cpp.obj : error LNK2001: unresolved external symbol __imp_fread
file_util.cpp.obj : error LNK2001: unresolved external symbol __imp_fread
capture.cpp.obj : error LNK2019: unresolved external symbol __imp_fsetpos referenced in function "protected: virtual class std::fpos<struct _Mbstatet> __cdecl std::basic_filebuf<char,struct std::char_traits<char> >::seekpos(class std::fpos<struct _Mbstatet>,int)" (?seekpos@?$basic_filebuf@DU?$char_traits@D@std@@@std@@MEAA?AV?$fpos@U_Mbstatet@@@2@V32@H@Z)
demomode.cpp.obj : error LNK2001: unresolved external symbol __imp_fsetpos
mpq_writer.cpp.obj : error LNK2001: unresolved external symbol __imp_fsetpos
file_util.cpp.obj : error LNK2001: unresolved external symbol __imp_fsetpos
libmpq.lib(mpq.c.obj) : error LNK2001: unresolved external symbol __imp__fseeki64
capture.cpp.obj : error LNK2001: unresolved external symbol __imp__fseeki64
demomode.cpp.obj : error LNK2001: unresolved external symbol __imp__fseeki64
mpq_writer.cpp.obj : error LNK2001: unresolved external symbol __imp__fseeki64
file_util.cpp.obj : error LNK2001: unresolved external symbol __imp__fseeki64
libpng16_staticd.lib(pngwio.c.obj) : error LNK2001: unresolved external symbol __imp_fwrite
fmtd.lib(format.cc.obj) : error LNK2001: unresolved external symbol __imp_fwrite
zt.lib(OSUtils.cpp.obj) : error LNK2001: unresolved external symbol __imp_fwrite
zt.lib(NodeService.cpp.obj) : error LNK2001: unresolved external symbol __imp_fwrite
capture.cpp.obj : error LNK2001: unresolved external symbol __imp_fwrite
demomode.cpp.obj : error LNK2001: unresolved external symbol __imp_fwrite
mpq_writer.cpp.obj : error LNK2001: unresolved external symbol __imp_fwrite
file_util.cpp.obj : error LNK2001: unresolved external symbol __imp_fwrite
capture.cpp.obj : error LNK2019: unresolved external symbol __imp_setvbuf referenced in function "protected: virtual class std::basic_streambuf<char,struct std::char_traits<char> > * __cdecl std::basic_filebuf<char,struct std::char_traits<char> >::setbuf(char *,__int64)" (?setbuf@?$basic_filebuf@DU?$char_traits@D@std@@@std@@MEAAPEAV?$basic_streambuf@DU?$char_traits@D@std@@@2@PEAD_J@Z)
demomode.cpp.obj : error LNK2001: unresolved external symbol __imp_setvbuf
mpq_writer.cpp.obj : error LNK2001: unresolved external symbol __imp_setvbuf
file_util.cpp.obj : error LNK2001: unresolved external symbol __imp_setvbuf
capture.cpp.obj : error LNK2019: unresolved external symbol __imp_ungetc referenced in function "bool __cdecl std::_Ungetc<char>(char const &,struct _iobuf *)" (??$_Ungetc@D@std@@YA_NAEBDPEAU_iobuf@@@Z)
demomode.cpp.obj : error LNK2001: unresolved external symbol __imp_ungetc
mpq_writer.cpp.obj : error LNK2001: unresolved external symbol __imp_ungetc
file_util.cpp.obj : error LNK2001: unresolved external symbol __imp_ungetc
capture.cpp.obj : error LNK2019: unresolved external symbol __imp__lock_file referenced in function "public: virtual void __cdecl std::basic_filebuf<char,struct std::char_traits<char> >::_Lock(void)" (?_Lock@?$basic_filebuf@DU?$char_traits@D@std@@@std@@UEAAXXZ)
demomode.cpp.obj : error LNK2001: unresolved external symbol __imp__lock_file
mpq_writer.cpp.obj : error LNK2001: unresolved external symbol __imp__lock_file
file_util.cpp.obj : error LNK2001: unresolved external symbol __imp__lock_file
capture.cpp.obj : error LNK2019: unresolved external symbol __imp__unlock_file referenced in function "public: virtual void __cdecl std::basic_filebuf<char,struct std::char_traits<char> >::_Unlock(void)" (?_Unlock@?$basic_filebuf@DU?$char_traits@D@std@@@std@@UEAAXXZ)
demomode.cpp.obj : error LNK2001: unresolved external symbol __imp__unlock_file
mpq_writer.cpp.obj : error LNK2001: unresolved external symbol __imp__unlock_file
file_util.cpp.obj : error LNK2001: unresolved external symbol __imp__unlock_file
zt.lib(portlistingparse.c.obj) : error LNK2001: unresolved external symbol strlen
zt.lib(NodeService.cpp.obj) : error LNK2001: unresolved external symbol strlen
zt.lib(Node.cpp.obj) : error LNK2001: unresolved external symbol strlen
zt.lib(miniupnpc.c.obj) : error LNK2001: unresolved external symbol strlen
zt.lib(minisoap.c.obj) : error LNK2001: unresolved external symbol strlen
zt.lib(Sockets.cpp.obj) : error LNK2001: unresolved external symbol strlen
zt.lib(dns.c.obj) : error LNK2001: unresolved external symbol strlen
zt.lib(netdb.c.obj) : error LNK2001: unresolved external symbol strlen
zt.lib(OSUtils.cpp.obj) : error LNK2001: unresolved external symbol strlen
sodium.lib(argon2.c.obj) : error LNK2001: unresolved external symbol strlen
sodium.lib(argon2-encoding.c.obj) : error LNK2001: unresolved external symbol strlen
zt.lib(def.c.obj) : error LNK2001: unresolved external symbol strlen
zt.lib(pbuf.c.obj) : error LNK2001: unresolved external symbol strlen
libpng16_staticd.lib(pngrutil.c.obj) : error LNK2001: unresolved external symbol strlen
fmtd.lib(format.cc.obj) : error LNK2001: unresolved external symbol strlen
libmpq.lib(mpq.c.obj) : error LNK2001: unresolved external symbol strlen
sodium.lib(pwhash_argon2i.c.obj) : error LNK2001: unresolved external symbol strlen
zerotier_lwip.cpp.obj : error LNK2001: unresolved external symbol strlen
game_controller.cpp.obj : error LNK2001: unresolved external symbol strlen
libpng16_staticd.lib(pngset.c.obj) : error LNK2001: unresolved external symbol strlen
libpng16_staticd.lib(pngwutil.c.obj) : error LNK2001: unresolved external symbol strlen
tcp_client.cpp.obj : error LNK2001: unresolved external symbol strlen
tcp_server.cpp.obj : error LNK2001: unresolved external symbol strlen
protocol_zt.cpp.obj : error LNK2001: unresolved external symbol strlen
zerotier_native.cpp.obj : error LNK2001: unresolved external symbol strlen
storm_svid.cpp.obj : error LNK2001: unresolved external symbol strlen
misc_msg.cpp.obj : error LNK2001: unresolved external symbol strlen
sound.cpp.obj : error LNK2001: unresolved external symbol strlen
soundsample.cpp.obj : error LNK2001: unresolved external symbol strlen
frame_queue.cpp.obj : error LNK2001: unresolved external symbol strlen
loopback.cpp.obj : error LNK2001: unresolved external symbol strlen
packet.cpp.obj : error LNK2001: unresolved external symbol strlen
storm_net.cpp.obj : error LNK2001: unresolved external symbol strlen
charpanel.cpp.obj : error LNK2001: unresolved external symbol strlen
spell_list.cpp.obj : error LNK2001: unresolved external symbol strlen
abstract_net.cpp.obj : error LNK2001: unresolved external symbol strlen
base.cpp.obj : error LNK2001: unresolved external symbol strlen
diabloui.cpp.obj : error LNK2001: unresolved external symbol strlen
dialogs.cpp.obj : error LNK2001: unresolved external symbol strlen
selgame.cpp.obj : error LNK2001: unresolved external symbol strlen
selhero.cpp.obj : error LNK2001: unresolved external symbol strlen
file_util.cpp.obj : error LNK2001: unresolved external symbol strlen
language.cpp.obj : error LNK2001: unresolved external symbol strlen
paths.cpp.obj : error LNK2001: unresolved external symbol strlen
art.cpp.obj : error LNK2001: unresolved external symbol strlen
chatlog.cpp.obj : error LNK2001: unresolved external symbol strlen
stash.cpp.obj : error LNK2001: unresolved external symbol strlen
xpbar.cpp.obj : error LNK2001: unresolved external symbol strlen
display.cpp.obj : error LNK2001: unresolved external symbol strlen
plrctrls.cpp.obj : error LNK2001: unresolved external symbol strlen
animationinfo.cpp.obj : error LNK2001: unresolved external symbol strlen
text_render.cpp.obj : error LNK2001: unresolved external symbol strlen
mpq_writer.cpp.obj : error LNK2001: unresolved external symbol strlen
options.cpp.obj : error LNK2001: unresolved external symbol strlen
scrollrt.cpp.obj : error LNK2001: unresolved external symbol strlen
controller_motion.cpp.obj : error LNK2001: unresolved external symbol strlen
joystick.cpp.obj : error LNK2001: unresolved external symbol strlen
items.cpp.obj : error LNK2001: unresolved external symbol strlen
misdat.cpp.obj : error LNK2001: unresolved external symbol strlen
msg.cpp.obj : error LNK2001: unresolved external symbol strlen
objects.cpp.obj : error LNK2001: unresolved external symbol strlen
dx.cpp.obj : error LNK2001: unresolved external symbol strlen
assets.cpp.obj : error LNK2001: unresolved external symbol strlen
init.cpp.obj : error LNK2001: unresolved external symbol strlen
inv.cpp.obj : error LNK2001: unresolved external symbol strlen
capture.cpp.obj : error LNK2001: unresolved external symbol strlen
codec.cpp.obj : error LNK2001: unresolved external symbol strlen
control.cpp.obj : error LNK2001: unresolved external symbol strlen
debug.cpp.obj : error LNK2001: unresolved external symbol strlen
zt.lib(miniupnpc.c.obj) : error LNK2001: unresolved external symbol strcmp
zt.lib(igd_desc_parse.c.obj) : error LNK2001: unresolved external symbol strcmp
zt.lib(upnpreplyparse.c.obj) : error LNK2001: unresolved external symbol strcmp
control.cpp.obj : error LNK2001: unresolved external symbol strcmp
language.cpp.obj : error LNK2001: unresolved external symbol strcmp
tcp_server.cpp.obj : error LNK2001: unresolved external symbol strcmp
zt.lib(OSUtils.cpp.obj) : error LNK2001: unresolved external symbol strcmp
selgame.cpp.obj : error LNK2001: unresolved external symbol strcpy
selhero.cpp.obj : error LNK2001: unresolved external symbol strcpy
control.cpp.obj : error LNK2001: unresolved external symbol strcpy
options.cpp.obj : error LNK2001: unresolved external symbol strcpy
pack.cpp.obj : error LNK2001: unresolved external symbol strcpy
pfile.cpp.obj : error LNK2001: unresolved external symbol strcpy
control.cpp.obj : error LNK2019: unresolved external symbol __imp_strncat referenced in function "void __cdecl devilution::control_new_text(class std::basic_string_view<char,struct std::char_traits<char> >)" (?control_new_text@devilution@@YAXV?$basic_string_view@DU?$char_traits@D@std@@@std@@@Z)
soundsample.cpp.obj : error LNK2019: unresolved external symbol abs referenced in function "public: int __cdecl devilution::SoundSample::SetChunkStream(class std::basic_string<char,struct std::char_traits<char>,class std::allocator<char> >,bool,bool)" (?SetChunkStream@SoundSample@devilution@@QEAAHV?$basic_string@DU?$char_traits@D@std@@V?$allocator@D@2@@std@@_N1@Z)
libpng16_staticd.lib(pngrtran.c.obj) : error LNK2001: unresolved external symbol abs
zt.lib(Bond.cpp.obj) : error LNK2001: unresolved external symbol abs
plrctrls.cpp.obj : error LNK2001: unresolved external symbol abs
random.cpp.obj : error LNK2001: unresolved external symbol abs
itemlabels.cpp.obj : error LNK2001: unresolved external symbol abs
effects.cpp.obj : error LNK2001: unresolved external symbol abs
scrollrt.cpp.obj : error LNK2001: unresolved external symbol abs
sync.cpp.obj : error LNK2001: unresolved external symbol abs
towners.cpp.obj : error LNK2001: unresolved external symbol abs
trigs.cpp.obj : error LNK2001: unresolved external symbol abs
multi.cpp.obj : error LNK2001: unresolved external symbol abs
objects.cpp.obj : error LNK2001: unresolved external symbol abs
path.cpp.obj : error LNK2001: unresolved external symbol abs
player.cpp.obj : error LNK2001: unresolved external symbol abs
inv.cpp.obj : error LNK2001: unresolved external symbol abs
missiles.cpp.obj : error LNK2001: unresolved external symbol abs
monster.cpp.obj : error LNK2001: unresolved external symbol abs
msg.cpp.obj : error LNK2001: unresolved external symbol abs
cursor.cpp.obj : error LNK2001: unresolved external symbol abs
debug.cpp.obj : error LNK2001: unresolved external symbol abs
diablo.cpp.obj : error LNK2001: unresolved external symbol abs
drlg_l2.cpp.obj : error LNK2001: unresolved external symbol abs
  Hint on symbols that are defined and could potentially match:
    "struct devilution::Displacement __cdecl devilution::abs(struct devilution::Displacement)" (?abs@devilution@@YA?AUDisplacement@1@U21@@Z)
text_render.cpp.obj : error LNK2001: unresolved external symbol __imp_ceilf
itemlabels.cpp.obj : error LNK2001: unresolved external symbol __imp_ceilf
debug.cpp.obj : error LNK2001: unresolved external symbol __imp_ceilf
loadsave.cpp.obj : error LNK2001: unresolved external symbol __imp_ceilf
options.cpp.obj : error LNK2001: unresolved external symbol __imp_ceilf
scrollrt.cpp.obj : error LNK2001: unresolved external symbol __imp_ceilf
zt.lib(OSUtils.cpp.obj) : error LNK2001: unresolved external symbol __imp__errno
sodium.lib(pwhash_argon2i.c.obj) : error LNK2001: unresolved external symbol __imp__errno
sodium.lib(pwhash_argon2id.c.obj) : error LNK2001: unresolved external symbol __imp__errno
sodium.lib(codecs.c.obj) : error LNK2001: unresolved external symbol __imp__errno
zt.lib(sockets.c.obj) : error LNK2001: unresolved external symbol __imp__errno
sodium.lib(core.c.obj) : error LNK2001: unresolved external symbol __imp__errno
sodium.lib(crypto_pwhash.c.obj) : error LNK2001: unresolved external symbol __imp__errno
sodium.lib(utils.c.obj) : error LNK2001: unresolved external symbol __imp__errno
sodium.lib(argon2-core.c.obj) : error LNK2001: unresolved external symbol __imp__errno
libpng16_staticd.lib(pngread.c.obj) : error LNK2001: unresolved external symbol __imp__errno
libpng16_staticd.lib(pngwrite.c.obj) : error LNK2001: unresolved external symbol __imp__errno
fmtd.lib(format.cc.obj) : error LNK2001: unresolved external symbol __imp__errno
libmpq.lib(mpq.c.obj) : error LNK2001: unresolved external symbol __imp__errno
debug.cpp.obj : error LNK2001: unresolved external symbol __imp__errno
demomode.cpp.obj : error LNK2001: unresolved external symbol __imp__errno
mpq_writer.cpp.obj : error LNK2001: unresolved external symbol __imp__errno
protocol_zt.cpp.obj : error LNK2001: unresolved external symbol __imp__errno
zt.lib(miniwget.c.obj) : error LNK2001: unresolved external symbol __imp_atoi
debug.cpp.obj : error LNK2001: unresolved external symbol __imp_atoi
zt.lib(netif.c.obj) : error LNK2001: unresolved external symbol __imp_atoi
zt.lib(netdb.c.obj) : error LNK2001: unresolved external symbol __imp_atoi
zt.lib(miniupnpc.c.obj) : error LNK2001: unresolved external symbol __imp_atoi
debug.cpp.obj : error LNK2019: unresolved external symbol __imp_strtoul referenced in function "unsigned long __cdecl std::stoul(class std::basic_string<char,struct std::char_traits<char>,class std::allocator<char> > const &,unsigned __int64 *,int)" (?stoul@std@@YAKAEBV?$basic_string@DU?$char_traits@D@std@@V?$allocator@D@2@@1@PEA_KH@Z)
zt.lib(InetAddress.cpp.obj) : error LNK2001: unresolved external symbol __imp_strtoul
zt.lib(upnpcommands.c.obj) : error LNK2001: unresolved external symbol __imp_strtoul
zt.lib(miniwget.c.obj) : error LNK2001: unresolved external symbol __imp_strtoul
zt.lib(dns.c.obj) : error LNK2001: unresolved external symbol __imp_tolower
debug.cpp.obj : error LNK2001: unresolved external symbol __imp_tolower
items.cpp.obj : error LNK2001: unresolved external symbol __imp_tolower
towners.cpp.obj : error LNK2001: unresolved external symbol __imp_tolower
misc_msg.cpp.obj : error LNK2001: unresolved external symbol __imp_tolower
fmtd.lib(format.cc.obj) : error LNK2001: unresolved external symbol memchr
towners.cpp.obj : error LNK2001: unresolved external symbol memchr
text_render.cpp.obj : error LNK2001: unresolved external symbol memchr
language.cpp.obj : error LNK2001: unresolved external symbol memchr
credits.cpp.obj : error LNK2001: unresolved external symbol memchr
minitext.cpp.obj : error LNK2001: unresolved external symbol memchr
misdat.cpp.obj : error LNK2001: unresolved external symbol memchr
options.cpp.obj : error LNK2001: unresolved external symbol memchr
options.cpp.obj : error LNK2001: unresolved external symbol memchr
debug.cpp.obj : error LNK2001: unresolved external symbol memchr
error.cpp.obj : error LNK2001: unresolved external symbol memchr
help.cpp.obj : error LNK2001: unresolved external symbol memchr
items.cpp.obj : error LNK2001: unresolved external symbol memchr
zt.lib(connecthostport.c.obj) : error LNK2001: unresolved external symbol memcmp
zt.lib(portlistingparse.c.obj) : error LNK2001: unresolved external symbol memcmp
zt.lib(IncomingPacket.cpp.obj) : error LNK2001: unresolved external symbol memcmp
zt.lib(igd_desc_parse.c.obj) : error LNK2001: unresolved external symbol memcmp
zt.lib(minissdpc.c.obj) : error LNK2001: unresolved external symbol memcmp
zt.lib(minixml.c.obj) : error LNK2001: unresolved external symbol memcmp
zt.lib(Topology.cpp.obj) : error LNK2001: unresolved external symbol memcmp
zt.lib(SelfAwareness.cpp.obj) : error LNK2001: unresolved external symbol memcmp
zt.lib(miniupnpc.c.obj) : error LNK2001: unresolved external symbol memcmp
zt.lib(Membership.cpp.obj) : error LNK2001: unresolved external symbol memcmp
zt.lib(PortMapper.cpp.obj) : error LNK2001: unresolved external symbol memcmp
zt.lib(Network.cpp.obj) : error LNK2001: unresolved external symbol memcmp
zt.lib(Bond.cpp.obj) : error LNK2001: unresolved external symbol memcmp
zt.lib(Peer.cpp.obj) : error LNK2001: unresolved external symbol memcmp
zt.lib(OSUtils.cpp.obj) : error LNK2001: unresolved external symbol memcmp
zt.lib(NodeService.cpp.obj) : error LNK2001: unresolved external symbol memcmp
zt.lib(NodeService.cpp.obj) : error LNK2001: unresolved external symbol memcmp
zt.lib(Node.cpp.obj) : error LNK2001: unresolved external symbol memcmp
zt.lib(ethernet.c.obj) : error LNK2001: unresolved external symbol memcmp
zt.lib(Utilities.cpp.obj) : error LNK2001: unresolved external symbol memcmp
zt.lib(InetAddress.cpp.obj) : error LNK2001: unresolved external symbol memcmp
zt.lib(OSUtils.cpp.obj) : error LNK2001: unresolved external symbol memcmp
libpng16_staticd.lib(pngset.c.obj) : error LNK2001: unresolved external symbol memcmp
fmtd.lib(format.cc.obj) : error LNK2001: unresolved external symbol memcmp
fmtd.lib(format.cc.obj) : error LNK2001: unresolved external symbol memcmp
SDL_audiolib.lib(stream_p.cpp.obj) : error LNK2001: unresolved external symbol memcmp
locale.cpp.obj : error LNK2001: unresolved external symbol memcmp
protocol_zt.cpp.obj : error LNK2001: unresolved external symbol memcmp
libpng16_staticd.lib(png.c.obj) : error LNK2001: unresolved external symbol memcmp
libpng16_staticd.lib(pngread.c.obj) : error LNK2001: unresolved external symbol memcmp
selgame.cpp.obj : error LNK2001: unresolved external symbol memcmp
abstract_net.cpp.obj : error LNK2001: unresolved external symbol memcmp
abstract_net.cpp.obj : error LNK2001: unresolved external symbol memcmp
base.cpp.obj : error LNK2001: unresolved external symbol memcmp
options.cpp.obj : error LNK2001: unresolved external symbol memcmp
towners.cpp.obj : error LNK2001: unresolved external symbol memcmp
display.cpp.obj : error LNK2001: unresolved external symbol memcmp
language.cpp.obj : error LNK2001: unresolved external symbol memcmp
items.cpp.obj : error LNK2001: unresolved external symbol memcmp
misdat.cpp.obj : error LNK2001: unresolved external symbol memcmp
misdat.cpp.obj : error LNK2001: unresolved external symbol memcmp
options.cpp.obj : error LNK2001: unresolved external symbol memcmp
debug.cpp.obj : error LNK2001: unresolved external symbol memcmp
diablo.cpp.obj : error LNK2001: unresolved external symbol memcmp
error.cpp.obj : error LNK2001: unresolved external symbol memcmp
init.cpp.obj : error LNK2001: unresolved external symbol memcmp
diablo.cpp.obj : error LNK2019: unresolved external symbol __imp_exit referenced in function "void __cdecl devilution::diablo_quit(int)" (?diablo_quit@devilution@@YAXH@Z)
zt.lib(Utils.cpp.obj) : error LNK2001: unresolved external symbol __imp_exit
dthread.cpp.obj : error LNK2019: unresolved external symbol __imp__beginthreadex referenced in function "public: __cdecl devilution::SdlThread::SdlThread(int (__cdecl*)(void *),void *)" (??0SdlThread@devilution@@QEAA@P6AHPEAX@Z0@Z)
nthread.cpp.obj : error LNK2001: unresolved external symbol __imp__beginthreadex
abstract_net.cpp.obj : error LNK2001: unresolved external symbol __imp__beginthreadex
tcp_server.cpp.obj : error LNK2001: unresolved external symbol __imp__beginthreadex
dthread.cpp.obj : error LNK2019: unresolved external symbol __imp__endthreadex referenced in function "public: __cdecl devilution::SdlThread::SdlThread(int (__cdecl*)(void *),void *)" (??0SdlThread@devilution@@QEAA@P6AHPEAX@Z0@Z)
nthread.cpp.obj : error LNK2001: unresolved external symbol __imp__endthreadex
encrypt.cpp.obj : error LNK2019: unresolved external symbol __imp_toupper referenced in function "unsigned int __cdecl devilution::Hash(char const *,int)" (?Hash@devilution@@YAIPEBDH@Z)
libmpq.lib(common.c.obj) : error LNK2001: unresolved external symbol __imp_toupper
zt.lib(Switch.cpp.obj) : error LNK2001: unresolved external symbol sqrt
lighting.cpp.obj : error LNK2001: unresolved external symbol sqrt
missiles.cpp.obj : error LNK2001: unresolved external symbol sqrt
plrctrls.cpp.obj : error LNK2001: unresolved external symbol sqrt
gamepad.cpp.obj : error LNK2001: unresolved external symbol sqrt
  Hint on symbols that are defined and could potentially match:
    "float __cdecl sqrt(float)" (?sqrt@@YAMM@Z)
SDL_audiolib.lib(Decoder.cpp.obj) : error LNK2001: unresolved external symbol _purecall
SDL_audiolib.lib(Resampler.cpp.obj) : error LNK2001: unresolved external symbol _purecall
zt.lib(OSUtils.cpp.obj) : error LNK2001: unresolved external symbol _purecall
zt.lib(Node.cpp.obj) : error LNK2001: unresolved external symbol _purecall
tcp_server.cpp.obj : error LNK2001: unresolved external symbol _purecall
gamepad.cpp.obj : error LNK2001: unresolved external symbol _purecall
renderers.cpp.obj : error LNK2001: unresolved external symbol _purecall
fmtd.lib(format.cc.obj) : error LNK2001: unresolved external symbol _purecall
misdat.cpp.obj : error LNK2001: unresolved external symbol _purecall
options.cpp.obj : error LNK2001: unresolved external symbol _purecall
abstract_net.cpp.obj : error LNK2001: unresolved external symbol _purecall
tcp_client.cpp.obj : error LNK2001: unresolved external symbol _purecall
misdat.cpp.obj : error LNK2019: unresolved external symbol __imp__dsign referenced in function "bool __cdecl signbit(double)" (?signbit@@YA_NN@Z)
options.cpp.obj : error LNK2001: unresolved external symbol __imp__dsign
fmtd.lib(format.cc.obj) : error LNK2001: unresolved external symbol __imp__dsign
zt.lib(OSUtils.cpp.obj) : error LNK2001: unresolved external symbol __imp__dsign
misdat.cpp.obj : error LNK2019: unresolved external symbol __imp__dtest referenced in function "int __cdecl fpclassify(double)" (?fpclassify@@YAHN@Z)
options.cpp.obj : error LNK2001: unresolved external symbol __imp__dtest
fmtd.lib(format.cc.obj) : error LNK2001: unresolved external symbol __imp__dtest
zt.lib(OSUtils.cpp.obj) : error LNK2001: unresolved external symbol __imp__dtest
misdat.cpp.obj : error LNK2019: unresolved external symbol __imp__ldtest referenced in function "int __cdecl fpclassify(long double)" (?fpclassify@@YAHO@Z)
options.cpp.obj : error LNK2001: unresolved external symbol __imp__ldtest
fmtd.lib(format.cc.obj) : error LNK2001: unresolved external symbol __imp__ldtest
misdat.cpp.obj : error LNK2019: unresolved external symbol __imp__fdtest referenced in function "int __cdecl fpclassify(float)" (?fpclassify@@YAHM@Z)
options.cpp.obj : error LNK2001: unresolved external symbol __imp__fdtest
fmtd.lib(format.cc.obj) : error LNK2001: unresolved external symbol __imp__fdtest
zt.lib(Bond.cpp.obj) : error LNK2001: unresolved external symbol __imp__time64
multi.cpp.obj : error LNK2001: unresolved external symbol __imp__time64
objects.cpp.obj : error LNK2001: unresolved external symbol __imp__time64
chatlog.cpp.obj : error LNK2001: unresolved external symbol __imp__time64
zt.lib(Utils.cpp.obj) : error LNK2001: unresolved external symbol __imp__time64
objects.cpp.obj : error LNK2019: unresolved external symbol __imp__localtime64 referenced in function localtime
chatlog.cpp.obj : error LNK2001: unresolved external symbol __imp__localtime64
zt.lib(Bond.cpp.obj) : error LNK2001: unresolved external symbol __imp__localtime64
options.cpp.obj : error LNK2019: unresolved external symbol __imp_strtod referenced in function "public: double __cdecl CSimpleIniTempl<char,struct SI_GenericNoCase<char>,class SI_ConvertA<char> >::GetDoubleValue(char const *,char const *,double,bool *)const " (?GetDoubleValue@?$CSimpleIniTempl@DU?$SI_GenericNoCase@D@@V?$SI_ConvertA@D@@@@QEBANPEBD0NPEA_N@Z)
zt.lib(OSUtils.cpp.obj) : error LNK2001: unresolved external symbol __imp_strtod
options.cpp.obj : error LNK2019: unresolved external symbol __imp_strtol referenced in function "public: long __cdecl CSimpleIniTempl<char,struct SI_GenericNoCase<char>,class SI_ConvertA<char> >::GetLongValue(char const *,char const *,long,bool *)const " (?GetLongValue@?$CSimpleIniTempl@DU?$SI_GenericNoCase@D@@V?$SI_ConvertA@D@@@@QEBAJPEBD0JPEA_N@Z)
demomode.cpp.obj : error LNK2001: unresolved external symbol __imp_strtol
options.cpp.obj : error LNK2019: unresolved external symbol __imp___stdio_common_vsprintf_s referenced in function _vsprintf_s_l
options.cpp.obj : error LNK2019: unresolved external symbol __imp_wcslen referenced in function "private: int __cdecl CSimpleIniTempl<char,struct SI_GenericNoCase<char>,class SI_ConvertA<char> >::CopyString(char const * &)" (?CopyString@?$CSimpleIniTempl@DU?$SI_GenericNoCase@D@@V?$SI_ConvertA@D@@@@AEAAHAEAPEBD@Z)
fmtd.lib(format.cc.obj) : error LNK2001: unresolved external symbol __imp_wcslen
zt.lib(Events.cpp.obj) : error LNK2001: unresolved external symbol __imp__wassert
zt.lib(OSUtils.cpp.obj) : error LNK2001: unresolved external symbol __imp__wassert
SDL_audiolib.lib(Resampler.cpp.obj) : error LNK2001: unresolved external symbol __imp__wassert
SDL_audiolib.lib(stream_p.cpp.obj) : error LNK2001: unresolved external symbol __imp__wassert
SDL_audiolib.lib(dr_mp3.c.obj) : error LNK2001: unresolved external symbol __imp__wassert
SDL_audiolib.lib(dr_wav.c.obj) : error LNK2001: unresolved external symbol __imp__wassert
sodium.lib(generichash_blake2b.c.obj) : error LNK2001: unresolved external symbol __imp__wassert
sodium.lib(blake2b-ref.c.obj) : error LNK2001: unresolved external symbol __imp__wassert
sodium.lib(codecs.c.obj) : error LNK2001: unresolved external symbol __imp__wassert
SDL_audiolib.lib(Decoder.cpp.obj) : error LNK2001: unresolved external symbol __imp__wassert
libsmackerdec.lib(BitReader.cpp.obj) : error LNK2001: unresolved external symbol __imp__wassert
sodium.lib(core.c.obj) : error LNK2001: unresolved external symbol __imp__wassert
sodium.lib(randombytes.c.obj) : error LNK2001: unresolved external symbol __imp__wassert
sodium.lib(utils.c.obj) : error LNK2001: unresolved external symbol __imp__wassert
options.cpp.obj : error LNK2001: unresolved external symbol __imp__wassert
abstract_net.cpp.obj : error LNK2001: unresolved external symbol __imp__wassert
tcp_client.cpp.obj : error LNK2001: unresolved external symbol __imp__wassert
tcp_server.cpp.obj : error LNK2001: unresolved external symbol __imp__wassert
palette.cpp.obj : error LNK2019: unresolved external symbol pow referenced in function "void __cdecl devilution::ApplyGamma(struct SDL_Color *,struct SDL_Color const *,int)" (?ApplyGamma@devilution@@YAXPEAUSDL_Color@@PEBU2@H@Z)
libpng16_staticd.lib(png.c.obj) : error LNK2001: unresolved external symbol pow
  Hint on symbols that are defined and could potentially match:
    "float __cdecl pow(float,float)" (?pow@@YAMMM@Z)
controller_motion.cpp.obj : error LNK2019: unresolved external symbol fabs referenced in function fabsf
SDL_audiolib.lib(resample.c.obj) : error LNK2001: unresolved external symbol fabs
  Hint on symbols that are defined and could potentially match:
    "float __cdecl fabs(float)" (?fabs@@YAMM@Z)
controller_motion.cpp.obj : error LNK2019: unresolved external symbol __imp_sqrtf referenced in function "float __cdecl sqrt(float)" (?sqrt@@YAMM@Z)
zt.lib(Bond.cpp.obj) : error LNK2001: unresolved external symbol __imp_sqrtf
demomode.cpp.obj : error LNK2019: unresolved external symbol __imp_strtof referenced in function "float __cdecl std::stof(class std::basic_string<char,struct std::char_traits<char>,class std::allocator<char> > const &,unsigned __int64 *)" (?stof@std@@YAMAEBV?$basic_string@DU?$char_traits@D@std@@V?$allocator@D@2@@1@PEA_K@Z)
zt.lib(miniupnpc.c.obj) : error LNK2001: unresolved external symbol strchr
zt.lib(miniwget.c.obj) : error LNK2001: unresolved external symbol strchr
mpq_writer.cpp.obj : error LNK2001: unresolved external symbol strchr
sodium.lib(codecs.c.obj) : error LNK2001: unresolved external symbol strchr
zt.lib(InetAddress.cpp.obj) : error LNK2001: unresolved external symbol strchr
zt.lib(OSUtils.cpp.obj) : error LNK2001: unresolved external symbol strchr
  Hint on symbols that are defined and could potentially match:
    "char * __cdecl strchr(char * const,int)" (?strchr@@YAPEADQEADH@Z)
mpq_writer.cpp.obj : error LNK2019: unresolved external symbol __imp_strerror referenced in function "public: bool __cdecl devilution::MpqWriter::Open(char const *)" (?Open@MpqWriter@devilution@@QEAA_NPEBD@Z)
protocol_zt.cpp.obj : error LNK2001: unresolved external symbol __imp_strerror
libpng16_staticd.lib(pngread.c.obj) : error LNK2001: unresolved external symbol __imp_strerror
libpng16_staticd.lib(pngwrite.c.obj) : error LNK2001: unresolved external symbol __imp_strerror
file_util.cpp.obj : error LNK2019: unresolved external symbol __imp__wfopen referenced in function "struct _iobuf * __cdecl devilution::FOpen(char const *,char const *)" (?FOpen@devilution@@YAPEAU_iobuf@@PEBD0@Z)
libmpq.lib(mpq.c.obj) : error LNK2001: unresolved external symbol __imp__wfopen
language.cpp.obj : error LNK2019: unresolved external symbol strstr referenced in function "void __cdecl `anonymous namespace'::ParsePluralForms(char const *)" (?ParsePluralForms@?A0xfa7e33cb@@YAXPEBD@Z)
diabloui.cpp.obj : error LNK2001: unresolved external symbol strstr
zt.lib(miniwget.c.obj) : error LNK2001: unresolved external symbol strstr
  Hint on symbols that are defined and could potentially match:
    "char * __cdecl strstr(char * const,char const * const)" (?strstr@@YAPEADQEADQEBD@Z)
language.cpp.obj : error LNK2019: unresolved external symbol __imp_isblank referenced in function "char * __cdecl `anonymous namespace'::StrTrimLeft(char *)" (?StrTrimLeft@?A0xfa7e33cb@@YAPEADPEAD@Z)
sodium.lib(core.c.obj) : error LNK2001: unresolved external symbol __imp_abort
sodium.lib(utils.c.obj) : error LNK2001: unresolved external symbol __imp_abort
sodium.lib(ed25519_ref10.c.obj) : error LNK2001: unresolved external symbol __imp_abort
tcp_client.cpp.obj : error LNK2001: unresolved external symbol __imp_abort
tcp_server.cpp.obj : error LNK2001: unresolved external symbol __imp_abort
game_controller.cpp.obj : error LNK2001: unresolved external symbol __imp_abort
libpng16_staticd.lib(pngerror.c.obj) : error LNK2001: unresolved external symbol __imp_abort
loopback.cpp.obj : error LNK2001: unresolved external symbol __imp_abort
packet.cpp.obj : error LNK2001: unresolved external symbol __imp_abort
storm_net.cpp.obj : error LNK2001: unresolved external symbol __imp_abort
misc_msg.cpp.obj : error LNK2001: unresolved external symbol __imp_abort
diabloui.cpp.obj : error LNK2001: unresolved external symbol __imp_abort
abstract_net.cpp.obj : error LNK2001: unresolved external symbol __imp_abort
base.cpp.obj : error LNK2001: unresolved external symbol __imp_abort
frame_queue.cpp.obj : error LNK2001: unresolved external symbol __imp_abort
diabloui.cpp.obj : error LNK2019: unresolved external symbol __imp_strpbrk referenced in function "bool __cdecl devilution::UiValidPlayerName(char const *)" (?UiValidPlayerName@devilution@@YA_NPEBD@Z)
selhero.cpp.obj : error LNK2019: unresolved external symbol __imp_rand referenced in function "char const * __cdecl devilution::`anonymous namespace'::SelheroGenerateName(enum devilution::HeroClass)" (?SelheroGenerateName@?A0xcccdbe3c@devilution@@YAPEBDW4HeroClass@2@@Z)
zt.lib(Node.cpp.obj) : error LNK2001: unresolved external symbol __imp__aligned_free
abstract_net.cpp.obj : error LNK2001: unresolved external symbol __imp__aligned_free
tcp_client.cpp.obj : error LNK2001: unresolved external symbol __imp__aligned_free
tcp_server.cpp.obj : error LNK2001: unresolved external symbol __imp__aligned_free
zt.lib(NodeService.cpp.obj) : error LNK2001: unresolved external symbol __imp__aligned_free
zt.lib(Node.cpp.obj) : error LNK2001: unresolved external symbol __imp__aligned_malloc
abstract_net.cpp.obj : error LNK2001: unresolved external symbol __imp__aligned_malloc
tcp_client.cpp.obj : error LNK2001: unresolved external symbol __imp__aligned_malloc
tcp_server.cpp.obj : error LNK2001: unresolved external symbol __imp__aligned_malloc
zt.lib(NodeService.cpp.obj) : error LNK2001: unresolved external symbol __imp__aligned_malloc
abstract_net.cpp.obj : error LNK2019: unresolved external symbol __imp_strnlen referenced in function "private: void __cdecl devilution::net::base_protocol<class devilution::net::protocol_zt>::recv_decrypted(class devilution::net::packet &,class devilution::net::protocol_zt::endpoint)" (?recv_decrypted@?$base_protocol@Vprotocol_zt@net@devilution@@@net@devilution@@AEAAXAEAVpacket@23@Vendpoint@protocol_zt@23@@Z)
zt.lib(Utilities.cpp.obj) : error LNK2001: unresolved external symbol __imp_strnlen
zt.lib(NodeService.cpp.obj) : error LNK2001: unresolved external symbol __imp_strnlen
zt.lib(Controls.cpp.obj) : error LNK2001: unresolved external symbol __imp_strnlen
SDL_audiolib.lib(stream_p.cpp.obj) : error LNK2001: unresolved external symbol __std_type_info_compare
abstract_net.cpp.obj : error LNK2001: unresolved external symbol __std_type_info_compare
sound.cpp.obj : error LNK2001: unresolved external symbol __std_type_info_compare
tcp_client.cpp.obj : error LNK2001: unresolved external symbol __std_type_info_compare
tcp_server.cpp.obj : error LNK2001: unresolved external symbol __std_type_info_compare
MSVCRTD.lib(ehvecdtr.obj) : error LNK2001: unresolved external symbol __C_specific_handler
MSVCRTD.lib(ehvecctr.obj) : error LNK2001: unresolved external symbol __C_specific_handler
abstract_net.cpp.obj : error LNK2001: unresolved external symbol __C_specific_handler
tcp_server.cpp.obj : error LNK2001: unresolved external symbol __C_specific_handler
zt.lib(sys_arch.c.obj) : error LNK2001: unresolved external symbol __C_specific_handler
MSVCRTD.lib(utility.obj) : error LNK2001: unresolved external symbol __C_specific_handler
soundsample.cpp.obj : error LNK2019: unresolved external symbol __imp_powf referenced in function "float __cdecl pow(float,float)" (?pow@@YAMMM@Z)
SDL_audiolib.lib(stream_p.cpp.obj) : error LNK2001: unresolved external symbol __imp_powf
soundsample.cpp.obj : error LNK2019: unresolved external symbol copysignf referenced in function "float __cdecl copysign(float,float)" (?copysign@@YAMMM@Z)
event_handlers.cpp.obj : error LNK2019: unresolved external symbol roundf referenced in function "float __cdecl round(float)" (?round@@YAMM@Z)
gamepad.cpp.obj : error LNK2001: unresolved external symbol roundf
zt.lib(Bond.cpp.obj) : error LNK2001: unresolved external symbol roundf
gamepad.cpp.obj : error LNK2019: unresolved external symbol atan2 referenced in function "double __cdecl atan2<int,int,0>(int,int)" (??$atan2@HH$0A@@@YANHH@Z)
gamepad.cpp.obj : error LNK2019: unresolved external symbol __imp_round referenced in function "public: void __cdecl devilution::VirtualDirectionPad::UpdatePosition(struct devilution::Point)" (?UpdatePosition@VirtualDirectionPad@devilution@@QEAAXUPoint@2@@Z)
SDL_image.lib(IMG_png.c.obj) : error LNK2019: unresolved external symbol longjmp referenced in function IMG_LoadPNG_RW
libpng16_staticd.lib(png.c.obj) : error LNK2001: unresolved external symbol longjmp
libpng16_staticd.lib(pngerror.c.obj) : error LNK2001: unresolved external symbol longjmp
SDL_image.lib(IMG_png.c.obj) : error LNK2019: unresolved external symbol _setjmp referenced in function IMG_LoadPNG_RW
libpng16_staticd.lib(png.c.obj) : error LNK2001: unresolved external symbol _setjmp
libpng16_staticd.lib(pngerror.c.obj) : error LNK2001: unresolved external symbol _setjmp
MSVCRTD.lib(_setjmp.obj) : error LNK2001: unresolved external symbol _setjmp
zt.lib(upnpreplyparse.c.obj) : error LNK2001: unresolved external symbol __imp_free
zt.lib(portlistingparse.c.obj) : error LNK2001: unresolved external symbol __imp_free
zt.lib(IncomingPacket.cpp.obj) : error LNK2001: unresolved external symbol __imp_free
zt.lib(minissdpc.c.obj) : error LNK2001: unresolved external symbol __imp_free
zt.lib(miniwget.c.obj) : error LNK2001: unresolved external symbol __imp_free
zt.lib(minisoap.c.obj) : error LNK2001: unresolved external symbol __imp_free
zt.lib(upnpdev.c.obj) : error LNK2001: unresolved external symbol __imp_free
zt.lib(miniupnpc.c.obj) : error LNK2001: unresolved external symbol __imp_free
zt.lib(upnpcommands.c.obj) : error LNK2001: unresolved external symbol __imp_free
zt.lib(Membership.cpp.obj) : error LNK2001: unresolved external symbol __imp_free
zt.lib(Peer.cpp.obj) : error LNK2001: unresolved external symbol __imp_free
zt.lib(Topology.cpp.obj) : error LNK2001: unresolved external symbol __imp_free
zt.lib(Switch.cpp.obj) : error LNK2001: unresolved external symbol __imp_free
zt.lib(SelfAwareness.cpp.obj) : error LNK2001: unresolved external symbol __imp_free
zt.lib(Node.cpp.obj) : error LNK2001: unresolved external symbol __imp_free
zt.lib(Multicaster.cpp.obj) : error LNK2001: unresolved external symbol __imp_free
zt.lib(Trace.cpp.obj) : error LNK2001: unresolved external symbol __imp_free
zt.lib(Network.cpp.obj) : error LNK2001: unresolved external symbol __imp_free
SDL_audiolib.lib(dr_wav.c.obj) : error LNK2001: unresolved external symbol __imp_free
zt.lib(mem.c.obj) : error LNK2001: unresolved external symbol __imp_free
zt.lib(Events.cpp.obj) : error LNK2001: unresolved external symbol __imp_free
zt.lib(NodeService.cpp.obj) : error LNK2001: unresolved external symbol __imp_free
sodium.lib(pwhash_argon2i.c.obj) : error LNK2001: unresolved external symbol __imp_free
sodium.lib(argon2.c.obj) : error LNK2001: unresolved external symbol __imp_free
SDL_audiolib.lib(resample.c.obj) : error LNK2001: unresolved external symbol __imp_free
SDL_audiolib.lib(dr_mp3.c.obj) : error LNK2001: unresolved external symbol __imp_free
libmpq.lib(extract.c.obj) : error LNK2001: unresolved external symbol __imp_free
ZLIB.lib(zutil.c.obj) : error LNK2001: unresolved external symbol __imp_free
BZip2.lib(bzlib.c.obj) : error LNK2001: unresolved external symbol __imp_free
sodium.lib(argon2-core.c.obj) : error LNK2001: unresolved external symbol __imp_free
libpng16_staticd.lib(png.c.obj) : error LNK2001: unresolved external symbol __imp_free
libpng16_staticd.lib(pngmem.c.obj) : error LNK2001: unresolved external symbol __imp_free
fmtd.lib(format.cc.obj) : error LNK2001: unresolved external symbol __imp_free
libmpq.lib(mpq.c.obj) : error LNK2001: unresolved external symbol __imp_free
libpng16_staticd.lib(png.c.obj) : error LNK2019: unresolved external symbol __imp_floor referenced in function png_fixed
libpng16_staticd.lib(pngrtran.c.obj) : error LNK2001: unresolved external symbol __imp_floor
SDL_audiolib.lib(resample.c.obj) : error LNK2001: unresolved external symbol __imp_floor
libpng16_staticd.lib(png.c.obj) : error LNK2019: unresolved external symbol __imp_frexp referenced in function png_ascii_from_fp
libpng16_staticd.lib(png.c.obj) : error LNK2019: unresolved external symbol __imp_modf referenced in function png_ascii_from_fp
libpng16_staticd.lib(pngread.c.obj) : error LNK2019: unresolved external symbol __imp_fopen referenced in function png_image_begin_read_from_file
libpng16_staticd.lib(pngwrite.c.obj) : error LNK2001: unresolved external symbol __imp_fopen
zt.lib(OSUtils.cpp.obj) : error LNK2001: unresolved external symbol __imp_fopen
zt.lib(NodeService.cpp.obj) : error LNK2001: unresolved external symbol __imp_fopen
libpng16_staticd.lib(pngwrite.c.obj) : error LNK2019: unresolved external symbol __imp_ferror referenced in function png_image_write_to_file
libpng16_staticd.lib(pngwrite.c.obj) : error LNK2019: unresolved external symbol __imp_remove referenced in function png_image_write_to_file
libpng16_staticd.lib(pngwrite.c.obj) : error LNK2019: unresolved external symbol __imp__gmtime64 referenced in function gmtime
zt.lib(miniwget.c.obj) : error LNK2001: unresolved external symbol __imp___acrt_iob_func
zt.lib(minisoap.c.obj) : error LNK2001: unresolved external symbol __imp___acrt_iob_func
zt.lib(connecthostport.c.obj) : error LNK2001: unresolved external symbol __imp___acrt_iob_func
zt.lib(receivedata.c.obj) : error LNK2001: unresolved external symbol __imp___acrt_iob_func
zt.lib(Utils.cpp.obj) : error LNK2001: unresolved external symbol __imp___acrt_iob_func
zt.lib(NodeService.cpp.obj) : error LNK2001: unresolved external symbol __imp___acrt_iob_func
zt.lib(Bond.cpp.obj) : error LNK2001: unresolved external symbol __imp___acrt_iob_func
zt.lib(minissdpc.c.obj) : error LNK2001: unresolved external symbol __imp___acrt_iob_func
zt.lib(ip4_frag.c.obj) : error LNK2001: unresolved external symbol __imp___acrt_iob_func
zt.lib(etharp.c.obj) : error LNK2001: unresolved external symbol __imp___acrt_iob_func
zt.lib(api_msg.c.obj) : error LNK2001: unresolved external symbol __imp___acrt_iob_func
zt.lib(tcp_out.c.obj) : error LNK2001: unresolved external symbol __imp___acrt_iob_func
zt.lib(api_lib.c.obj) : error LNK2001: unresolved external symbol __imp___acrt_iob_func
zt.lib(tcp.c.obj) : error LNK2001: unresolved external symbol __imp___acrt_iob_func
zt.lib(udp.c.obj) : error LNK2001: unresolved external symbol __imp___acrt_iob_func
zt.lib(memp.c.obj) : error LNK2001: unresolved external symbol __imp___acrt_iob_func
zt.lib(sys_arch.c.obj) : error LNK2001: unresolved external symbol __imp___acrt_iob_func
zt.lib(pbuf.c.obj) : error LNK2001: unresolved external symbol __imp___acrt_iob_func
zt.lib(netif.c.obj) : error LNK2001: unresolved external symbol __imp___acrt_iob_func
zt.lib(netbuf.c.obj) : error LNK2001: unresolved external symbol __imp___acrt_iob_func
libpng16_staticd.lib(pngerror.c.obj) : error LNK2001: unresolved external symbol __imp___acrt_iob_func
fmtd.lib(format.cc.obj) : error LNK2001: unresolved external symbol __imp___acrt_iob_func
BZip2.lib(bz_internal_error.c.obj) : error LNK2001: unresolved external symbol __imp___acrt_iob_func
zt.lib(sockets.c.obj) : error LNK2001: unresolved external symbol __imp___acrt_iob_func
zt.lib(miniwget.c.obj) : error LNK2001: unresolved external symbol __imp___stdio_common_vfprintf
zt.lib(minisoap.c.obj) : error LNK2001: unresolved external symbol __imp___stdio_common_vfprintf
zt.lib(connecthostport.c.obj) : error LNK2001: unresolved external symbol __imp___stdio_common_vfprintf
zt.lib(receivedata.c.obj) : error LNK2001: unresolved external symbol __imp___stdio_common_vfprintf
zt.lib(Utils.cpp.obj) : error LNK2001: unresolved external symbol __imp___stdio_common_vfprintf
zt.lib(NodeService.cpp.obj) : error LNK2001: unresolved external symbol __imp___stdio_common_vfprintf
zt.lib(Bond.cpp.obj) : error LNK2001: unresolved external symbol __imp___stdio_common_vfprintf
zt.lib(minissdpc.c.obj) : error LNK2001: unresolved external symbol __imp___stdio_common_vfprintf
zt.lib(ip4_frag.c.obj) : error LNK2001: unresolved external symbol __imp___stdio_common_vfprintf
zt.lib(etharp.c.obj) : error LNK2001: unresolved external symbol __imp___stdio_common_vfprintf
zt.lib(api_msg.c.obj) : error LNK2001: unresolved external symbol __imp___stdio_common_vfprintf
zt.lib(tcp_out.c.obj) : error LNK2001: unresolved external symbol __imp___stdio_common_vfprintf
zt.lib(api_lib.c.obj) : error LNK2001: unresolved external symbol __imp___stdio_common_vfprintf
zt.lib(tcp.c.obj) : error LNK2001: unresolved external symbol __imp___stdio_common_vfprintf
zt.lib(udp.c.obj) : error LNK2001: unresolved external symbol __imp___stdio_common_vfprintf
zt.lib(memp.c.obj) : error LNK2001: unresolved external symbol __imp___stdio_common_vfprintf
zt.lib(sys_arch.c.obj) : error LNK2001: unresolved external symbol __imp___stdio_common_vfprintf
zt.lib(pbuf.c.obj) : error LNK2001: unresolved external symbol __imp___stdio_common_vfprintf
zt.lib(netif.c.obj) : error LNK2001: unresolved external symbol __imp___stdio_common_vfprintf
zt.lib(netbuf.c.obj) : error LNK2001: unresolved external symbol __imp___stdio_common_vfprintf
libpng16_staticd.lib(pngerror.c.obj) : error LNK2001: unresolved external symbol __imp___stdio_common_vfprintf
fmtd.lib(format.cc.obj) : error LNK2001: unresolved external symbol __imp___stdio_common_vfprintf
BZip2.lib(bz_internal_error.c.obj) : error LNK2001: unresolved external symbol __imp___stdio_common_vfprintf
zt.lib(sockets.c.obj) : error LNK2001: unresolved external symbol __imp___stdio_common_vfprintf
libpng16_staticd.lib(pngget.c.obj) : error LNK2019: unresolved external symbol __imp_atof referenced in function png_get_sCAL
zt.lib(minissdpc.c.obj) : error LNK2001: unresolved external symbol __imp_malloc
zt.lib(miniwget.c.obj) : error LNK2001: unresolved external symbol __imp_malloc
zt.lib(minisoap.c.obj) : error LNK2001: unresolved external symbol __imp_malloc
zt.lib(upnpreplyparse.c.obj) : error LNK2001: unresolved external symbol __imp_malloc
zt.lib(SelfAwareness.cpp.obj) : error LNK2001: unresolved external symbol __imp_malloc
zt.lib(miniupnpc.c.obj) : error LNK2001: unresolved external symbol __imp_malloc
zt.lib(Membership.cpp.obj) : error LNK2001: unresolved external symbol __imp_malloc
zt.lib(IncomingPacket.cpp.obj) : error LNK2001: unresolved external symbol __imp_malloc
zt.lib(Network.cpp.obj) : error LNK2001: unresolved external symbol __imp_malloc
zt.lib(Peer.cpp.obj) : error LNK2001: unresolved external symbol __imp_malloc
zt.lib(Topology.cpp.obj) : error LNK2001: unresolved external symbol __imp_malloc
zt.lib(Switch.cpp.obj) : error LNK2001: unresolved external symbol __imp_malloc
zt.lib(NodeService.cpp.obj) : error LNK2001: unresolved external symbol __imp_malloc
zt.lib(Node.cpp.obj) : error LNK2001: unresolved external symbol __imp_malloc
zt.lib(Multicaster.cpp.obj) : error LNK2001: unresolved external symbol __imp_malloc
zt.lib(Trace.cpp.obj) : error LNK2001: unresolved external symbol __imp_malloc
SDL_audiolib.lib(dr_wav.c.obj) : error LNK2001: unresolved external symbol __imp_malloc
zt.lib(sys_arch.c.obj) : error LNK2001: unresolved external symbol __imp_malloc
zt.lib(mem.c.obj) : error LNK2001: unresolved external symbol __imp_malloc
zt.lib(Events.cpp.obj) : error LNK2001: unresolved external symbol __imp_malloc
BZip2.lib(bzlib.c.obj) : error LNK2001: unresolved external symbol __imp_malloc
sodium.lib(argon2-core.c.obj) : error LNK2001: unresolved external symbol __imp_malloc
sodium.lib(argon2.c.obj) : error LNK2001: unresolved external symbol __imp_malloc
SDL_audiolib.lib(dr_mp3.c.obj) : error LNK2001: unresolved external symbol __imp_malloc
libpng16_staticd.lib(pngmem.c.obj) : error LNK2001: unresolved external symbol __imp_malloc
libmpq.lib(mpq.c.obj) : error LNK2001: unresolved external symbol __imp_malloc
libmpq.lib(extract.c.obj) : error LNK2001: unresolved external symbol __imp_malloc
ZLIB.lib(zutil.c.obj) : error LNK2001: unresolved external symbol __imp_malloc
fmtd.lib(format.cc.obj) : error LNK2019: unresolved external symbol __imp_ceil referenced in function "int __cdecl fmt::v8::detail::format_float<double>(double,int,struct fmt::v8::detail::float_specs,class fmt::v8::detail::buffer<char> &)" (??$format_float@N@detail@v8@fmt@@YAHNHUfloat_specs@012@AEAV?$buffer@D@012@@Z)
fmtd.lib(format.cc.obj) : error LNK2019: unresolved external symbol __imp__fileno referenced in function "void __cdecl fmt::v8::detail::print(struct _iobuf *,class fmt::v8::basic_string_view<char>)" (?print@detail@v8@fmt@@YAXPEAU_iobuf@@V?$basic_string_view@D@23@@Z)
fmtd.lib(format.cc.obj) : error LNK2019: unresolved external symbol __imp___stdio_common_vsnprintf_s referenced in function _vsnprintf_s_l
fmtd.lib(format.cc.obj) : error LNK2019: unresolved external symbol __imp__calloc_dbg referenced in function "char * __cdecl std::_Maklocstr<char>(char const *,char *,struct _Cvtvec const &)" (??$_Maklocstr@D@std@@YAPEADPEBDPEADAEBU_Cvtvec@@@Z)
fmtd.lib(format.cc.obj) : error LNK2019: unresolved external symbol __imp_terminate referenced in function "void __cdecl fmt::v8::detail::assert_fail(char const *,int,char const *)" (?assert_fail@detail@v8@fmt@@YAXPEBDH0@Z)
fmtd.lib(format.cc.obj) : error LNK2019: unresolved external symbol __imp__get_osfhandle referenced in function "void __cdecl fmt::v8::detail::print(struct _iobuf *,class fmt::v8::basic_string_view<char>)" (?print@detail@v8@fmt@@YAXPEAU_iobuf@@V?$basic_string_view@D@23@@Z)
fmtd.lib(format.cc.obj) : error LNK2019: unresolved external symbol __imp__isatty referenced in function "void __cdecl fmt::v8::detail::print(struct _iobuf *,class fmt::v8::basic_string_view<char>)" (?print@detail@v8@fmt@@YAXPEAU_iobuf@@V?$basic_string_view@D@23@@Z)
zt.lib(miniupnpc.c.obj) : error LNK2001: unresolved external symbol __imp_calloc
zt.lib(upnpcommands.c.obj) : error LNK2001: unresolved external symbol __imp_calloc
zt.lib(portlistingparse.c.obj) : error LNK2001: unresolved external symbol __imp_calloc
libmpq.lib(mpq.c.obj) : error LNK2001: unresolved external symbol __imp_calloc
sodium.lib(pwhash_argon2i.c.obj) : error LNK2001: unresolved external symbol __imp_calloc
SDL_audiolib.lib(resample.c.obj) : error LNK2001: unresolved external symbol __imp_calloc
zt.lib(mem.c.obj) : error LNK2001: unresolved external symbol __imp_calloc
libmpq.lib(common.c.obj) : error LNK2019: unresolved external symbol strrchr referenced in function get_basename
zt.lib(NodeService.cpp.obj) : error LNK2019: unresolved external symbol __imp_strncmp referenced in function "public: int __cdecl ZeroTier::NodeService::nodeVirtualNetworkConfigFunction(unsigned __int64,void * *,enum ZT_VirtualNetworkConfigOperation,struct ZT_VirtualNetworkConfig const *)" (?nodeVirtualNetworkConfigFunction@NodeService@ZeroTier@@QEAAH_KPEAPEAXW4ZT_VirtualNetworkConfigOperation@@PEBUZT_VirtualNetworkConfig@@@Z)
zt.lib(upnpcommands.c.obj) : error LNK2001: unresolved external symbol __imp_strncmp
sodium.lib(crypto_pwhash.c.obj) : error LNK2001: unresolved external symbol __imp_strncmp
sodium.lib(argon2-encoding.c.obj) : error LNK2001: unresolved external symbol __imp_strncmp
zt.lib(def.c.obj) : error LNK2001: unresolved external symbol __imp_strncmp
zt.lib(Sockets.cpp.obj) : error LNK2001: unresolved external symbol __imp_strncmp
sodium.lib(utils.c.obj) : error LNK2019: unresolved external symbol __imp_raise referenced in function _out_of_bounds
SDL_audiolib.lib(Resampler.cpp.obj) : error LNK2019: unresolved external symbol ceilf referenced in function "float __cdecl ceil(float)" (?ceil@@YAMM@Z)
zt.lib(Bond.cpp.obj) : error LNK2001: unresolved external symbol ceilf
SDL_audiolib.lib(resample.c.obj) : error LNK2019: unresolved external symbol __imp_realloc referenced in function speex_realloc
SDL_audiolib.lib(dr_mp3.c.obj) : error LNK2001: unresolved external symbol __imp_realloc
SDL_audiolib.lib(dr_wav.c.obj) : error LNK2001: unresolved external symbol __imp_realloc
zt.lib(miniwget.c.obj) : error LNK2001: unresolved external symbol __imp_realloc
SDL_audiolib.lib(resample.c.obj) : error LNK2019: unresolved external symbol sin referenced in function sinc
zt.lib(Revocation.cpp.obj) : error LNK2001: unresolved external symbol _byteswap_ushort
zt.lib(Membership.cpp.obj) : error LNK2001: unresolved external symbol _byteswap_ushort
zt.lib(IncomingPacket.cpp.obj) : error LNK2001: unresolved external symbol _byteswap_ushort
zt.lib(Capability.cpp.obj) : error LNK2001: unresolved external symbol _byteswap_ushort
zt.lib(Tag.cpp.obj) : error LNK2001: unresolved external symbol _byteswap_ushort
zt.lib(Topology.cpp.obj) : error LNK2001: unresolved external symbol _byteswap_ushort
zt.lib(Switch.cpp.obj) : error LNK2001: unresolved external symbol _byteswap_ushort
zt.lib(OutboundMulticast.cpp.obj) : error LNK2001: unresolved external symbol _byteswap_ushort
zt.lib(CertificateOfOwnership.cpp.obj) : error LNK2001: unresolved external symbol _byteswap_ushort
zt.lib(NetworkConfig.cpp.obj) : error LNK2001: unresolved external symbol _byteswap_ushort
zt.lib(Network.cpp.obj) : error LNK2001: unresolved external symbol _byteswap_ushort
zt.lib(Bond.cpp.obj) : error LNK2001: unresolved external symbol _byteswap_ushort
zt.lib(Peer.cpp.obj) : error LNK2001: unresolved external symbol _byteswap_ushort
zt.lib(Node.cpp.obj) : error LNK2001: unresolved external symbol _byteswap_ushort
zt.lib(PortMapper.cpp.obj) : error LNK2001: unresolved external symbol _byteswap_ushort
zt.lib(VirtualTap.cpp.obj) : error LNK2001: unresolved external symbol _byteswap_ushort
zt.lib(Multicaster.cpp.obj) : error LNK2001: unresolved external symbol _byteswap_ushort
SDL_audiolib.lib(dr_wav.c.obj) : error LNK2001: unresolved external symbol _byteswap_ushort
zt.lib(Utilities.cpp.obj) : error LNK2001: unresolved external symbol _byteswap_ushort
zt.lib(InetAddress.cpp.obj) : error LNK2001: unresolved external symbol _byteswap_ushort
zt.lib(NodeService.cpp.obj) : error LNK2001: unresolved external symbol _byteswap_ushort
zt.lib(Revocation.cpp.obj) : error LNK2001: unresolved external symbol _byteswap_ulong
zt.lib(Membership.cpp.obj) : error LNK2001: unresolved external symbol _byteswap_ulong
zt.lib(IncomingPacket.cpp.obj) : error LNK2001: unresolved external symbol _byteswap_ulong
zt.lib(Capability.cpp.obj) : error LNK2001: unresolved external symbol _byteswap_ulong
zt.lib(Tag.cpp.obj) : error LNK2001: unresolved external symbol _byteswap_ulong
zt.lib(Switch.cpp.obj) : error LNK2001: unresolved external symbol _byteswap_ulong
zt.lib(AES.cpp.obj) : error LNK2001: unresolved external symbol _byteswap_ulong
zt.lib(OutboundMulticast.cpp.obj) : error LNK2001: unresolved external symbol _byteswap_ulong
zt.lib(CertificateOfOwnership.cpp.obj) : error LNK2001: unresolved external symbol _byteswap_ulong
zt.lib(VirtualTap.cpp.obj) : error LNK2001: unresolved external symbol _byteswap_ulong
zt.lib(Multicaster.cpp.obj) : error LNK2001: unresolved external symbol _byteswap_ulong
zt.lib(NetworkConfig.cpp.obj) : error LNK2001: unresolved external symbol _byteswap_ulong
zt.lib(Network.cpp.obj) : error LNK2001: unresolved external symbol _byteswap_ulong
SDL_audiolib.lib(dr_wav.c.obj) : error LNK2001: unresolved external symbol _byteswap_ulong
zt.lib(InetAddress.cpp.obj) : error LNK2001: unresolved external symbol _byteswap_ulong
zt.lib(SHA512.cpp.obj) : error LNK2001: unresolved external symbol _byteswap_ulong
zt.lib(Node.cpp.obj) : error LNK2001: unresolved external symbol _byteswap_ulong
zt.lib(Capability.cpp.obj) : error LNK2001: unresolved external symbol _byteswap_uint64
zt.lib(Tag.cpp.obj) : error LNK2001: unresolved external symbol _byteswap_uint64
zt.lib(Revocation.cpp.obj) : error LNK2001: unresolved external symbol _byteswap_uint64
zt.lib(CertificateOfMembership.cpp.obj) : error LNK2001: unresolved external symbol _byteswap_uint64
zt.lib(CertificateOfOwnership.cpp.obj) : error LNK2001: unresolved external symbol _byteswap_uint64
zt.lib(Membership.cpp.obj) : error LNK2001: unresolved external symbol _byteswap_uint64
zt.lib(IncomingPacket.cpp.obj) : error LNK2001: unresolved external symbol _byteswap_uint64
zt.lib(Switch.cpp.obj) : error LNK2001: unresolved external symbol _byteswap_uint64
zt.lib(AES.cpp.obj) : error LNK2001: unresolved external symbol _byteswap_uint64
zt.lib(AES_aesni.cpp.obj) : error LNK2001: unresolved external symbol _byteswap_uint64
zt.lib(OutboundMulticast.cpp.obj) : error LNK2001: unresolved external symbol _byteswap_uint64
zt.lib(Network.cpp.obj) : error LNK2001: unresolved external symbol _byteswap_uint64
zt.lib(Bond.cpp.obj) : error LNK2001: unresolved external symbol _byteswap_uint64
zt.lib(Peer.cpp.obj) : error LNK2001: unresolved external symbol _byteswap_uint64
zt.lib(Topology.cpp.obj) : error LNK2001: unresolved external symbol _byteswap_uint64
zt.lib(SHA512.cpp.obj) : error LNK2001: unresolved external symbol _byteswap_uint64
zt.lib(Node.cpp.obj) : error LNK2001: unresolved external symbol _byteswap_uint64
zt.lib(Multicaster.cpp.obj) : error LNK2001: unresolved external symbol _byteswap_uint64
zt.lib(NetworkConfig.cpp.obj) : error LNK2001: unresolved external symbol _byteswap_uint64
SDL_audiolib.lib(dr_wav.c.obj) : error LNK2001: unresolved external symbol _byteswap_uint64
zt.lib(Utilities.cpp.obj) : error LNK2001: unresolved external symbol _byteswap_uint64
zt.lib(InetAddress.cpp.obj) : error LNK2001: unresolved external symbol _byteswap_uint64
zt.lib(Identity.cpp.obj) : error LNK2001: unresolved external symbol _byteswap_uint64
zt.lib(ip6_addr.c.obj) : error LNK2019: unresolved external symbol __imp_islower referenced in function ip6addr_aton
zt.lib(ip4_addr.c.obj) : error LNK2001: unresolved external symbol __imp_islower
zt.lib(ip6_addr.c.obj) : error LNK2019: unresolved external symbol __imp_isdigit referenced in function ip6addr_aton
zt.lib(ip4_addr.c.obj) : error LNK2001: unresolved external symbol __imp_isdigit
zt.lib(ip6_addr.c.obj) : error LNK2019: unresolved external symbol __imp_isxdigit referenced in function ip6addr_aton
zt.lib(ip4_addr.c.obj) : error LNK2001: unresolved external symbol __imp_isxdigit
zt.lib(miniwget.c.obj) : error LNK2001: unresolved external symbol __imp_isxdigit
zt.lib(ip4_addr.c.obj) : error LNK2019: unresolved external symbol __imp_isspace referenced in function ip4addr_aton
zt.lib(miniwget.c.obj) : error LNK2001: unresolved external symbol __imp_strncpy
zt.lib(connecthostport.c.obj) : error LNK2001: unresolved external symbol __imp_strncpy
zt.lib(upnpreplyparse.c.obj) : error LNK2001: unresolved external symbol __imp_strncpy
zt.lib(netdb.c.obj) : error LNK2001: unresolved external symbol __imp_strncpy
zt.lib(NodeService.cpp.obj) : error LNK2001: unresolved external symbol __imp_strncpy
zt.lib(miniupnpc.c.obj) : error LNK2001: unresolved external symbol __imp_strncpy
zt.lib(upnpcommands.c.obj) : error LNK2001: unresolved external symbol __imp_strncpy
zt.lib(Identity.cpp.obj) : error LNK2019: unresolved external symbol __imp__strtoui64 referenced in function "public: static unsigned __int64 __cdecl ZeroTier::Utils::hexStrToU64(char const *)" (?hexStrToU64@Utils@ZeroTier@@SA_KPEBD@Z)
zt.lib(OSUtils.cpp.obj) : error LNK2001: unresolved external symbol __imp__strtoui64
zt.lib(NodeService.cpp.obj) : error LNK2001: unresolved external symbol __imp__strtoui64
zt.lib(NetworkConfig.cpp.obj) : error LNK2001: unresolved external symbol __imp__strtoui64
zt.lib(Identity.cpp.obj) : error LNK2019: unresolved external symbol __imp_strtok_s referenced in function "public: static char * __cdecl ZeroTier::Utils::stok(char *,char const *,char * *)" (?stok@Utils@ZeroTier@@SAPEADPEADPEBDPEAPEAD@Z)
zt.lib(OSUtils.cpp.obj) : error LNK2019: unresolved external symbol __imp_strtoll referenced in function "private: enum nlohmann::detail::lexer_base<class nlohmann::basic_json<class std::map,class std::vector,class std::basic_string<char,struct std::char_traits<char>,class std::allocator<char> >,bool,__int64,unsigned __int64,double,class std::allocator,struct nlohmann::adl_serializer,class std::vector<unsigned char,class std::allocator<unsigned char> > > >::token_type __cdecl nlohmann::detail::lexer<class nlohmann::basic_json<class std::map,class std::vector,class std::basic_string<char,struct std::char_traits<char>,class std::allocator<char> >,bool,__int64,unsigned __int64,double,class std::allocator,struct nlohmann::adl_serializer,class std::vector<unsigned char,class std::allocator<unsigned char> > >,class nlohmann::detail::iterator_input_adapter<char const *> >::scan_number(void)" (?scan_number@?$lexer@V?$basic_json@Vmap@std@@Vvector@2@V?$basic_string@DU?$char_traits@D@std@@V?$allocator@D@2@@2@_N_J_KNVallocator@2@Uadl_serializer@nlohmann@@V?$vector@EV?$allocator@E@std@@@2@@nlohmann@@V?$iterator_input_adapter@PEBD@detail@2@@detail@nlohmann@@AEAA?AW4token_type@?$lexer_base@V?$basic_json@Vmap@std@@Vvector@2@V?$basic_string@DU?$char_traits@D@std@@V?$allocator@D@2@@2@_N_J_KNVallocator@2@Uadl_serializer@nlohmann@@V?$vector@EV?$allocator@E@std@@@2@@nlohmann@@@23@XZ)
zt.lib(OSUtils.cpp.obj) : error LNK2019: unresolved external symbol __imp_strtoull referenced in function "private: enum nlohmann::detail::lexer_base<class nlohmann::basic_json<class std::map,class std::vector,class std::basic_string<char,struct std::char_traits<char>,class std::allocator<char> >,bool,__int64,unsigned __int64,double,class std::allocator,struct nlohmann::adl_serializer,class std::vector<unsigned char,class std::allocator<unsigned char> > > >::token_type __cdecl nlohmann::detail::lexer<class nlohmann::basic_json<class std::map,class std::vector,class std::basic_string<char,struct std::char_traits<char>,class std::allocator<char> >,bool,__int64,unsigned __int64,double,class std::allocator,struct nlohmann::adl_serializer,class std::vector<unsigned char,class std::allocator<unsigned char> > >,class nlohmann::detail::iterator_input_adapter<char const *> >::scan_number(void)" (?scan_number@?$lexer@V?$basic_json@Vmap@std@@Vvector@2@V?$basic_string@DU?$char_traits@D@std@@V?$allocator@D@2@@2@_N_J_KNVallocator@2@Uadl_serializer@nlohmann@@V?$vector@EV?$allocator@E@std@@@2@@nlohmann@@V?$iterator_input_adapter@PEBD@detail@2@@detail@nlohmann@@AEAA?AW4token_type@?$lexer_base@V?$basic_json@Vmap@std@@Vvector@2@V?$basic_string@DU?$char_traits@D@std@@V?$allocator@D@2@@2@_N_J_KNVallocator@2@Uadl_serializer@nlohmann@@V?$vector@EV?$allocator@E@std@@@2@@nlohmann@@@23@XZ)
zt.lib(Controls.cpp.obj) : error LNK2001: unresolved external symbol __imp_strtoull
zt.lib(OSUtils.cpp.obj) : error LNK2019: unresolved external symbol __imp__stat64i32 referenced in function stat
zt.lib(OSUtils.cpp.obj) : error LNK2019: unresolved external symbol __imp_localeconv referenced in function "public: __cdecl nlohmann::detail::serializer<class nlohmann::basic_json<class std::map,class std::vector,class std::basic_string<char,struct std::char_traits<char>,class std::allocator<char> >,bool,__int64,unsigned __int64,double,class std::allocator,struct nlohmann::adl_serializer,class std::vector<unsigned char,class std::allocator<unsigned char> > > >::serializer<class nlohmann::basic_json<class std::map,class std::vector,class std::basic_string<char,struct std::char_traits<char>,class std::allocator<char> >,bool,__int64,unsigned __int64,double,class std::allocator,struct nlohmann::adl_serializer,class std::vector<unsigned char,class std::allocator<unsigned char> > > >(class std::shared_ptr<struct nlohmann::detail::output_adapter_protocol<char> >,char,enum nlohmann::detail::error_handler_t)" (??0?$serializer@V?$basic_json@Vmap@std@@Vvector@2@V?$basic_string@DU?$char_traits@D@std@@V?$allocator@D@2@@2@_N_J_KNVallocator@2@Uadl_serializer@nlohmann@@V?$vector@EV?$allocator@E@std@@@2@@nlohmann@@@detail@nlohmann@@QEAA@V?$shared_ptr@U?$output_adapter_protocol@D@detail@nlohmann@@@std@@DW4error_handler_t@12@@Z)
zt.lib(NodeService.cpp.obj) : error LNK2019: unresolved external symbol __imp_wcstombs referenced in function "public: void __cdecl ZeroTier::Binder::refresh<class ZeroTier::NodeService *,class ZeroTier::NodeService>(class ZeroTier::Phy<class ZeroTier::NodeService *> &,unsigned int *,unsigned int,class std::vector<struct ZeroTier::InetAddress,class std::allocator<struct ZeroTier::InetAddress> >,class ZeroTier::NodeService &)" (??$refresh@PEAVNodeService@ZeroTier@@V12@@Binder@ZeroTier@@QEAAXAEAV?$Phy@PEAVNodeService@ZeroTier@@@1@PEAIIV?$vector@UInetAddress@ZeroTier@@V?$allocator@UInetAddress@ZeroTier@@@std@@@std@@AEAVNodeService@1@@Z)
zt.lib(NetworkConfig.cpp.obj) : error LNK2019: unresolved external symbol __imp__strtoi64 referenced in function "public: static __int64 __cdecl ZeroTier::Utils::hexStrTo64(char const *)" (?hexStrTo64@Utils@ZeroTier@@SA_JPEBD@Z)
zt.lib(Bond.cpp.obj) : error LNK2019: unresolved external symbol __imp_expf referenced in function "public: void __cdecl ZeroTier::Bond::estimatePathQuality(__int64)" (?estimatePathQuality@Bond@ZeroTier@@QEAAX_J@Z)
zt.lib(Bond.cpp.obj) : error LNK2019: unresolved external symbol __imp_strftime referenced in function "public: void __cdecl ZeroTier::Bond::log(char const *,...)" (?log@Bond@ZeroTier@@QEAAXPEBDZZ)
zt.lib(miniupnpc.c.obj) : error LNK2019: unresolved external symbol __imp__strdup referenced in function GetUPNPUrls
zt.lib(upnpcommands.c.obj) : error LNK2019: unresolved external symbol __imp___stdio_common_vsscanf referenced in function _vsscanf_l
zt.lib(minissdpc.c.obj) : error LNK2019: unresolved external symbol __imp__memicmp referenced in function parseMSEARCHReply
zt.lib(miniwget.c.obj) : error LNK2001: unresolved external symbol __imp__memicmp
zt.lib(miniwget.c.obj) : error LNK2019: unresolved external symbol __imp_perror referenced in function miniwget3
msvcprtd.lib(locale0_implib.obj) : error LNK2019: unresolved external symbol __imp__free_dbg referenced in function "public: static void __cdecl std::_Fac_node::operator delete(void *)" (??3_Fac_node@std@@SAXPEAX@Z)
msvcprtd.lib(locale0_implib.obj) : error LNK2019: unresolved external symbol __imp__malloc_dbg referenced in function "public: static void * __cdecl std::_Fac_node::operator new(unsigned __int64)" (??2_Fac_node@std@@SAPEAX_K@Z)
MSVCRTD.lib(init.obj) : error LNK2019: unresolved external symbol _CrtDbgReport referenced in function _CRT_RTC_INIT
MSVCRTD.lib(init.obj) : error LNK2019: unresolved external symbol _CrtDbgReportW referenced in function _CRT_RTC_INITW
MSVCRTD.lib(thread_safe_statics.obj) : error LNK2001: unresolved external symbol _CrtDbgReportW
MSVCRTD.lib(new_scalar.obj) : error LNK2019: unresolved external symbol _callnewh referenced in function "void * __cdecl operator new(unsigned __int64)" (??2@YAPEAX_K@Z)
MSVCRTD.lib(new_scalar.obj) : error LNK2019: unresolved external symbol malloc referenced in function "void * __cdecl operator new(unsigned __int64)" (??2@YAPEAX_K@Z)
  Hint on symbols that are defined and could potentially match:
    "public: static void * __cdecl moodycamel::ConcurrentQueueDefaultTraits::malloc(unsigned __int64)" (?malloc@ConcurrentQueueDefaultTraits@moodycamel@@SAPEAX_K@Z)
MSVCRTD.lib(utility.obj) : error LNK2019: unresolved external symbol __vcrt_initialize referenced in function __scrt_initialize_crt
MSVCRTD.lib(utility.obj) : error LNK2019: unresolved external symbol __vcrt_uninitialize referenced in function __scrt_initialize_crt
MSVCRTD.lib(utility.obj) : error LNK2019: unresolved external symbol __vcrt_uninitialize_critical referenced in function __scrt_dllmain_uninitialize_critical
MSVCRTD.lib(utility.obj) : error LNK2019: unresolved external symbol __vcrt_thread_attach referenced in function __scrt_dllmain_crt_thread_attach
MSVCRTD.lib(utility.obj) : error LNK2019: unresolved external symbol __vcrt_thread_detach referenced in function __scrt_dllmain_crt_thread_attach
MSVCRTD.lib(utility.obj) : error LNK2019: unresolved external symbol _seh_filter_dll referenced in function __scrt_dllmain_exception_filter
MSVCRTD.lib(utility.obj) : error LNK2019: unresolved external symbol _is_c_termination_complete referenced in function __scrt_dllmain_uninitialize_c
MSVCRTD.lib(utility.obj) : error LNK2019: unresolved external symbol _configure_narrow_argv referenced in function "public: static int __cdecl __scrt_narrow_argv_policy::configure_argv(void)" (?configure_argv@__scrt_narrow_argv_policy@@SAHXZ)
MSVCRTD.lib(utility.obj) : error LNK2019: unresolved external symbol _initialize_narrow_environment referenced in function "public: static int __cdecl __scrt_narrow_environment_policy::initialize_environment(void)" (?initialize_environment@__scrt_narrow_environment_policy@@SAHXZ)
MSVCRTD.lib(utility.obj) : error LNK2019: unresolved external symbol _initialize_onexit_table referenced in function __scrt_initialize_onexit_tables
MSVCRTD.lib(utility.obj) : error LNK2019: unresolved external symbol _register_onexit_function referenced in function _onexit
MSVCRTD.lib(utility.obj) : error LNK2019: unresolved external symbol _execute_onexit_table referenced in function __scrt_dllmain_uninitialize_c
MSVCRTD.lib(utility.obj) : error LNK2019: unresolved external symbol _crt_atexit referenced in function _onexit
MSVCRTD.lib(utility.obj) : error LNK2019: unresolved external symbol _crt_at_quick_exit referenced in function at_quick_exit
MSVCRTD.lib(utility.obj) : error LNK2019: unresolved external symbol __acrt_initialize referenced in function __scrt_initialize_crt
MSVCRTD.lib(utility.obj) : error LNK2019: unresolved external symbol __acrt_uninitialize referenced in function __scrt_uninitialize_crt
MSVCRTD.lib(utility.obj) : error LNK2019: unresolved external symbol __acrt_uninitialize_critical referenced in function __scrt_dllmain_uninitialize_critical
MSVCRTD.lib(utility.obj) : error LNK2019: unresolved external symbol __acrt_thread_attach referenced in function __scrt_dllmain_crt_thread_attach
MSVCRTD.lib(utility.obj) : error LNK2019: unresolved external symbol __acrt_thread_detach referenced in function __scrt_dllmain_crt_thread_detach
MSVCRTD.lib(utility.obj) : error LNK2019: unresolved external symbol _cexit referenced in function __scrt_dllmain_uninitialize_c
MSVCRTD.lib(ehvecdtr.obj) : error LNK2019: unresolved external symbol terminate referenced in function "int __cdecl ArrayUnwindFilter(struct _EXCEPTION_POINTERS *)" (?ArrayUnwindFilter@@YAHPEAU_EXCEPTION_POINTERS@@@Z)
MSVCRTD.lib(utility_desktop.obj) : error LNK2001: unresolved external symbol terminate
  Hint on symbols that are defined and could potentially match:
    "public: void __cdecl ZeroTier::NodeService::terminate(void)" (?terminate@NodeService@ZeroTier@@QEAAXXZ)
MSVCRTD.lib(ehvecdtr.obj) : error LNK2019: unresolved external symbol __current_exception referenced in function "int __cdecl ArrayUnwindFilter(struct _EXCEPTION_POINTERS *)" (?ArrayUnwindFilter@@YAHPEAU_EXCEPTION_POINTERS@@@Z)
MSVCRTD.lib(utility_desktop.obj) : error LNK2001: unresolved external symbol __current_exception
MSVCRTD.lib(ehvecdtr.obj) : error LNK2019: unresolved external symbol __current_exception_context referenced in function "int __cdecl ArrayUnwindFilter(struct _EXCEPTION_POINTERS *)" (?ArrayUnwindFilter@@YAHPEAU_EXCEPTION_POINTERS@@@Z)
MSVCRTD.lib(utility_desktop.obj) : error LNK2001: unresolved external symbol __current_exception_context
MSVCRTD.lib(error.obj) : error LNK2019: unresolved external symbol strcpy_s referenced in function "void __cdecl _RTC_StackFailure(void *,char const *)" (?_RTC_StackFailure@@YAXPEAXPEBD@Z)
MSVCRTD.lib(error.obj) : error LNK2019: unresolved external symbol strcat_s referenced in function "void __cdecl _RTC_StackFailure(void *,char const *)" (?_RTC_StackFailure@@YAXPEAXPEBD@Z)
MSVCRTD.lib(error.obj) : error LNK2019: unresolved external symbol __stdio_common_vsprintf_s referenced in function _vsprintf_s_l
MSVCRTD.lib(error.obj) : error LNK2001: unresolved external symbol __C_specific_handler_noexcept
MSVCRTD.lib(_setjmp.obj) : error LNK2001: unresolved external symbol __intrinsic_setjmp
MSVCRTD.lib(delete_scalar.obj) : error LNK2019: unresolved external symbol _free_dbg referenced in function "void __cdecl operator delete(void *)" (??3@YAXPEAX@Z)
MSVCRTD.lib(pdblkup.obj) : error LNK2019: unresolved external symbol _wmakepath_s referenced in function "int __cdecl GetPdbDllPathFromFilePath(wchar_t const *,wchar_t *,unsigned __int64)" (?GetPdbDllPathFromFilePath@@YAHPEB_WPEA_W_K@Z)
MSVCRTD.lib(pdblkup.obj) : error LNK2019: unresolved external symbol _wsplitpath_s referenced in function "int __cdecl GetPdbDllPathFromFilePath(wchar_t const *,wchar_t *,unsigned __int64)" (?GetPdbDllPathFromFilePath@@YAHPEB_WPEA_W_K@Z)
MSVCRTD.lib(pdblkup.obj) : error LNK2019: unresolved external symbol wcscpy_s referenced in function "int __cdecl GetPdbDllPathFromFilePath(wchar_t const *,wchar_t *,unsigned __int64)" (?GetPdbDllPathFromFilePath@@YAHPEB_WPEA_W_K@Z)
MSVCRTD.lib(pdblkup.obj) : error LNK2019: unresolved external symbol __vcrt_GetModuleFileNameW referenced in function "struct HINSTANCE__ * __cdecl GetPdbDll(void)" (?GetPdbDll@@YAPEAUHINSTANCE__@@XZ)
MSVCRTD.lib(pdblkup.obj) : error LNK2019: unresolved external symbol __vcrt_GetModuleHandleW referenced in function "struct HINSTANCE__ * __cdecl GetPdbDll(void)" (?GetPdbDll@@YAPEAUHINSTANCE__@@XZ)
MSVCRTD.lib(pdblkup.obj) : error LNK2019: unresolved external symbol __vcrt_LoadLibraryExW referenced in function "struct HINSTANCE__ * __cdecl GetPdbDll(void)" (?GetPdbDll@@YAPEAUHINSTANCE__@@XZ)

libdevilutionx_so.dll : fatal error LNK1120: 167 unresolved externals

ninja: build stopped: subcommand failed.
```


</details>

Reply:
I found the cause of the issue.
The [entry code for the MSVCRT](https://github.com/libsdl-org/SDL/blob/def60ce6fe53de7250fcb3cb2f301fa93b004434/src/SDL.c#L619-L639) was still present in the static library.

I think what happened next is the Microsoft Linker saw the `_DllMainCRTStartup` symbol and decided not linking the Microsoft C Runtime, which causes [all these link errors](https://github.com/libsdl-org/SDL/pull/5969#pullrequestreview-1054342086).

Adding `HAVE_LIBC=1` to the compile definitions also removes `_DllMainCRTStartup` from the static library, but has lots of "collateral damage" :)